---
title: "Gene2Phenotype Cardiac Gene Panel"
always_allow_html: true
output:
  html_document:
    toc: true
    toc_depth: 5
    toc_float: true
    theme: lumen
    keep_md: true
  github_document:
    preserve_yaml: false
    html_preview: false
    keep_html: false
knit: (function(input, ...) {
  rmarkdown::render(
    input, output_dir=".", clean=T, output_format="html_document");
  rmarkdown::render(
    input, output_dir=".", clean=T, output_format="github_document")
  })
---


```{css, echo=FALSE}
  #TOC {
    max-width: 100%;
  }
  
  div:has(> #TOC) {
    display: flex;
    flex-direction: row-reverse;
}
```


```{r message=FALSE, warning=FALSE, include=FALSE}
suppressWarnings(suppressPackageStartupMessages({
library(tidyverse)
library(readxl)
library(kableExtra)
}))



```

```{r read_g2p, message=FALSE, warning=FALSE, include=FALSE}

g2p <- read_csv("Cardiac_G2P.csv", col_types = cols(.default = col_character())) %>% 
  select(-Narrative_2)

# get arrhythmia genes for chapter change
arri_genes <- g2p %>% filter(!str_detect(.$`Referral indication`,c("ARVC|DCM|HCM"))) %>% pull(Gene)

chapter1 <- "Inherited Arrhythmia Syndromes"
chapter2 <- "Cardiomyopathy"


```

<h2>Overview</h2>

CardiacG2P is an evidence-based dataset for inherited cardiac condition gene disease pairs. Each entry annotates a disease-associated gene with information about the gene-disease relationship, including the inheritance mode and allelic requirement, information pertaining to disease mechanism (represented as a disease-associated variant consequence), and known disease-relevant variant classes at a defined locus.

This dataset is also available through G2P <https://www.ebi.ac.uk/gene2phenotype>

<h2>Citation</h2>

*Katherine S Josephs, Angharad M Roberts, Pantazis Theotokis, et al. Beyond gene-disease validity: capturing structured data on inheritance, allelic-requirement, disease-relevant variant classes, and disease mechanism for inherited cardiac conditions.*

<https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-023-01246-8>

```{r render_tables, echo=FALSE, results='asis'}
options(knitr.kable.NA = 'NA')
ref_ind <- g2p[1, 3]
disease_group <- g2p[1, 4]

cat(paste0("<h2>",chapter1,"</h2>"))
cat(paste0("<h3>",ref_ind,"</h3>"))


for (i in 1:nrow(g2p)){
  
  gene <- g2p[i,] %>% pivot_longer(everything()) %>% set_names(slice(.,1)) %>% slice(.,-1) %>% tibble() %>% slice_head(n=-1)
  
  gene[11,1]<- paste0(gene[11,1],"\n\n[(Other potentially relevant variant classes)](#other-potentially-relevant-variant-classes)")

  if (i==(length(arri_genes)+1)) {
    cat(paste0("<h2>",chapter2,"</h2>"))
  }
  
  
  if ((g2p[i, 3]) != ref_ind) {
    cat(paste0("<h3>",g2p[i, 3],"</h3>"))
    ref_ind <- g2p[i, 3]
  }
  
  # if ((g2p[i, 1]) == "ACTC1") {
  #   cat(paste0("<h4>Isolated LVH</h4>"))
  #   #ref_ind <- g2p[i, 3]
  # }
  
  if ((g2p[i, 4]) != disease_group) {
    cat(paste0("<h4>",g2p[i, 4],"</h4>"))
    disease_group <- g2p[i, 4]
  }
  
  cat(paste0("<h5>",g2p[i, 1],"</h5>"))
  
  # print gene text
  print(
    kbl(gene, 
        full_width = T,
        booktabs = TRUE
        )%>%
    kable_styling(font_size =12,
                  html_font = "sans-serif",
                  bootstrap_options = c("striped", "condensed", "scale_down"))%>% 
      column_spec(1, width = "6cm") %>% 
      row_spec(0, bold = TRUE,font_size =14) %>% 
      row_spec(4, bold = TRUE)
  
  )
  
  # Add a new line before the narrative free text so the md format of the first paragraph is rendered correctly 
  narrative <- g2p[i,ncol(g2p)] %>% 
    tibble() %>%  set_names("Narrative") %>% transmute(Narrative = paste0("<br /> \n\n",Narrative))

  # print narrative text
  print(
    kbl(narrative,escape = F,
        full_width = T,
        booktabs = TRUE
        ) %>%
    kable_styling(font_size =12,
                  html_font = "sans-serif",
                bootstrap_options = c("striped", "condensed")) %>% 
      row_spec(0, bold = TRUE,font_size =14)
  
  )
  
  # Add page breaks after every gene to be rendered by md and html
  cat("\n")
  cat("<br>")
  cat("\n")
  cat("<p style=\"page-break-before: always;\">&nbsp;</p>")

   
}
```

## Other potentially relevant variant classes

Which variant classes should I filter for? The main table shows variant classes that have been reported to cause disease. We would expect other variant classes with similar consequences to cause disease (e.g. if NMD_triggering frameshift cause disease, then NMD_triggering nonsense probably do too). This table below shows the full list of variant classes mapped to each variant consequence.


```{r echo=FALSE, message=FALSE, warning=FALSE}
options(knitr.kable.NA = '')
Decreased_gene_product_level <- c("splice_region_variant","splice_acceptor_variant","splice_acceptor_variant_NMD_triggering","splice_donor_variant","splice_donor_variant_NMD_triggering","start_lost","frameshift_variant","frameshift_variant_NMD_triggering","stop_gained","stop_gained_NMD_triggering")

Absent_gene_product_level <- c("splice_acceptor_variant","splice_acceptor_variant_NMD_triggering","splice_donor_variant","splice_donor_variant_NMD_triggering","start_lost","frameshift_variant","frameshift_variant_NMD_triggering","stop_gained","stop_gained_NMD_triggering",NA)

Altered_gene_product_sequence <- c("splice_acceptor_variant","splice_acceptor_variant_NMD_escaping","splice_donor_variant","splice_donor_variant_NMD_escaping","frameshift_variant_NMD_escaping","stop_gained_NMD_escaping","stop_lost","missense_variant","inframe_insertion","inframe_deletion")


tibble(Decreased_gene_product_level,Absent_gene_product_level,Altered_gene_product_sequence) %>% 
  kbl(full_width = T,
      booktabs = TRUE) %>% 
  add_header_above(c("Disease-associated variant consequence" = 3))%>%
  kable_styling(bootstrap_options = c("striped", "condensed", "scale_down")) %>% 
  footnote(general = "Variant classes with a likelihood score of '3:possible', '4:probable' and '5:almost always' for the different disease associated variant consequences, see Tables 2 and 3 from Roberts et al.",
           footnote_as_chunk = F, title_format = c("italic", "underline"))


```

*Roberts AM, DiStefano MT, Rooney Riggs E, Josephs KS, Alkuraya FS, Amberger J, et al. Towards robust clinical genome interpretation: developing a consistent terminology to characterize disease-gene relationships - allelic requirement, inheritance modes and disease mechanisms. MedRxiv. 2023* 

<https://doi.org/10.1101/2023.03.30.23287948>.

```{r session_info, message=FALSE, warning=FALSE, include=FALSE}

sessionInfo()

```
