---
output:
  html_document:
    toc: true
    toc_depth: 5
    toc_float: true
    theme: lumen
    keep_md: true
---



```{r message=FALSE, warning=FALSE, include=FALSE}
suppressWarnings(suppressPackageStartupMessages({
library(tidyverse)
library(readxl)
library(kableExtra)
}))
```


<h1 style="font-size: 50px">Gene2Phenotype Cardiac Gene Panel</h1>

<p style=\"page-break-before: always;\">&nbsp;</p>


```{r read_g2p, message=FALSE, warning=FALSE, include=FALSE}

g2p <- read_csv("Cardiac_G2P.csv", col_types = cols(.default = col_character())) %>% 
  select(-Narrative_2)

# get arrhythmia genes for chapter change
arri_genes <- g2p %>% filter(!str_detect(.$`Referral indication`,c("ARVC|DCM|HCM"))) %>% pull(Gene)

chapter1 <- "Inherited Arrhythmia Syndromes"
chapter2 <- "Cardiomyopathy"


```


## Overview
CardiacG2P is an evidence-based dataset for inherited cardiac condition gene disease pairs. Each dataset entry annotates a disease with an allelic requirement, information pertaining to disease mechanism (represented as a disease-associated variant consequence), and known disease-relevant variant classes at a defined locus. This dataset is compatible with the existing G2P Ensembl Variant Effect Predictor plugin to support automated filtering of genomic variants accounting for inheritance pattern and mutational consequence. 

This dataset is also available through G2P https://www.ebi.ac.uk/gene2phenotype

## Citation
https://www.medrxiv.org/content/10.1101/2023.04.03.23287612v1

```{r render_tables, echo=FALSE, results='asis'}

ref_ind <- g2p[1, 3]
disease_group <- g2p[1, 4]

cat(paste0("<h2>",chapter1,"</h2>"))
cat(paste0("<h3>",ref_ind,"</h3>"))


for (i in 1:nrow(g2p)){
  
  gene <- g2p[i,] %>% pivot_longer(everything()) %>% set_names(slice(.,1)) %>% slice(.,-1) %>% tibble() %>% slice_head(n=-1)
  
  if (i==(length(arri_genes)+1)) {
    cat(paste0("<h2>",chapter2,"</h2>"))
  }
  
  
  if ((g2p[i, 3]) != ref_ind) {
    cat(paste0("<h3>",g2p[i, 3],"</h3>"))
    ref_ind <- g2p[i, 3]
  }
  
  # if ((g2p[i, 1]) == "ACTC1") {
  #   cat(paste0("<h4>Isolated LVH</h4>"))
  #   #ref_ind <- g2p[i, 3]
  # }
  
  if ((g2p[i, 4]) != disease_group) {
    cat(paste0("<h4>",g2p[i, 4],"</h4>"))
    disease_group <- g2p[i, 4]
  }
  
  cat(paste0("<h5>",g2p[i, 1],"</h5>"))
  
  # print gene text
  print(
    kbl(gene, 
        full_width = T,
        booktabs = TRUE
        )%>%
    kable_styling(font_size =12,
                  html_font = "sans-serif",
                  bootstrap_options = c("striped", "condensed", "scale_down"))%>% 
      column_spec(1, width = "6cm") %>% 
      row_spec(0, bold = TRUE,font_size =14) %>% 
      row_spec(4, bold = TRUE)
  
  )
  
  # Add a new line before the narrative free text so the md format of the first paragraph is rendered correctly 
  narrative <- g2p[i,ncol(g2p)] %>% 
    tibble() %>%  set_names("Narrative") %>% transmute(Narrative = paste0("<br /> \n\n",Narrative))

  # print narrative text
  print(
    kbl(narrative,escape = F,
        full_width = T,
        booktabs = TRUE
        ) %>%
    kable_styling(font_size =12,
                  html_font = "sans-serif",
                bootstrap_options = c("striped", "condensed")) %>% 
      row_spec(0, bold = TRUE,font_size =14)
  
  )
  
  # Add page breaks after every gene to be rendered by md and html
  cat("\n")
  cat("<br>")
  cat("\n")
  cat("<p style=\"page-break-before: always;\">&nbsp;</p>")

   
}
```

```{r session_info, message=FALSE, warning=FALSE, include=FALSE}

sessionInfo()

```

