---
output:
  html_document:
    toc: true
    toc_depth: 3
    toc_float: true
    theme: lumen
    keep_md: true
---



<h1 style="font-size: 50px">Cardiac G2P curations</h1>

<p style=\"page-break-before: always;\">&nbsp;</p>




<h2>INHERITED ARRHYTHMIA SYNDROMES</h2><h3>Brugada syndrome</h3><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> SCN5A </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 600163 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Brugada syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Brugada syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> SCN5A-related Brugada syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011001 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice region; splice acceptor; splice donor; frameshift variant NMD triggering; stop gained variant NMD triggering; missense; inframe insertion; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 9521325; 11748104; NBK1517; 25905440; 32850980; 25829473; 17075016; 17442746; 20564468; 20031634; 33164571; 29798782; 32533946; 20129283; 33131149; 30203441; 32893267; 29959160 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 29.01.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> *SCN5A*-related Brugada syndrome is caused by **decreased gene product level or altered gene product sequence** due to a variety of mechanisms (e.g. decreased expression of cardiac sodium channel, Nav1.5 in the sarcolemma, expression of non-functional channels, or altered gating properties leading to a decreased INa sodium current (e.g., delayed activation or earlier or faster inactivation)) (PMID: 29798782). The disease mechanism is loss of function.  

 To date, *SCN5A* is the only gene classified as having definitive evidence as a cause of monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). *SCN5A* pathogenic variants are identified in approx. 20- 30% of cases of European ancestry (PMID: 30139433; 33164571). This contribution may differ in other populations. 

 *SCN5A*-related Brugada syndrome is characterized by autosomal dominant inheritance with incomplete penetrance (PMID: 9521325; 11748104; 25905440; 20031634; 33164571; NBK1517). Hundreds of variants, both truncating and non-truncating, have been described in association with Brugada syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct variants in *SCN5A*: 193 missense, 32 nonsense, 38 frameshift, 21 splice-site, and 9 inframe deletions/insertions. More recently Walsh et al 2020 (PMID: 32893267) found that non truncating variants were highly enriched in European cases in the *SCN5A* transmembrane regions.

 Gain of function variants are associated with Long QT syndrome. Although loss of function appears to be the accepted mechanism in Brugada syndrome the *SCN5A* genotype/phenotype association is still not completely understood. Uncommonly, a single variant can cause both loss of peak current and gain of late current and can lead to a mixed phenotype of LQTS and BrS (PMID: 29806494). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><h3>Catecholaminergic polymorphic ventricular tachycardia (CPVT)</h3><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> RYR2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 180902 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Catecholaminergic polymorphic ventricular tachycardia (CPVT) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Classic CPVT phenotype </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> RYR2-related CPVT </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0017990 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; exon deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> Exon 3 deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 29453246; 11208676; 12093772; 17875969; 19216760; 23479668; 24394973; 26018045; 17081562; 30696458; 31112425 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 08.01.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 19.05.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> *RYR2* is associated with CPVT with autosomal dominant inheritance. The allelic requirement for pathogenicity is monoallelic autosomal, and the disease is typified by incomplete penetrance. Disease is due to an **altered gene product sequence**. 

  The first reports of *RYR2* variants in CPVT included numerous examples of *de novo* inheritance (Priori et al, 2001, PMID: 11208676; Priori et al, 2002, PMID: 12093772), although *RYR2* missense variants often occur through familial autosomal dominant inheritance. 

 The predicted functional consequence of *RYR2* pathogenic variants is altered gene product sequence. The majority of causative *RYR2* variants in CPVT patients are heterozygous missense variants which are detected in up to 60% of cases with a definitive CPVT diagnosis (Kapplinger et al, 2018, PMID: 29453246) and act through a gain-of-function mechanism. Pathogenic variants in CPVT cases tend to be clustered in a number of regions in the *RYR2* gene/protein:
 -  Original exon hotspots: 3–15, 44–50, 83–90 and 93–105 (George et al, 2007, PMID: 17081562).
 -  Updated exon hotspots: 3, 8, 14, 43, 47–49, 81, 83, 88–90, 93, 95, 97–101, 103, 105 (Kapplinger et al, 2018, PMID: 29453246).
 -  Hotspots based on unsupervised clustering algorithm (not restricted to exon boundaries): amino acid residues 2138–2538, 3935–4196 and 4721–4959 (Walsh et al, 2019, PMID: 30696458). 

 Rare cases of whole exon deletions of *RYR2* exon 3 have been described that can lead to a CPVT and/or left ventricular non-compaction phenotype (Bhuiyan et al, 2007, PMID: 17875969; Marjamaa et al, 2009, PMID: 19216760; Szentpali et al, 2013, PMID: 23479668; Onho et al, 2014, PMID: 24394973; Campbell et al, 2015, PMID: 26018045; Mazzarotto et al, 2021, PMID: 33500567). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CASQ2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 114251 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Catecholaminergic polymorphic ventricular tachycardia (CPVT) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Classic CPVT phenotype </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CASQ2-related CPVT </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0017990 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Absent gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice region; splice donor variant NMD triggering; splice acceptor variant NMD triggering; frameshift variant NMD triggering; stop gained variant NMD triggering; missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 12386154; 16908766; 21618644; 15176429; 32693635; 27157848 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 08.01.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 19.05.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> *CASQ2* is an established gene for autosomal recessive CPVT (the second most common genetic cause of the disease, responsible for up to 5% of cases); disease is due to **absent gene product level or altered gene product sequence**. For the ClinGen curation, the maximum points were achieved with only a small subset of initial genetic and experimental reports leading to a definitive classification. 

Biallelic loss-of-function variants in *CASQ2* (both homozygous and compound heterozygous) have been reported in numerous CPVT probands, including frameshift, nonsense and splice donor/acceptor variants, as well as other splice region variants with verified effects on splicing, and missense variants with verified loss-of-function effects. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CASQ2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 114251 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Catecholaminergic polymorphic ventricular tachycardia (CPVT) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Classic CPVT phenotype </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CASQ2-related CPVT </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0017990 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> MODERATE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice region; splice donor variant NMD triggering; splice acceptor variant NMD triggering; frameshift variant NMD triggering; stop gained variant NMD triggering; missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 12386154; 16908766; 21618644; 15176429; 32693635; 27157848 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 08.01.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 19.05.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> *CASQ2* is an established gene for autosomal recessive CPVT (the second most common genetic cause of the disease, responsible for up to 5% of cases).
 Recent reports have also associated monoallelic or heterozygous *CASQ2* variants with CPVT.  Although the evidence for this association is not definitive, available data suggests that disease is likely due to **decreased gene product level or altered gene product sequence**. The main evidence for autosomal dominant *CASQ2* association comes from an international multi-centre study describing CPVT patients with *CASQ2* variants (Ng et al, 2020, PMID: 32693635). This study included 12 probands with heterozygous variants in *CASQ2*, as well as an assessment of heterozygous relatives of probands with homozygous/compound heterozygous *CASQ2* variants (8/37 of these heterozygous relatives had a positive CPVT phenotype). While this study provides a substantive body of evidence to support autosomal dominant *CASQ2* association with CPVT, the data should be cautiously interpreted. The multi-centre nature of the study precluded standardised phenotyping of the probands and relatives and therefore not every phenotype-positive individual may have a definitive diagnosis of CPVT. Additionally, several of the variants described have a gnomAD population minor allele frequency that is incompatible with being a penetrant autosomal dominant variant for a disease with the prevalence of CPVT (&gt;1x10-5). The variants also included presumed nonsense mediated decay-escaping C-terminal truncating variants and a splice region variant without a proven effect on splicing. In contrast to 97% penetrance for homozygous/compound heterozygous individuals, penetrance for heterozygotes was only 33% although it is expected that the population level penetrance of *CASQ2* heterozygous variants will be lower still.
 Further support for the pathogenicity of heterozygous *CASQ2* variants comes from a study that described the heterozygous p.Lys180Arg variant segregating with disease in a family (the published LOD score was 3.0 although there were only five meioses between genotype and phenotype positive individuals) (Gray et al, 2016, PMID: 27157848). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CALM1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 114180 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Catecholaminergic polymorphic ventricular tachycardia (CPVT) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Atypical CPVT phenotype </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CALM1-related CPVT </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0017990 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> MODERATE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typically de novo </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 31983240; 23040497; 25557436; 32929985; 24563457; 26309258; 31170290 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 08.01.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 19.05.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> The disease mechanism in *CALM1*-related CPVT is likely a dominant negative effect caused by **altered gene product sequence** which reduces the calcium dependent inactivation of the calcium channel CaV1.2, resulting in increased inward calcium channel current (ICaL) and repolarization delay. 

The three *CALM* genes encode an identical 149aa protein, calmodulin. The *CALM1* gene is located on chromosome 14, *CALM2* on chromosome 2 and *CALM3* on chromosome 19. Calmodulin protein is involved in many calcium-dependent intracellular processes.  All three *CALM* genes have been classified as Definitive for LQTS and Moderate for CPVT, certain variants are only associated with one or other phenotype, whereas others are associated with a mixed or variable phenotypes.  

 All variants with evidence of pathogenicity identified in the CALM genes are missense: at least 35 distinct missense variants have been identified and reported in the International Calmodulinopathy Registry including 11 (31%) in *CALM1*, 16 (46%) in *CALM2*, and 8 (23%) in *CALM3*. The majority of variants across the CALM genes affect amino acid residues in the EF-hand Ca2+ binding loop III and IV. Some variants have additional evidence in that the same substitution has been seen in paralogous CALM genes, or that the same substitution has been seen in a related phenotype (LQT). Most variants are unique, but 9 were present in more than one index case and among these, three (p.Asn98Ser, p.Asp130Gly, and p.Phe142Leu, identified in 10, 5, and 4 families, respectively.) appear to be recurrent. While p.Asp130Gly and p.Phe142Leu have always been reported as associated with the LQTS phenotype, the p.Asn98Ser has phenotypic variability, including LQTS, CPVT, idiopathic VF and sudden death. The majority of CALM variants are ***de novo***. 

 Pathogenic variants in the CALM genes have been associated with: presentation in infancy or early childhood (up to 5 years); marked sinus bradycardia or atrioventricular block and QT prolongation; and, predominantly with *CALM1* variants, a mild-to-severe neurological impairment, including seizures, development delay, motor and/or cognitive disability. 

 The phenotype most frequently shown by patients with *CALM* variants is LQTS, but some patients display other phenotypes including CPVT, idiopathic VF and sudden unexplained death. Some variants have been associated with both LQTS and CPVT, however Crotti et al (PMID: 31170290) report that "despite the relatively small numbers of cases, a significant association (P = 0.001) was observed between location of mutation and phenotype (Supplementary material online, Figure S1). Indeed, a pathogenic variant in EF-hand IV Ca2+ binding loop was found in the majority (17/32, 53%) of CALM-LQTS index cases but in only one of the nine CALM-CPVTs (11%). Conversely, variants identified in CALM-CPVT index cases were mostly located either in EF-hand III (n = 5, 56%), or in the inter-EF hand I-II linker (n = 3, 33%)."  

 For the ClinGen CPVT curation, *CALM1* scored in the moderate range based on two reported variants in patients with CPVT or mixed CPVT/LQT phenotypes (p.Asn54Ile and p.Asn98Ser) and associated functional data. However, due to the unique characteristics of the calmodulin genes and their gene-disease associations, it was agreed by the expert panel to include them in this curation. The reasons were: 1) all three genes have already been established as disease-causing for an inherited arrhythmia syndrome (LQTS), 2) there are multiple examples of patients presenting with a phenotype indistinguishable from CPVT for each, either **de novo** or supported by functional evidence and 3) the genes encode for identical proteins, are all expressed in the heart and equivalent **de novo** variants in the three genes have been shown to lead to similar phenotypes, highlighting the functional equivalence of these genes. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CALM2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 114182 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Catecholaminergic polymorphic ventricular tachycardia (CPVT) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Atypical CPVT phenotype </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CALM2-related CPVT </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0017990 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> MODERATE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typically de novo </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 31983240; 27100291; 24917665; 25557436; 26309258; 31170290 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 08.01.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 19.05.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> The disease mechanism in *CALM2*-related CPVT is likely a dominant negative effect caused by **altered gene product sequence** which reduces the calcium dependent inactivation of the calcium channel CaV1.2, resulting in increased inward calcium channel current (ICaL) and repolarization delay.

 The three *CALM* genes encode an identical 149aa protein, calmodulin. The *CALM1* gene is located on chromosome 14, *CALM2* on chromosome 2 and *CALM3* on chromosome 19. Calmodulin protein is involved in many calcium-dependent intracellular processes.  All three *CALM* genes have been classified as Definitive for LQTS and Moderate for CPVT, certain variants are only associated with one or other phenotype, whereas others are associated with a mixed or variable phenotypes.

 All variants with evidence of pathogenicity identified in the *CALM* genes are missense: at least 35 distinct missense variants have been identified and reported in the International Calmodulinopathy Registry including 11 (31%) in *CALM1*, 16 (46%) in *CALM2*, and 8 (23%) in *CALM3*. The majority of variants across the *CALM* genes affect amino acid residues in the EF-hand Ca2+ binding loop III and IV. Some variants have additional evidence in that the same substitution has been seen in paralogous *CALM* genes, or that the same substitution has been seen in a related phenotype (LQTS). Most variants are unique, but 9 were present in more than one index case and among these, three (p.Asn98Ser, p.Asp130Gly, and p.Phe142Leu, identified in 10, 5, and 4 families, respectively.) appear to be recurrent. While variants p.Asp130Gly and p.Phe142Leu have always been reported as associated with the LQTS phenotype, the p.Asn98Ser has phenotypic variability, including LQTS, CPVT, idiopathic VF and sudden death. The majority of CALM variants are ***de novo***.

 Pathogenic variants in the *CALM* genes have been associated with: presentation in infancy or early childhood (up to 5 years); marked sinus bradycardia or atrioventricular block and QT prolongation; and, predominantly with *CALM1* variants, a mild-to-severe neurological impairment, including seizures, development delay, motor and/or cognitive disability.

 The phenotype most frequently shown by patients with *CALM* variants is LQTS, but some patients display other phenotypes including CPVT, idiopathic VF and sudden unexplained death. Some variants have been associated with both LQTS and CPVT, however Crotti et al (PMID: 31170290) report that "despite the relatively small numbers of cases, a significant association (P = 0.001) was observed between location of mutation and phenotype (Supplementary material online, Figure S1). Indeed, a pathogenic variant in EF-hand IV Ca2+ binding loop was found in the majority (17/32, 53%) of CALM-LQTS index cases but in only one of the nine CALM-CPVTs (11%). Conversely, variants identified in CALM-CPVT index cases were mostly located either in EF-hand III (n = 5, 56%), or in the inter-EF hand I-II linker (n = 3, 33%)."

For the ClinGen CPVT curation,*CALM2* scored in the moderate range based on four reported variants in patients with CPVT or mixed CPVT/LQT phenotypes (p.Glu46Lys in two **de novo** cases, p.Asn98Ser and p.Asp132Glu) and associated functional data. However, due to the unique characteristics of the calmodulin genes and their gene-disease associations, it was agreed by the expert panel to include them in this curation. The reasons were: 1) all three genes have already been established as disease-causing for an inherited arrhythmia syndrome (LQTS), 2) there are multiple examples of patients presenting with a phenotype indistinguishable from CPVT for each, either **de novo** or supported by functional evidence and 3) the genes encode for identical proteins, are all expressed in the heart and equivalent **de novo** variants in the three genes have been shown to lead to similar phenotypes, highlighting the functional equivalence of these genes. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CALM3 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 114183 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Catecholaminergic polymorphic ventricular tachycardia (CPVT) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Atypical CPVT phenotype </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CALM3-related CPVT </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0017990 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> MODERATE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typically de novo </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 31983240; 27516456; 31170290 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 08.01.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 19.05.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> The disease mechanism in *CALM3*-related CPVT is likely a dominant negative effect caused by **altered gene product sequence** which reduces the calcium dependent inactivation of the calcium channel CaV1.2, resulting in increased inward calcium channel current (ICaL) and repolarization delay.

 The three *CALM* genes encode an identical 149aa protein, calmodulin. The *CALM1* gene is located on chromosome 14, *CALM2* on chromosome 2 and *CALM3* on chromosome 19. Calmodulin protein is involved in many calcium-dependent intracellular processes.  All three *CALM* genes have been classified as Definitive for LQTS and Moderate for CPVT, certain variants are only associated with one or other phenotype, whereas others are associated with a mixed or variable phenotypes.

 All variants with evidence of pathogenicity identified in the *CALM* genes are missense: at least 35 distinct missense variants have been identified and reported in the International Calmodulinopathy Registry including 11 (31%) in *CALM1*, 16 (46%) in *CALM2*, and 8 (23%) in *CALM3*. The majority of variants across the *CALM* genes affect amino acid residues in the EF-hand Ca2+ binding loop III and IV. Some variants have additional evidence in that the same substitution has been seen in paralogous *CALM* genes, or that the same substitution has been seen in a related phenotype (LQTS). Most variants are unique, but 9 were present in more than one index case and among these, three (p.Asn98Ser, p.Asp130Gly, and p.Phe142Leu, identified in 10, 5, and 4 families, respectively.) appear to be recurrent. While variants p.Asp130Gly and p.Phe142Leu have always been reported as associated with the LQTS phenotype, the p.Asn98Ser has phenotypic variability, including LQTS, CPVT, idiopathic VF and sudden death. The majority of CALM variants are ***de novo***.

 Pathogenic variants in the *CALM* genes have been associated with: presentation in infancy or early childhood (up to 5 years); marked sinus bradycardia or atrioventricular block and QT prolongation; and, predominantly with *CALM1* variants, a mild-to-severe neurological impairment, including seizures, development delay, motor and/or cognitive disability.

 The phenotype most frequently shown by patients with *CALM* variants is LQTS, but some patients display other phenotypes including CPVT, idiopathic VF and sudden unexplained death. Some variants have been associated with both LQTS and CPVT, however Crotti et al (PMID: 31170290) report that "despite the relatively small numbers of cases, a significant association (P = 0.001) was observed between location of mutation and phenotype (Supplementary material online, Figure S1). Indeed, a pathogenic variant in EF-hand IV Ca2+ binding loop was found in the majority (17/32, 53%) of CALM-LQTS index cases but in only one of the nine CALM-CPVTs (11%). Conversely, variants identified in CALM-CPVT index cases were mostly located either in EF-hand III (n = 5, 56%), or in the inter-EF hand I-II linker (n = 3, 33%)."

For the ClinGen CPVT curation, *CALM3* scored in the moderate range based on four reported variants in patients with CPVT or mixed CPVT/LQT phenotypes (p.Glu46Lys in two **de novo** cases, p.Asn98Ser and p.Asp132Glu) and associated functional data. However, due to the unique characteristics of the calmodulin genes and their gene-disease associations, it was agreed by the expert panel to include them in this curation. The reasons were: 1) all three genes have already been established as disease-causing for an inherited arrhythmia syndrome (LQTS), 2) there are multiple examples of patients presenting with a phenotype indistinguishable from CPVT for each, either **de novo** or supported by functional evidence and 3) the genes encode for identical proteins, are all expressed in the heart and equivalent **de novo** variants in the three genes have been shown to lead to similar phenotypes, highlighting the functional equivalence of these genes. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TECRL </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 614021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Catecholaminergic polymorphic ventricular tachycardia (CPVT) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Atypical CPVT phenotype </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TECRL-related CPVT </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0017990 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Absent gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice donor variant NMD triggering; stop gained variant NMD triggering; missense; exon deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> Exon 2 deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 27861123; 30790670; 33367594; 32173957 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 08.01.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 19.05.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> **Absent gene product level and altered gene product sequence** of *TECRL* is associated with CPVT. The disease mechanism appears to be loss of function.
 *TECRL* is rarely associated with CPVT, but several reports have described biallelic loss of function variants in cases.  These include both homozygous and compound heterozygous inheritance, with a variety of variant types described – splice donor variant, stop gained, exon deletion (exon 2), missense variants (including homozygous p.Arg196Gln detected in 2 patients with exome sequencing), and a large duplication encompassing all of the *TECRL* gene with an uncertain consequence.

 These cases presented with phenotypic features typical of CPVT, including exercise and emotion induced syncope and cardiac arrest and ventricular arrhythmias during exercise testing. A mild prolonged QT interval was observed in several cases, especially after stimulation by epinephrine or exercise, although overall the phenotypes are much more typical of CPVT than LQTS. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TRDN </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 603283 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Catecholaminergic polymorphic ventricular tachycardia (CPVT) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Atypical CPVT phenotype </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TRDN-related CPVT </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0017990 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Absent gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Frameshift variant NMD triggering; stop gained variant NMD triggering; missense; exon deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> Exon 2 deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 22422768; 26200674; 26768964; 30479949; 25922419 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 08.01.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 19.05.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> **Absent gene product level and altered gene product sequence** of *TRDN* is associated with CPVT. The disease mechanism appears to be loss of function.
 *TRDN* is rarely associated with CPVT, but several reports have described biallelic loss of function variants in cases.  These include both homozygous and compound heterozygous variants, with a variety of variant types described - stop gained, frameshift, exon deletion (exon 2), intronic variants (with proven effect on splicing) and missense variants (proven to lead to degraded protein).
 Biallelic loss of function variants in *TRDN* have also been associated with LQTS with an atypical presentation (ClinGen: strong evidence). This, "triadin knockout syndrome" can lead to variable phenotypes. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><h3>Long QT Syndrome (LQTS)</h3><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> KCNQ1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 607542 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Long QT Syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> KCNQ1-related LQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0008646 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant; autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal; biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice acceptor; splice donor; frameshift; frameshift variant NMD triggering; stop gained; stop gained variant NMD triggering; missense; inframe insertion; inframe deletion; structural variants (whole exon deletions); structural variants (intragenic multi exon tandem duplication) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 31983240; 23900354; 23392653; 23591039; 27041150; 28944242; NBK1129; 15840476; 19716085; 9927399; 22995932; 29650123; 21185501; 27807201; 17470695; 25854863; 29532034; 25174857; 32893267; 22456477; 21787999; 27262388; 19841300; 29504689 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 01.11.2020 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 16.06.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _KCNQ1_ cause LQTS due to **decreased gene product level or altered gene product sequence**. This leads to loss of function due to a variety of mechanisms including defects in ion permeation, channel gating and trafficking.
 _KCNQ1_ variants account for approx. 30%-45% of LQTS cases (PMID: 22456477; 21787999; 27262388; 29504689).

 _KCNQ1_-related LQTS is typically inherited as an autosomal dominant trait characterised by incomplete penetrance. Rarely biallelic _KCNQ1_ variants can cause autosomal recessive LQTS or Jervell and Lange-Nielsen syndrome (JLNS) (severe cardiac phenotype and sensorineural hearing loss). JLNS results when there is complete absence of IKs (slowly activating delayed rectifier potassium channel/current) (PMID: 23591039).

 It has been noted that approximately 10% of genotype positive LQT patients have more than 1 pathogenic variant in &gt;=1 LQTS-related gene (PMID: 15840476; 19716085). Biallelic disease appears to be more penetrant with a more severe phenotype.

 Missense variants are responsible for the majority of _KCNQ1_-related LQTS cases although truncating variants including nonsense, frameshift, splice site and structural variants (multi exon deletions and a multi exon duplication (inframe tandem repeat of exons 3-6, leading to duplication of the second to fifth transmembrane domains of the channel)) are also reported (PMID: 19716085; 18774102; 25174857). 

 ClinGen have concluded there is sufficient evidence for haploinsufficiency. Variants conferring a dominant negative effect have also been described. Moss et al (PMID: 17470695) reported that dominant-negative variants are associated with a more severe phenotype than variants conferring haploinsufficiency.

 Phenotypes can differ according to regionality with some regions associated with more malignant phenotypes e.g. C-loop variants are associated with higher risk of sudden cardiac death (PMID: 22456477). In addition, there are gene regions where there is a high confidence for pathogenicity e.g. transmembrane regions and C-terminus domains (PMID: 32893267; 19841300).
  </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> KCNH2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 152427 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Long QT Syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> KCNH2-related LQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0013367 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice acceptor; splice donor; frameshift variant NMD triggering; stop gained variant NMD triggering; missense; inframe insertion; inframe deletion; structural variants (whole exon deletions &amp; duplications) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 31983240; 26320108; 27761161; NBK1129; 15840476; 18774102; 19926013; 32893267; 21185499; 10491368; 10841244; 12621127; 19716085; 21185501; 9927399; 22995932; 29650123; 25417810; 24932360 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 14.10.2020 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 16.06.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _KCNH2_-related LQTS is due to **decreased gene product level or altered gene product sequence**. The disease mechanism is **loss of function** due to a variety of mechanisms including disruption of synthesis of channel subunits, reduction in intracellular transport or trafficking, defects in ion permeation or channel gating.

 Both haploinsufficiency and a dominant negative effect are proposed mechanisms causing loss of function of _KCNH2_. About 60% of LQT2 mutations are missense variants, the remaining 40% are nonsense, frameshift, insertions, deletions, duplications (intragenic tandem duplication), or involve a splice site. 

 _KCNH2_ variants account for approximately 30% of LQTS cases (PMID: 24932360).
 Walsh et al showed that non truncating variants in _KCNH2_ are enriched in ion channel transmembrane regions and specific N-terminus and C-terminus domains and have &gt;95% probability of pathogenicity (PMID: 32893267).
 Shimizu et al report correlations between specific variant types and location and clinical phenotype (PMID: 19926013). Patients with missense variants in the transmembrane pore region have significantly higher cardiac event rates than those with missense variants in either N-terminus, transmembrane non-pore, or C-terminus regions. 

 It has been noted that approximately 10% of genotype positive LQT patients have more than 1 pathogenic variant in &gt;=1 gene. Biallelic pathogenic variants or digenic pathogenic variants appear to be generally associated with a more severe phenotype with longer QTc interval and a higher incidence of cardiac events. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> SCN5A </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 600163 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Long QT Syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> SCN5A-related LQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011377 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe insertion; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 7889574; NBK1129; 29806494; 29798782; 31983240; 11463728; 26320108; 15840476; 19716085; 21185501; 9927399; 22995932; 29650123; 27566755; 32893267; 24932360 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 14.10.2020 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 16.06.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> **Altered gene product sequence** of _SCN5A_ causes long QT syndrome. The likely disease mechanism is gain of function.
 Over 200 pathogenic missense variants and inframe deletions or insertions have been reported. The sodium current mediated by Nav1.5 consists of peak and late components (INa-P and INa-L). It is thought that gain-of-function _SCN5A_ pathogenic variants lead to enhanced INa-P and INa-L, which can trigger life-threating arrhythmias. 
 _SCN5A_-related LQTS accounts for 5-10% of LQTS cases (PMID: 27566755).
 Rare missense variants are estimated to occur in around 2% of healthy White and 5% of healthy non-White subjects so collectively missense variants are not rare in the healthy population.

 _SCN5A_-related LQTS can present with specific features: patients may have marked resting bradycardia, QT interval prolongation more pronounced during slow heart rate (which might explain why arrhythmic events occur more frequently at rest), a first cardiac event that is lethal, and onset after puberty (PMID: 29798782).

 It has been noted that approximately 10% of genotype positive LQT patients have more than 1 mutation in &gt;=1 gene. Biallelic pathogenic variants or digenic pathogenic variants appear to be generally associated with a more severe phenotype with longer QTc interval and a higher incidence of cardiac events.

 Note: loss of function variants in _SCN5A_ are associated with Brugada syndrome and individual variants can have hybrid loss of function and gain of function effects causing a mixed phenotype. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CALM1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 114180 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Long QT syndrome with an atypical presentation </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CALM1-related LQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0014548 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typically de novo </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 23388215; 26969752; 31983240; 31170290 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 14.10.2020 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 16.06.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _CALM1_-related LQTS is caused by an **altered gene product sequence**. Reduction in the calcium dependent inactivation of the calcium channel CaV1.2 results in increased inward calcium channel current (ICaL) and repolarization delay. The disease mechanism is likely a dominant negative effect.

 The three _CALM_ genes encode an identical 149aa protein, calmodulin. The _CALM1_ gene is located on chromosome 14, _CALM2_ on chromosome 2 and _CALM3_ on chromosome 19. Calmodulin protein is involved in many calcium-dependent intracellular processes.  All three _CALM_ genes have been classified as Definitive for LQTS and Moderate for CPVT, certain variants are only associated with one or other phenotype, whereas others are associated with a mixed or variable phenotypes. 

 All variants with evidence of pathogenicity identified in the _CALM_ genes are missense: at least 35 distinct missense variants have been identified and reported in the International Calmodulinopathy Registry including 11 (31%) in _CALM1_, 16 (46%) in _CALM2_, and 8 (23%) in _CALM3_. The majority of variants across the _CALM_ genes affect amino acid residues in the EF-hand Ca2+ binding loop III and IV. Some variants have additional evidence in that the same substitution has been seen in paralogous _CALM_ genes, or that the same substitution has been seen in a related phenotype (LQTS). Most variants are unique, but 9 were present in more than one index case and among these, three (p.Asn98Ser, p.Asp130Gly, and p.Phe142Leu, identified in 10, 5, and 4 families, respectively.) appear to be recurrent. While variants p.Asp130Gly and p.Phe142Leu have always been reported as associated with the LQTS phenotype, the variant p.Asn98Ser has phenotypic variability, including LQTS, CPVT, idiopathic VF and sudden death. The majority of _CALM_ variants are __de novo__.

 Pathogenic variants in the _CALM_ genes have been associated with: presentation in infancy or early childhood (up to 5 years); marked sinus bradycardia or atrioventricular block and QT prolongation; and, predominantly with _CALM1_ variants, a mild-to-severe neurological impairment, including seizures, development delay, motor and/or cognitive disability.

 The phenotype most frequently shown by patients with _CALM_ variants is LQTS, but some patients display other phenotypes including CPVT, idiopathic VF and sudden unexplained death. Some variants have been associated with both LQTS and CPVT, however Crotti et al (PMID: 31170290) report that "despite the relatively small numbers of cases, a significant association (P = 0.001) was observed between location of mutation and phenotype (Supplementary material online, Figure S1). Indeed, a pathogenic variant in EF-hand IV Ca2+ binding loop was found in the majority (17/32, 53%) of CALM-LQTS index cases but in only one of the nine CALM-CPVTs (11%). Conversely, variants identified in CALM-CPVT index cases were mostly located either in EF-hand III (n = 5, 56%), or in the inter-EF hand I-II linker (n = 3, 33%)." </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CALM2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 114182 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Long QT syndrome with an atypical presentation </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CALM2-related LQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0014550 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typically de novo </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 23388215; 26969752; 31983240; 31170290; 27765793 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 14.10.2020 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 16.06.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _CALM2_-related LQTS is caused by an**altered gene product sequence** which reduces the calcium dependent inactivation of the calcium channel CaV1.2, resulting in increased inward calcium channel current (ICaL) and repolarization delay. The disease mechanism is likely a dominant negative effect.

 The three _CALM_ genes encode an identical 149aa protein, calmodulin. The _CALM1_ gene is located on chromosome 14, _CALM2_ on chromosome 2 and _CALM3_ on chromosome 19. Calmodulin protein is involved in many calcium-dependent intracellular processes.  All three _CALM_ genes have been classified as Definitive for LQTS and Moderate for CPVT, certain variants are only associated with one or other phenotype, whereas others are associated with a mixed or variable phenotypes. 

 All variants with evidence of pathogenicity identified in the _CALM_ genes are missense: at least 35 distinct missense variants have been identified and reported in the International Calmodulinopathy Registry including 11 (31%) in _CALM1_, 16 (46%) in _CALM2_, and 8 (23%) in _CALM3_. The majority of variants across the _CALM_ genes affect amino acid residues in the EF-hand Ca2+ binding loop III and IV. Some variants have additional evidence in that the same substitution has been seen in paralogous _CALM_ genes, or that the same substitution has been seen in a related phenotype (LQTS). Most variants are unique, but 9 were present in more than one index case and among these, three (p.Asn98Ser, p.Asp130Gly, and p.Phe142Leu, identified in 10, 5, and 4 families, respectively.) appear to be recurrent. While variants p.Asp130Gly and p.Phe142Leu have always been reported as associated with the LQTS phenotype, the variant p.Asn98Ser has phenotypic variability, including LQTS, CPVT, idiopathic VF and sudden death. The majority of _CALM_ variants are __de novo__.

 Pathogenic variants in the _CALM_ genes have been associated with: presentation in infancy or early childhood (up to 5 years); marked sinus bradycardia or atrioventricular block and QT prolongation; and, predominantly with _CALM1_ variants, a mild-to-severe neurological impairment, including seizures, development delay, motor and/or cognitive disability.

 The phenotype most frequently shown by patients with _CALM_ variants is LQTS, but some patients display other phenotypes including CPVT, idiopathic VF and sudden unexplained death. Some variants have been associated with both LQTS and CPVT, however Crotti et al (PMID: 31170290) report that "despite the relatively small numbers of cases, a significant association (P = 0.001) was observed between location of mutation and phenotype (Supplementary material online, Figure S1). Indeed, a pathogenic variant in EF-hand IV Ca2+ binding loop was found in the majority (17/32, 53%) of CALM-LQTS index cases but in only one of the nine CALM-CPVTs (11%). Conversely, variants identified in CALM-CPVT index cases were mostly located either in EF-hand III (n = 5, 56%), or in the inter-EF hand I-II linker (n = 3, 33%)." </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CALM3 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 114183 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Long QT syndrome with an atypical presentation </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CALM3-related LQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0019171 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typically de novo </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 23388215; 26969752; 31983240; 31170290 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 14.10.2020 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 16.06.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> For LQTS, _CALM3_ variants lead to an **altered gene product sequence**. The disease mechanism is likely a dominant negative effect which reduces the calcium dependent inactivation of the calcium channel CaV1.2, resulting in increased inward calcium channel current (ICaL) and repolarization delay.

 The three _CALM_ genes encode an identical 149aa protein, calmodulin. The _CALM1_ gene is located on chromosome 14, _CALM2_ on chromosome 2 and _CALM3_ on chromosome 19. Calmodulin protein is involved in many calcium-dependent intracellular processes.  All three _CALM_ genes have been classified as Definitive for LQTS and Moderate for CPVT, certain variants are only associated with one or other phenotype, whereas others are associated with a mixed or variable phenotypes. 

 All variants with evidence of pathogenicity identified in the _CALM_ genes are missense: at least 35 distinct missense variants have been identified and reported in the International Calmodulinopathy Registry including 11 (31%) in _CALM1_, 16 (46%) in _CALM2_, and 8 (23%) in _CALM3_. The majority of variants across the _CALM_ genes affect amino acid residues in the EF-hand Ca2+ binding loop III and IV. Some variants have additional evidence in that the same substitution has been seen in paralogous _CALM_ genes, or that the same substitution has been seen in a related phenotype (LQTS). Most variants are unique, but 9 were present in more than one index case and among these, three (p.Asn98Ser, p.Asp130Gly, and p.Phe142Leu, identified in 10, 5, and 4 families, respectively.) appear to be recurrent. While variants p.Asp130Gly and p.Phe142Leu have always been reported as associated with the LQTS phenotype, the variant p.Asn98Ser has phenotypic variability, including LQTS, CPVT, idiopathic VF and sudden death. The majority of _CALM_ variants are __de novo__.

 Pathogenic variants in the _CALM_ genes have been associated with: presentation in infancy or early childhood (up to 5 years); marked sinus bradycardia or atrioventricular block and QT prolongation; and, predominantly with _CALM1_ variants, a mild-to-severe neurological impairment, including seizures, development delay, motor and/or cognitive disability.

 The phenotype most frequently shown by patients with _CALM_ variants is LQTS, but some patients display other phenotypes including CPVT, idiopathic VF and sudden unexplained death. Some variants have been associated with both LQTS and CPVT, however Crotti et al (PMID: 31170290) report that "despite the relatively small numbers of cases, a significant association (P = 0.001) was observed between location of mutation and phenotype (Supplementary material online, Figure S1). Indeed, a pathogenic variant in EF-hand IV Ca2+ binding loop was found in the majority (17/32, 53%) of CALM-LQTS index cases but in only one of the nine CALM-CPVTs (11%). Conversely, variants identified in CALM-CPVT index cases were mostly located either in EF-hand III (n = 5, 56%), or in the inter-EF hand I-II linker (n = 3, 33%)." </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TRDN </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 603283 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Long QT syndrome with an atypical presentation </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TRDN-related LQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0019171 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Absent gene product level </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Frameshift variant NMD triggering </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 31983240; 25922419; 26768964; 22422768; 26200674; 30479949; 30649896 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 04.11..2020 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 16.06.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _TRDN_-related LQTS is due to an **absent gene product level**. The disease mechanism is loss of function (LoF).
 **_TRDN_ is rarely associated with LQTS with an atypical presentation**. Following identification of a homozygous frameshift variant in _TRDN_, 4 other patients with homozygous or compound heterozygous frameshift variants were identified in a cohort of 33 unrelated genotype-negative LQTS patients.
 The atypical LQTS phenotype described: extensive T-wave inversions in precordial leads V1 through V4, persistent or transient QT prolongation and severe disease expression of exercise-induced cardiac arrest in early childhood (≤3 years of age).

 Biallelic loss of function variants in _TRDN_ have also been associated with catecholaminergic polymorphic ventricular tachycardia (CPVT). As patients can have overlapping phenotypes, there is a term “triadin knockout syndrome”. The Triadin Knock Out Syndrome (TKOS) registry had 21 patients in 2019. All were homozygous or compound heterozygous for _TRDN_ variants. The majority were frameshift or nonsense. There were a small number of missense or splice altering variants.
 _TRDN_ is expressed in both cardiac and skeletal muscle and undergoes extensive alternative splicing to produce several isoforms. Only biallelic LoF variants affecting cardiac isoforms should be considered as pathogenic for LQTS.
  </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> KCNQ1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 607542 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndrome with QT prolongation and cardiac arrhythmias </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> KCNQ1- related Jervell and Lange-Nielsen syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0024540 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Absent gene product; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice acceptor; splice donor; frameshift variant NMD triggering; stop gained variant NMD triggering; missense; structural variants (whole exon deletions; complex rearrangements) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 31983240; 9020846; 23392653; 23591039; 27041150; 27868350; 23392653; 11226272; 25187895 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 01.05.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 16.06.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Biallelic pathogenic variants in _KCNQ1_ cause JLNS due to **absent gene product level or altered gene product sequence**.
 Both homozygous and compound heterozygous pathogenic variants in _KCNQ1_ have been reported in JLNS. Nonsense, frameshift, splice site, whole exon deletions and missense variants have been reported as pathogenic.

 Bhuiyan and Wilde (PMID: 23591039) compared two groups of patients with homozygous variants in _KCNQ1_. Those patients where there was residual IKs (slowly activating delayed rectifier potassium channel/current) even as little as 10%, had QT prolongation but no hearing loss (autosomal recessive LQTS), whereas those patients with complete absence of IKs had both QT prolongation and hearing loss (JLNS). 
 They concluded that homozygous or compound heterozygous nonsense, frameshift or exon skipping variants resulting in 100% loss of IKs would result in JLNS. In addition, biallelic missense mutations that lead to a protein product that does not traffic to the cell membrane (or is subject to nonsense mediated decay) will lead to JLNS.
Note: heterozygous variants in _KCNQ1_ lead to dominant LQTS. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> KCNE1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 176261 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndromic QT prolongation and cardiac arrhythmias </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> KCNE1-related Jervell and Lange-Nielsen syndrome 2 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012871 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> MODERATE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe insertion; inframe deletion, stop gained NMD escaping </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30461122; 9354783; NBK1405; 31941373; 9445165; 9328483; 10973849; 19716085; 16461811 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 01.08.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 30.08.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _KCNE1_-related JLNS is due to **altered gene product sequence**. Inheritance is autosomal recessive. Homozygous or compound heterozygous _KCNE1_ variants cause JLNS through loss of function (PMID: 9354783). The majority of JLNS patients have biallelic KCNQ1 variants (JLNS1), only a minority have JLNS2 caused by biallelic _KCNE1_ variants (PMID: 16461811).
 The _KCNE1_ protein (previous names LQT5, mink, IsK) functions as a regulatory subunit of KCNQ1. It has 7 annotated exons but only one is protein coding (PMID: 30461122).

 A small number of variants have been reported, mainly missense and inframe indels. In more recent studies, 4 nonsense variants have also been reported (PMID: 30461122; 31941373). However, in one of these studies (PMID: 30461122), patients were ascertained for deafness and QT prolongation is either variable or not reported in some family members. The authors did not carry out functional studies to determine the effect of these variants on mRNA but suggest that given there is only one coding exon, these nonsense variants would be unlikely to result in nonsense mediated decay. 
The phenotype of _KCNE1_-related JLNS2 appears to be milder than JLNS1 (caused by biallelic KCNQ1 variants) (PMID: 16461811; 31941373)
 Heterozygous variants in _KCNE1_ have been associated with LQT syndrome without deafness. However, Roberts et al observed a low ECG penetrance in these individuals with the majority not manifesting clinically (PMID: 31941373).
 ClinGen classified the gene disease association, _KCNE1_ and JLNS2, as Moderate stating that although there was a large amount of experimental evidence and no contradictory evidence, more case level evidence was required to reach a Definitive classification. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> KCNJ2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 600681 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndromic QT prolongation and cardiac arrhythmias </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> KCNJ2-related Andersen Tawil syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0008222 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe insertion; inframe deletion; stop gained NMD escaping </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 31983240; 32947483; 32843460; NBK1264; 32299589; 21493816; 16217063; 32499698; 11371347; 12163457 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 01.05.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 16.06.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _KCNJ2_ pathogenic variants cause Andersen Tawil syndrome (ATS) through **altered gene product sequence**. Most of the pathogenic variants cause Kir2.1 (inwardly-rectifying potassium channel) loss-of-function, either via trafficking or gating defects.
 Kir2.1, encoded by _KCNJ2_, contributes a major component of the cardiac action potential repolarization phase. Pathogenic variants of _KCNJ2_ gene account for 60-70% of clinical ATS cases, termed type-1 ATS.

 The cardiac manifestation includes QT-U abnormalities but not typical QT prolongation. Ventricular arrhythmias also differ from typical LQTS with frequent premature ventricular complexes and polymorphic non-sustained ventricular tachycardia but only rarely torsades de pointes. Extracardiac manifestations include dysmorphic features and periodic paralysis with hypo- and hyperkalemic episodes in some patients.

 The majority of pathogenic variants in _KCNJ2_ are missense changes. A small number of inframe deletions, insertions and stop gained variants predicted to escape nonsense mediated decay (NMD) have also been reported in this single-exon gene.
 The p.Arg218Trp pathogenic variant is considered a potential mutational hot spot (Davies et al 2005 PMID: 16217063).
Penetrance in families appears high, with estimates of ≥80% </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CACNA1C </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 114205 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Long QT Syndrome (LQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndromic QT prolongation and cardiac arrhythmias </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CACNA1C-related Timothy syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0010979 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typically de novo </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 28211989; 25633834; NBK1403; 15863612; 15454078; 22106044; 26253506; 24728418; 31983240 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 11.11.2020 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 16.06.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _CACNA1C_-related Timothy syndrome is caused by variants leading to **altered gene product sequence**. A recurrent, ___de novo___, missense variant in _CACNA1C_ was described in 13 Timothy syndrome patients, p.Gly406Arg in exon 8A (PMID 15454078; 15863612). The disease mechanism appears to be gain-of-function through failed channel inactivation.

 Classic Timothy syndrome (TS1) is a very rare multisystem disorder characterized by marked QT prolongation, syndactyly, immune deficiency, seizures, congenital heart defects, cognitive abnormalities, learning difficulties, and intermittent hypoglycaemia (PMID: 28211989). 

 _CACNA1C_ has a complex genomic structure that undergoes extensive alternative splicing. Splawski et al identified 2 patients with **___de novo___ missense variants in exon 8** of an alternate splice form (p.Gly406Arg, analogous to the exon 8a variant, and p.Gly402Ser). This splice form represents 80% of all cardiac mRNAs. The patients were described as having atypical Timothy syndrome (TS2), presenting with a more severe cardiac phenotype and without syndactyly (PMID: 15863612; 25633834).
  Other missense variants in _CACNA1C_ have been reported in association with isolated LQTS (PMID: 26253506; 25633834; 24728418). However as of 2020, the ClinGen Cardiovascular Domain Working Group have classified the strength of evidence supporting an association between _CACNA1C_ and LQTS as moderate (PMID: 31983240). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><h3>Short QT Syndrome (SQTS)</h3><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> KCNH2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 609620 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Short QT Syndrome (SQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Classic SQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> KCNH2-related SQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012312 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 14676148; 19340359; 25335996; 21130771; 29876509; 29016797; 28491588; 30571592; 15828882; 18692916; 25974115; 21310316; 31072576; 7736582; 7889573; 29574456; 30582453; 9547387; 15673388; 29759541; 31049424; 30175559; 19088443; 30496390; 30947366 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> SQTS is a rare (&lt; 1/10,000) AD inherited arrhythmia syndrome associated with atrial fibrillation, ventricular arrhythmia, and risk of sudden cardiac arrest. Diagnosis is based on a diagnostic scorecard (PMID: 21310316), similar to LQTS. Approximately 20% of diagnosed cases will have a genetic cause, and _KCNH2_ is the most common identified gene.
It is noteworthy that of the 18 probands with SQTS in whom _KCNH2_ variants have been identified, 13 had one of 2 variants; 7 with p.Thr618Ile variant (ClinVar Variation ID# 67297) and 6 with p.Asn588Lys (ClinVar Variation ID# 14436; NM_172056.2). There is high penetrance for these recurrent variants.
 Experimental evidence derived from non-patient cells, human-induced pluripotent stem cell-derived cells and a rabbit animal model (PMID: 30496390) all support this gene’s relationship with SQTS, with **a gain-of-function mechanism**. These experimental studies demonstrate that genetic variants identified in SQTS patients lead to potassium current perturbations concordant with SQTS phenotype and shortening of the QT interval.

 Note loss-of-function variants in _KCNH2_ are associated with LQTS. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> KCNQ1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 609621 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Short QT Syndrome (SQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Classic SQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> KCNQ1-related SQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012313 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> STRONG </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> Almost all evidence derived from: p.Val141Met; p.Val307Leu </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 16109388; 28491547; 15159330; 24380499; 25974115; 26168993; 26346102; 26279191; 28491751; 8900283; 8528244; 29213224 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Almost all evidence for _KCNQ1_ as a cause of SQTS is derived from a single variant (p.Val141Met) identified in 9 unrelated probands. All 9 cases presented with severe bradycardia in-utero or at birth and in 6 atrial fibrillation was also reported. In one case complete atrioventricular block was documented. (PMIDs: 24818999; 26279191; 16109388; 24380499; 25974115; 28491547).

 Five other variants have been reported for SQTS, but the phenotypic features of SQTS for 4 of these have been unclear or functional data supporting the mechanism of disease lacking in the published manuscripts. For the fifth additionally reported variant, 70-year-old patient presented with ‘idiopathic VF” and a short QT interval, a _KCNQ1_ variant, p.Val307Leu, was reported as the likely genetic culprit, with in vitro studies reporting a gain-of-function effect of the mutant (Bellocq et al, Circ 2004).

 Functional characterisation in cell models confirms a gain of function mechanism for Val141Met and Val307Leu.
 Four cases of Val141Met are reported as ___de novo___ without noted confirmed paternity. Importantly, in none of the p.Val141Met cases was cardiac arrest or sudden cardiac death described. 
Note: loss-of-function variants in _KCNQ1_ are associated with LQTS. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> SLC4A3 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 106195 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Short QT Syndrome (SQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Classic SQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> SLC4A3-related SQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0000453 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> MODERATE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> p.Arg370His </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 29167417 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _SLC4A3 _ encodes a plasma membrane anion exchange protein. Genetic evidence supporting _SLC4A3 _ as a SQTS-causing gene is derived from a single publication in which exome sequencing was performed in 2 families, including one large pedigree (PMID: 29167417). The same rare genetic variant (p.Arg370His, c.1109G&gt;A) was identified in both families, suggesting they are possibly distantly related. Experimental evidence from in vitro and zebrafish models suggests reduced membrane localization of the mutated protein leads to intracellular alkalinization and shortening of the cardiomyocyte action potential duration.  The genetic evidence, including the unbiased gene discovery approach of whole exome sequencing and segregation of the identified genetic variant with a large number of affected individuals within the presented pedigree, was considered relatively strong by the ClinGen GCEP.  However, lack of other publications supporting this gene-disease relationship led to a score in the "Moderate" range (for gene-disease validity). Further information on ClinGen gene-disease validity can be found here: https://search.clinicalgenome.org/kb/genes/HGNC:11029 </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> KCNJ2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 609622 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Short QT Syndrome (SQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Classic SQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> KCNJ2-related SQTS </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012314 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> MODERATE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 15761194; 23440193; 2479485; 29615871; 22155372; 11410627; 19285083; 24794859; 19710529 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Genetic variants in _KCNJ2_ have been identified in 6 patients from 5 families with unique variants, including at least 2 probands with a _de-novo_ variant (paternity not confirmed). Experimental evidence demonstrated these variants lead to gain-of-function of the late repolarizing, _KCNJ2_-encoded Ik1 current in the heart, and abbreviation of the action potential duration. The ClinGen GCEP reviewing gene-disease validity considered these data sufficient for classifying the gene-disease relationship of _KCNJ2_ as “Moderate” but, in the absence of segregation or case-control data, the genetic evidence was not sufficient for a stronger classification. Further information on ClinGen gene-disease validity can be found here: https://search.clinicalgenome.org/kb/genes/HGNC:6263 </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> SLC22A5 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 212140 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Short QT Syndrome (SQTS) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndrome including shortened QT and cardiac arrhythmias </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> SLC22A5-related primary systemic carnitine deficiency </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0008919 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 26190315; 31472821; 3009296 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 21.07.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Variants in _SLC22A5_ cause autosomal recessive primary systemic carnitine deficiency (PSCD), a syndrome characterized by hypoketotic hypoglycemia, hyperammonemia, liver dysfunction, hypotonia and cardiomyopathy (PMID: 26190315). 
 Homozygote or compound heterozygote variants have been identified in unexplained sudden cardiac death or resuscitated cardiac arrest cases without overt extra-cardiac manifestations (PMIDs: 31472821; 3009296). Furthermore, a short QT interval has been demonstrated in a carnitine deficient mouse model (PMID: 31472821) as well as in patients with PSCD (PMID: 31472821; 3009296). Importantly, however, the QT interval in these patients returns to normal with carnitine supplementation treatment. 
  Information on ClinGen gene-disease validity conclusion: the ClinGen GCEP  conclude that PSCD is a metabolic and reversible SQTS-mimic.
 While there is no robust evidence of a relationship between _SLC22A5_ and true SQTS [ClinGen “disputed” classification], there is "definitive" evidence for a relationship between _SLC22A5_ and PSCD which may justify testing in individuals being investigated for possible SQTS. Further information on ClinGen gene-disease validity can be found here: https://search.clinicalgenome.org/kb/genes/HGNC:10969 </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><h2>CARDIOMYOPATHY</h2><h3>Arrhythmogenic right ventricular cardiomyopathy (ARVC)</h3><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> DSC2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 125645 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Arrhythmogenic right ventricular cardiomyopathy (ARVC) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial isolated arrhythmogenic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> DSC2-related ARVC </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012506 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant; Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal; Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice region; splice acceptor; splice donor; frameshift; stop gained; missense; inframe insertion; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 31028357; 23911551; NBK1131; 21636032; 33831308; 26310507; 23863954; 24793512; 24070718; 34400560; 17033975; 17963498; 17186466; 20031616; 19863551; 31402444 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 25.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 05.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _DSC2_-related ARVC is due to **decreased gene product level or altered gene product sequence** due to a variety of mechanisms (e.g. null alleles, trafficking defects, impaired proteolytic processing, absence of or impaired protein-protein interactions) (PMID: 31028357; 23911551; NBK1131).
 Loss of function is the likely disease mechanism. ClinGen found there was some evidence to support haploinsufficiency as a mechanism. https://search.clinicalgenome.org/kb/gene-dosage/HGNC:3036. 
 Autosomal dominant inheritance with incomplete penetrance is the most common mode of transmission (PMID: 21636032; 33831308). Homozygous and compound heterozygous variants have also been described in association with ARVC with or without cutaneous features (PMID: 26310507; 23863954; 24793512; 24070718; 34400560). In some cases, these appear to reflect autosomal recessive inheritance (PMID 24793512; 23863954, 33831308). Instances of digenic inheritance have been identified with _DSC2_ variants along with other desmosomal gene pathogenic variants (PMID: 24070718).
 A number of _DSC2_ variants have been reported in the literature including nonsense, frameshift, splice, missense and inframe insertions and deletions (NBK1131; 17033975; 17963498; 17186466; 20031616; 19863551; 31402444).
 _DSC2_-related ARVC appears to be characterised by an increased risk of biventricular involvement and heart failure when compared to PKP2-related ARVC (PMID: 34400560). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> DSG2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 125671 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Arrhythmogenic right ventricular cardiomyopathy (ARVC) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial isolated arrhythmogenic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> DSG2-related ARVC </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012434 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant; Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal; Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice acceptor; splice donor; frameshift; stop gained; missense; inframe insertion; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 21636032; 33831308; 33917638; 34400560; 24070718; 30454721; 25616645; 30790397; 34400560; 16505173; NBK1131; 27532257; 16823493; 27170944 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 12.12.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 05.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _DSG2_-related ARVC is due to **decreased gene product level or altered gene product sequence** due to a variety of mechanisms.  Much of the underlying pathogenesis of _DSG2_ pathogenic variants is still unknown; it is believed that loss of _DSG2_ compromises cell-to-cell adhesion between cardiomyocytes (PMID: 26085008; NBK1131). There is also work revealing that desmosomal variants can reduce canonical Wnt signaling and activating Wnt with a GSK3B inhibitor can block disease pathogenesis (PMID: 16823493; PMID: 27170944).
 The usual mode of inheritance is **autosomal dominant characterized by incomplete penetrance** (PMID: 21636032; 33831308). Compound heterozygous and homozygous variants have been described. In some families, heterozygous carriers of these variants were not affected suggesting **autosomal recessive inheritance** (PMID: 33917638; 34400560; 24070718; 33831308; 30454721). Patients with &gt;1 variant appear to have a more severe phenotype (PMID: 25616645; PMID: 30790397).
 
 The majority of _DSG2_ variants are rare missense variants with unknown significance/unknown mechanism of pathogenicity. In addition, nonsense, frameshift, insertions, deletions, and splice site variants have all been described (PMID: 16505173; NBK1131; 30790397; 27532257; 33917638). 
 _DSG2_-related ARVC appears to be characterised by an increased risk of biventricular involvement and heart failure when compared to PKP2-related ARVC (PMID: 34400560; 30790397). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> DSP </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 125647 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Arrhythmogenic right ventricular cardiomyopathy (ARVC) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial isolated arrhythmogenic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> DSP-related ARVC </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011831 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant; Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal; Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice donor; splice acceptor; frameshift; stop gained; missense; inframe insertion; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 32372669; 23137101; 21636032; 33831308; 31319917; 20716751; 23810894; 24503780; 27532257; 31514951; 27761164; 21636032; 32808748; 33275305; 11063735; 27761164; 20940358; 22795705; 26604139; 30382575 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 12.12.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 05.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _DSP_-related ARVC is due to **decreased gene product level or altered gene product sequence**.
 The disease mechanism is loss of function via haploinsufficiency, dominant negative or both (PMID 32372669; 23137101; 16917092; NBK1131).
_DSP_-related ARVC is inherited in an **autosomal dominant manner characterized by incomplete penetrance** (PMID: 21636032; 33831308). However, **_DSP_ is associated with multiple phenotypes which are heterogeneous and overlapping (including DCM, DCM with cutaneous features, ARVC, and Carvajal syndrome) and autosomal recessive inheritance has been reported**. There does not appear to be distinct mechanisms leading to different phenotypes https://search.clinicalgenome.org/kb/gene-dosage/HGNC:3052.

 The initial variant description in _DSP_ was in Carvajal syndrome characterized by woolly hair, keratoderma and ARVC. In 2000 three families from Ecuador were found to be homozygous for the variant 7901delG in _DSP_ which produces a premature stop codon leading to a truncated desmoplakin protein missing the C domain of the tail region (PMID 11063735). Since then both autosomal dominant and autosomal recessive patterns of inheritance have been described in Carvajal syndrome (PMID: 27761164; 20940358; 22795705; 26604139; 23137101). This was followed by the description of a heterozygous variant in _DSP_ in an Italian family with ARVC with co-segregation of the variant with disease. There have been reports of digenic inheritance with other desmosomal pathogenic variants.

 In a retrospective multicentre study, curly hair and/or thick skin on the palms or soles (palmoplantar keratoderma) was commonly present in _DSP_ patients (54/98, 55%) but not in PKP2 patients (1/46, 2%) (PMID: 32372669). Maruthappu et al 2019 also describe 38 patients with arrhythmogenic cardiomyopathy who were carriers of a dominant loss-of-function (nonsense or frameshift) variants in _DSP_. Nearly all were found to have curly hair and palmoplantar keratoderma. However, there was one family described where the majority did not demonstrate a curly hair/cutaneous phenotype. The variant in this family was located in a fragment (c.3585–5379, (p.1195–1793)) only included in isoform 1 of _DSP_ (it has previously been shown that isoform 2 is the major isoform regulating keratinocyte adhesion (PMID: 30382575).

 Both truncating (stop gained, frame shift, splice site) and non-truncating variants in _DSP_ have been reported in the literature associated with ARVC (PMID 31319917; 20716751; 23810894; 24503780; 27532257; 31514951; 27761164; 21636032; 32808748). Pathogenic truncating variants are more common.
 Grondin et al 2020 re-evaluated reported missense variants and found an enrichment localizing to the spectrin repeat domain (SRD) in cases vs gnomAD. A similar hot spot location (amino acid residues 250-604) was reported by Kapplinger et al in 2011 (PMID: 32808748; 21636032).
 Smith E et al 2020 report that _DSP_ variants are associated with a distinct type of cardiomyopathy with a high prevalence of LV inflammation, fibrosis, and systolic dysfunction, and _DSP_ cardiomyopathy should be considered in the differential diagnosis for myocarditis and sarcoidosis (PMID: 32372669).
  </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> PKP2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 602861 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Arrhythmogenic right ventricular cardiomyopathy (ARVC) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial isolated arrhythmogenic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> PKP2-related ARVC </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012180 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant, Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice region; splice acceptor; splice donor; stop gained; frameshift; missense; structural variants (deletions; duplications) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 33831308; 21636032; 23736219; 34120153; 30830208; 25616645; 24070718; 17010805; NBK1131; 30619891; 24704780; 28740174; 22781308; 20301310; 17041889 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 14.12.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 05.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _PKP2_ pathogenic variants cause ARVC through **decreased gene product level or altered gene product sequence**. _PKP2_ encodes plakophilin-2 which is a protein of the desmosome and provides structural and functional integrity to adjacent cells. The disease mechanism in ARVC is loss of function (LoF). Rasmussen et al showed that truncating variants in _PKP2_ resulted in _PKP2_ transcript and protein levels reduced to ≈50% (PMID: 24704780). Cerrone et al showed that loss of _PKP2_ in adult myocytes was sufficient to generate an arrhythmogenic cardiomyopathy of right ventricle predominance in mice (PMID: 28740174).

 _PKP2_ is the major causative gene for ARVC and accounts for 34%-74% of cases (PMID: 20301310).
 **Inheritance is predominantly autosomal dominant characterised by variable expression and incomplete penetrance** (PMID: 34120153; 21636032; 17010805).

 Both recessive and digenic inheritance (with one pathogenic variant in _PKP2_ and a second in another desmosomal gene) have been reported (including a recessive cryptic splice variant PMID: 17041889) and appear to confer a more severe phenotype (PMID: 30830208; 25616645; 24070718; NBK1131). The expert panel noted instances where _PKP2_ LoF variants on both alleles had resulted in neonatal lethality.

 There are over 250 _PKP2_ variants listed in ClinVar for ARVC (nonsense, frameshift, splice, missense, deletions, duplications, and complex rearrangements (PMID: 30619891; 25616645; 21636032; 34120153).
 Dries et al report that _PKP2_ truncating variants explain a large proportion of ARVC cases but there is no clear relationship between their transcript position and their likelihood of disease association (PMID: 30619891). Although missense variants are associated with disease and validated with functional studies (PMID: 22781308), their mechanism and overall impact in ARVC is not completely understood. The majority of missense variants in ClinVar are classified as variants of uncertain significance. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TMEM43 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 612048 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Arrhythmogenic right ventricular cardiomyopathy (ARVC) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial isolated arrhythmogenic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TMEM43-related ARVC </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011459 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> p.Ser358Leu </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 20301310; 18313022; 21214875; 23812740; 24598986; 33831308; 21391237; 29980933; 25343256; 22725725; 32062046 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 14.12.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 05.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _TMEM43_-related ARVC is due to **altered gene product sequence.**
 Pathogenic variants in _TMEM43_ are a rare cause of ARVC (PMID: 20301310).
 **The majority of genetic evidence comes from one founder missense variant, (NM_024334.3, c.1073C&gt;T (p.S358L))** (PMID 18313022; 21214875; 23812740; 20301310; 24598986; 33831308). This was originally identified in Newfoundland and has subsequently been found in patients from other countries including USA, Germany, and Denmark (PMID: 33831308; 18313022; 23812740). It is reported that “[the variant occurs on] a common haplotype with those from Newfoundland, USA, and Denmark, suggesting that the mutation originated from a common founder. Examination of 40 control chromosomes revealed an estimated age of 1300-1500 years for the mutation, which proves the European origin of the Newfoundland mutation.” (PMID:  24598986)
  The disease mechanism is largely unknown. There is no evidence currently for haploinsufficiency (https://search.clinicalgenome.org/kb/gene-dosage/HGNC:28472).

 Although ARVC is known to display incomplete penetrance, this particular founder variant appears to be more penetrant.
  _TMEM43_-related ARVC is associated with a high risk of sudden cardiac death and characteristic clinical and electrocardiographic features (PMID: 32062046). Ventricular ectopy on Holter monitoring is commonly seen and can occur early in the natural history (PMID: 22725725).
 PMID 21391237 described two patients with _TMEM43_ heterozygous missense variants in Emery Dreifuss Muscular Dystrophy Related Myopathy. Other missense variants have been reported but their pathogenicity is debated. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> JUP </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 173325 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Arrhythmogenic right ventricular cardiomyopathy (ARVC) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare familial disorder with ARVC </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> JUP-related Naxos disease </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011017 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> STRONG </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Frameshift variant NMD escaping, missense, inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 10902626; 32966140; 17924338; 25820315 20031617; 25705887; 21673311; 11691526; 28098346; 15851108; 31402444; 20130592; 21320868; 8954745; 8858175 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 12.12.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 05.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _JUP_-related Naxos disease (ARVC, woolly hair and palmoplantar keratoderma) is due to **altered gene product sequence** causing loss of function of _JUP_ (PMID: 25705887; 21673311; 11691526; 10902626). _JUP_ encodes the protein plakoglobin. Kaplan et al found that in 4 Naxos patients, Connexin43 expression at intercellular junctions was significantly reduced and mutant plakoglobin was expressed but failed to localize normally at intercellular junctions (PMID: 15851108).
 **Inheritance is autosomal recessive**.
  The initial nine patients described ranged in age from 7 to 41 years. Since then, more patients have been discovered carrying the disease with an estimate of 1:1000 in the population of the Greek islands. The disease has also been diagnosed in other countries (PMID: 32966140).
 **A homozygous 2bp deletion** in plakoglobin (_JUP_), c.2157delTG, causing a frameshift and premature termination of the protein and expression of a truncated plakoglobin lacking 56 residues from the C terminus was described in 2000 (PMID: 10902626). The truncated protein was identified on western blot.
  **In 2017, a homozygous missense variant** was described in 7 unrelated French-Canadian individuals. All had typical hair and skin findings; 4/7 had ARVC presenting after 28 years (PMID: 28098346). The effect of this variant in the heterozygous state was not investigated.
 Two siblings of consanguineous parents were found to have a **homozygous 3bp deletion** in _JUP_ c.901_903delGAG (p.Glu301del). Both had woolly hair and skin findings, only the older sister had ARVC and neither had palmoplantar keratoderma (PMID: 28098346).
 In OMIM there have been reports of other types of homozygous variants (nonsense, splice, missense) in _JUP_ causing overlapping phenotypes and segregating with disease. Data on biallelic LoF variants are sparse.  In mice, generation of a null mutation of the plakoglobin gene by homologous recombination results in embryonic lethality (PMID: 8954745; 8858175). There are 2 reports in humans who had skin features but no obvious cardiomyopathy (PMID: 20130592; 21320868). In one, _JUP_ expression in the skin was absent. Cardiac _JUP_ expression was not directly measured to establish the consequence in the heart – it is not known whether variant allele was expressed, degraded, or rescued by alternate splicing.

 To note dominant pathogenic variants in _JUP_ have also been rarely described in association with ARVC. Asimaki et al reported a dominant variant in _JUP_ in a German family with ARVC and no obvious cutaneous abnormalities (PMID: 17924338). Other studies have identified heterozygous missense variants however their pathogenicity is still debated (PMID: 25820315; 20031617; 31402444). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><h3>Dilated Cardiomyopathy (DCM)</h3><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> BAG3 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 603883 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> BAG3-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0013479 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; stop gained NMD triggering; frameshift; splice acceptor; splice donor; structural variants (whole exon deletions), copy number variation (whole gene deletion) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 23518596; 20884878; 26796036; 21353195; 25008357; 27391596; 30442290; 31983221; 21898660; 20884878; 28737513; 31808029; 32160020 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 31.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 08.12.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _BAG3_ cause an estimated 2-4% of familial DCM due to **decreased gene product level or altered gene product sequence**. This likely leads to loss of function causing several pathological effects on cardiomyocytes including direct destabilization of the Z-disc, impaired protein homeostasis leading to proteotoxicity and increased susceptibility to apoptosis.
 **Truncating variants are responsible for most _BAG3_-related DCM cases**, the majority of which are nonsense and frameshift, and include several single, multi-exon, and whole gene deletions. ClinGen have concluded there is good evidence for haploinsufficiency. There is not yet evidence for dominant-negative effects.
 There is no apparent enrichment for non-truncating mutations in any specific domain, and it should be noted that different missense variants in similar domains can lead to different phenotypes (e.g., p.Pro209Leu leading to myofibrillar myopathy, and p.Arg218Trp leading to DCM).
 _BAG3_-related DCM is inherited as an autosomal dominant trait, characterised by incomplete penetrance.
  </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> DES </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 604765 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> DES-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011482 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; splice acceptor variant NMD escaping </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 17626518; 17325244; 10430757; 11728149; 17325244; 23300193; 23349452; 26724190; 17626518 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 31.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 08.12.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _DES_ cause DCM due to **altered gene product sequence**.
 **Only missense variants in _DES_ have been confidently reported as pathogenic in DCM**. There are no reports of truncating variants; _DES_ is yet to undergo ClinGen Dosage Haploinsufficiency investigations. _DES_ has a pLI of 0.01 in gnomAD (o/e = 0.33 (0.19 - 0.6)).

  There is some indication that truncating variants in _DES_ are associated with myofibrillar myopathy rather than DCM. There is a report of a splice site variant in a DCM case, however, it causes exon 3 skipping to produce an inframe transcript (PMID: 17626518).

 _DES_-related DCM is inherited in an autosomal dominant manner, and disease is generally highly penetrant. _DES_ missense variants account for a very small proportion (estimated 2%) of genetically-explained DCM (PMID: 17325244). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> DSP </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 615821 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> DSP-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0005021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Stop gained NMD triggering; frameshift variant; splice acceptor variant NMD triggering; splice donor variant NMD triggering; missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 31983221; 31317183; 27532257; 32005173; 24503780; 23022708; 20716751; 32013205 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 06.08.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 01.02.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _DSP_ cause DCM due to **decreased gene product level or altered gene product sequence** due to a variety of mechanisms.

 Both missense and truncating mutations have been reported in DCM cases, however, there is significantly more evidence in support of truncating variants being pathogenic.

 _DSP_-related DCM is inherited in an autosomal dominant manner, and disease is generally penetrant in families with an increased burden of LV fibrosis and ventricular tachyarrhythmia. Pathogenic _DSP_ variants are likely to account for approximately 2-3% of familial DCM cases (PMID: 24503780; 23022708).
 Of note _DSP_ is also associated ARVC for which _DSP_ is one of the most common causes. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> FLNC </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 102565 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> FLNC-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0005021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice acceptor; splice donor; frameshift variant NMD triggering; stop gained NMD triggering </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 27206985; 321126565; 27908349; 30067491; 2843697; 31627847; 27601210; 32150467; 22020047; 32160020; 32154132 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 31.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 08.12.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _FLNC_ account for approximately 2-4% of familial DCM due to **decreased gene product level**. The mechanism is likely loss of function, leading to Z-disc disarray and weakened cell-cell adhesion, promoting arrhythmogenesis and fibrosis.
 Truncating variants (stop-gained, frameshift and splicing) are responsible for almost all reported cases of _FLNC_-related DCM to date. It is possible that some loss-of-function missense variants could cause DCM and experts commented on their experience of families with missense variants segregating with disease. However, there is limited evidence for this. Xiao et al 2020 report an infant girl presenting with DCM and a paternally inherited missense variant, p.Arg441Ile in _FLNC_ identified on exome sequencing. No functional studies were carried out (PMID: 32154132).
  The _FLNC_-related DCM phenotype is frequently arrhythmogenic, characterized by a high burden of ventricular arrhythmias and myocardial fibrosis.
 _FLNC_-related DCM is inherited as an autosomal dominant trait, penetrance in close relatives of an affected proband is generally high, although not complete, with mean age of onset in the late 4th or early 5th decade, +/- 10-15 years. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> LMNA </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 115200 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> LMNA-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0007269 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by age-related onset </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; stop gained NMD triggering; frameshift variant; splice acceptor variant NMD triggering; splice donor variant NMD triggering; structural variants (single and multi-exon deletion) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 28912180; NBK1674; 28912180; 10580070; 10662742; 11897440; 29095976; 18926329; 29367541; 20127487; 12854972; 12920062; 31983221 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 30.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 08.12.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _LMNA_ cause DCM due to **decreased gene product level or altered gene product sequence**, likely due to a variety of mechanisms. Missense variants are more prevalent, and are responsible for a larger proportion of DCM cases, but truncating variants (nonsense, frameshift, and splice variants) are also associated with disease, most probably through loss of function mechanisms.
 _LMNA_ has a ClinGen Dosage sensitivity score of 2, indicating there is some evidence for dosage pathogenicity. Non-missense variants are reported to convey a higher risk of life-threatening arrhythmia (PMID: 31155932).

 _LMNA_-related DCM is inherited in an autosomal dominant manner, and disease is generally a highly penetrant and aggressive arrhythmogenic phenotype with high rates of heart failure and sudden cardiac death. _LMNA_ missense and truncating variants account for ~5-8% of genetic DCM.

 Of note _LMNA_ is also associated with several other conditions (the laminopathies) including muscular dystrophies and Hutchinson-Gilford progeria. Some of these conditions are autosomal recessive and some dominant. There is evidence of clustering of variants to specific regions of the gene. However, DCM causing variants have been recorded across the gene and are so far all reported with dominant inheritance. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> MYH7 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 160760 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> MYH7-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0013262 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 29300372; 31983221; 11106718; 29666183; 29093449 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 31.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 08.12.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _MYH7_ encodes the β-myosin heavy chain, part of the sarcomere which plays a major role in cardiac muscle contraction.
  Pathogenic variants in _MYH7_ cause DCM due to **altered gene product sequence** primarily due to decrease in sarcomere force generation.
 Dominant-negative missense variants resulting in an altered protein with reduced function are responsible for most _MYH7_-associated DCM cases, and a small number of inframe indels have also been reported associated with disease. The likely disease mechanism is a reduction in the passive stiffness of myofibrils (PMID: 29093449) and deficit in force generation and force-holding capacity (PMID: 29666183). There is no good evidence that loss of function is a disease mechanism (e.g. PMID: 31983221), and _MYH7_ is not known to be haploinsufficient.
 _MYH7_-related DCM is inherited in an autosomal dominant manner, and disease has incomplete penetrance, age-related onset, and variable expressivity. Pathogenic _MYH7_ variants are likely to account for approximately 5-6% of familial DCM cases.  There is evidence that biallelic variants can have an additive effect resulting in more severe cardiomyopathic phenotypes. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> RBM20 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 613171 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> RBM20-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0013168 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; stop gained NMD triggering </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> There is a mutation hotspot in exon 9 (amino acids 634-638) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 19712804; 22466703; 29650543; 32789749; 25979592; 26084686; 21846512; 20590677; 22004663; 29895960; 30871348; 30871351; 32851336; 29367541; 27496873; 29650543 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 31.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 08.12.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _RBM20_ cause DCM due to **decreased gene product level or altered gene product sequence** due to a variety of mechanisms including altered splicing of targets. A dominant negative effect causing disrupted RNA binding is also a possible/likely mechanism.
Missense variants are responsible for the majority of DCM cases although a small number of truncating variants (nonsense) have been reported. Of note, there are multiple ClinVar entries of truncating variants associated with DCM reported by diagnostic laboratories, comparing to the very low number of truncating variants detected in gnomAD.
 **Loss of function via truncating variants is not absolutely established, but highly likely.**
 _RBM20_-related DCM is inherited in an autosomal dominant manner, and disease is often an aggressive arrhythmogenic phenotype with high rates of heart failure and sudden cardiac death. Pathogenic _RBM20_ variants are likely to account for approximately 1.5-3% of familial DCM cases. There is a pathogenic variant hotspot in exon 9 (RS motif, amino acids 634-638). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> SCN5A </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 601154 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> SCN5A-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011003 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; stop gained NMD triggering </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 26916278; 15671429; 15466643; 19808398; 21596231; 22675453; 22999724; 30847666; 27532257 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 06.08.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 01.02.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _SCN5A_ cause DCM mainly due to **altered gene product sequence leading to gain of function, and more rarely decreased/absent gene product level**. The mechanism by which changes in sodium conductivity lead to cardiomyopathy is not fully understood; however, it is thought to be due to disruption of the voltage-sensing mechanism of this channel and subsequent disruption to action potential and cardiac contraction over time. It is unclear whether loss of function is a true disease mechanism (for DCM).

 Pathogenic variants in _SCN5A_ are likely to cause &lt;2% of familial DCM cases (PMID: 26916278, 21596231) and are associated with an arrhythmogenic phenotype with high rates of sudden cardiac death. There are no defined hotspot regions for _SCN5A_ missense variants in DCM, however, they do appear to commonly lie within voltage sensing regions (S3 and S4 transmembrane segments) of the protein.

 Of note _SCN5A_ loss of function is generally more associated with Brugada syndrome (BrS); _SCN5A_ is the only gene definitively associated with BrS. Gain of function variants are also associated with Long QT syndrome. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TNNC1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 191040 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TNNC1-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012745 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 1554228; 19808376; 17021793; 18803402; 32038292; 27604170; 18212018; 20458010; 27532257; 17977476 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 31.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 08.12.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _TNNC1_ are a rare (&lt;1%) cause of familial DCM due to **altered gene product sequence**. This leads to a likely ‘poison peptide’ dominant negative effect, causing alteration of troponin interactions, and altered (decreased) calcium binding of myofilaments and resulting in decreased force production.

 Missense variants are responsible for all reported cases of _TNNC1_-related DCM, with no evidence that truncating variants are causative of disease.
 There is no apparent enrichment for non-truncating mutations in any specific domain, although exon 1-3 show regional constraint (gnomAD database).

 _TNNC1_-related DCM is inherited as an autosomal dominant trait, with high penetrance observed in families. There is currently insufficient evidence for autosomal recessive inheritance, although there are some reports of early onset DCM and other cardiomyopathy associated with compound heterozygosity for _TNNC1_ variants (PMID: 27604170). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TNNT2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 601494 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TNNT2-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011095 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 27532257; 11106718; 15542288; 20031601; 20978592; 29367541 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 31.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 08.12.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _TNNT2_ cause DCM due to **altered gene product sequence**.
 Only missense variants have confidently been reported as pathogenic in _TNNT2_ DCM cases. There are no reports of truncating variants and _TNNT2_ has a ClinGen Dosage haploinsufficiency score of 0, and a PLi of 0 in gnomAD (19.6 expected and 20 observed).

 _TNNT2_-related DCM is inherited in an autosomal dominant manner, and disease is often an aggressive arrhythmogenic phenotype with high rates of heart failure and sudden cardiac death. _TNNT2_ missense mutations account for 3% of genetic DCM (PMID: 27532257).
 _TNNT2_ variants are associated with an early-onset and more severe form of DCM.
 Of note pathogenic variants in _TNNT2_ are also associated hypertrophic and restrictive cardiomyopathies. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TTN </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 188840 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TTN-related DCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011400 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance; Typified by age-related onset </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice acceptor; splice donor; frameshift variant NMD triggering; stop gained NMD triggering; missense [v rare]; structural variants (whole exon deletions); [classes must occur in A-band or &gt;90% PSI exons] </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> Variants must occur in exons (PSI &gt; 0.9). Missense variants with segregation evidence: p.Trp976Arg; p.Ala178Asp; p.Cys3575Ser </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 9817758; 9826585; 19789381; 18765796; 25589632; 11788824; 22335739; 32013205; 23418287; 26084686; 28045975; 25759365; 26315439; 29316444; 27869827; 29238064; 32160020; 27869827; 31849696; 27625337; 11788824 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 31.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 08.12.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _TTN_ cause an estimated 15-20% of familial DCM due to **decreased gene product level and altered gene product sequence**. The likely disease mechanism is loss of function however, it is unclear whether this is due to haploinsufficiency or a dominant negative effect, and it is likely that both mechanisms contribute. It is likely that the reduced function has a direct effect on the sarcomere, leading to impaired contractility.
  Truncating variants, specifically in exons constitutively expressed in cardiac tissue (PSI &gt;0.9) see (PMID: 25589632; 27869827; 32160020), are responsible for the vast majority of _TTN_-related DCM cases. Missense variants are difficult to interpret, and generally not classified as disease-causing, although there are reports of at least three missense variants with evidence of pathogenicity.  
_TTN_ missense variants with segregation evidence:
 p.Trp976Arg (PMID: 11788824)
 p.Ala178Asp (PMID: 27625337)
 p.Cys3575Ser (https://www.biorxiv.org/content/10.1101/2020.09.05.282913v1.full.pdf) – note not yet peer reviewed


_TTN_-related DCM is inherited as an autosomal dominant trait and displays incomplete and age-related onset. _TTN_ truncating variants are present in ~1% of the general population, although these variants are more likely to reside in isoforms with lower functional expression in cardiac tissue. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> PLN </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 172405 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Dilated Cardiomyopathy (DCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Dilated Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> PLN-related intrinsic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012362 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by age-related onset </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe deletion; frameshift; stop gained; structural variants (whole exon deletions) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 12639993; 16432188; 33020536; 21167350 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 17.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 24.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _PLN_ cause cardiomyopathy by **decreased gene product level or altered gene product sequence**.
 Inheritance is typically autosomal dominant with incomplete penetrance but biallelic variants have been described and appear to confer an earlier onset and more severe phenotype (PMID: 12639993).

 **_PLN_ is encoded by one coding exon (52 amino acids)**. Schmitt et al described a missense variant (p.Arg9Cys) in a patient with DCM. The variant segregated with disease in the family; transgenic mice developed biventricular dilatation (PMID: 12610310). A stop gained variant and inframe deletion have also been described (PMID: 12639993; 16432188). There are 6 pathogenic/likely pathogenic variants reported on ClinVar: 1 missense, 2 stop gained, 2 frameshift and a large deletion all associated with dilated cardiomyopathy.
 In the Netherlands there is a founder mutation p.Arg14del. Up to 10–15% of both dilated cardiomyopathy and arrhythmogenic cardiomyopathy patients are reported to be caused by _PLN_-R14del. (PMID: 33020536).

 ClinGen found no difference in the molecular mechanism(s) underlying _PLN_-related DCM and HCM and observed that inter and intrafamilial variability in phenotype had been reported. Haghighi et al describe the same variant (p.Leu39X) in a family causing severe DCM in the homozygous state and both DCM and HCM phenotypes in the heterozygous state (PMID: 12639993). This variant has also been reported in other HCM families (PMID: 21167350). As a result, ClinGen curated _PLN_ for an association with intrinsic cardiomyopathy and did not separately evaluate the evidence for the role in hypertrophic vs dilated phenotypes.

 _PLN_ is definitively associated with cardiomyopathy and the majority of variants reported appear to be associated with DCM. Experts commented that further investigation into _PLN_ and HCM and the variant classes associated needs to be undertaken. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><h3>Hypertrophic Cardiomyopathy (HCM)</h3><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> ACTC1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 102540 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial hypertrophic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> ACTC1-related HCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012799 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 10330430; NBK1768; 20031618; 28007147; 27532257; 10966831; 17611253; 26061005; 23604709; 28972856 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 10.10.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 10.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _ACTC1_ pathogenic variants cause HCM through **altered gene product sequence**. The disease mechanism is not definitively known but may involve impact on sarcomere force generation. There is currently little evidence to support haploinsufficiency as a mechanism.
 Inheritance is autosomal dominant. There is limited information regarding penetrance. The initial missense variant identified by Mogensen et al was reported to be highly penetrant in family members but with a variable age of onset and severity (PMID: 10330430).
 _ACTC1_ variants account for &lt;3-5% of HCM cases (NBK1768; 20031618). 
 **Heterozygous missense variants are the major type of pathogenic variants found**. One inframe deletion (a deletion of a single amino acid, Phe92del) has been reported in association with HCM (PMID: 20031618). This variant is classified as likely pathogenic on ClinVar. Liu et al suggest it may act to change the local structure and arrangement of amino acids in the actomyosin binding site (PMID: 28972856).
 Large deletions or duplications have not been described. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> MYBPC3 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 600958 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial hypertrophic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> MYBPC3-related HCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0007268 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice region; splice acceptor; splice donor; frameshift; frameshift variant NMD triggering; stop gained; stop gained NMD triggering; missense; inframe insertion; inframe deletion; intronic; structural variants (whole exon deletions) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 28007147; 28912181; 7493025; 26573135; 25611685; 32731933; 29300372; NBK1768; 25611685; 30696458; 12707239; 20624503; 28912181; 27532257; 31877118; 22057632; 32163302; 32396390; 18467358; 17937428; 19151713; 32841044 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 10.10.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 10.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _MYBPC3_ pathogenic variants cause HCM through **decreased gene product level or altered gene product sequence** either leading to a reduction in MyBP-C content in the sarcomere or altered function. **The disease mechanism is loss of function**; There is evidence of haploinsufficiency (PMID 31877118; 22057632; 32841044).
 Variants in _MYBPC3_ and MYH7 collectively account for up to 50% of all clinically recognised cases of HCM and constitute at least 75% of probands where a variant is identified (PMID: 28007147; NBK1768).
 Inheritance is usually autosomal dominant, typified by incomplete penetrance and variable expressivity.
  Homozygous and compound heterozygous variants have been reported and can lead to severe, early onset phenotypes (PMID: 26573135; 25611685; PMID: 18467358; PMID: 17937428).

  **The majority of variants are heterozygous frameshift, nonsense, or splice site variants** that result in premature termination codons (PMID: 25611685; 31877118). Missense and inframe indels are also frequently reported and a subset have been shown to cause loss of function through failure of myofilament incorporation and rapid degradation, further supporting haploinsufficiency as a mechanism (PMID: 32841044). Variants in _MYBPC3_ affecting canonical splice site dinucleotides are a well-characterised cause of HCM.  Furthermore, recent work has identified more deeply intronic variants associated with disease (https://doi.org/10.3390/cardiogenetics11020009; PMID: 32396390). There are 39 pathogenic/likely pathogenic _MYBPC3_ intronic variants submitted on ClinVar.
 The common intronic deletion, _MYBPC3_Δ25, detected in 4% to 8% of South Asian populations, is associated with cardiomyopathy as a risk allele (PMID: 19151713).
 Although there is significant genetic and allelic heterogeneity in HCM, there are also several _MYBPC3_ founder variants (PMID: 28912181; 27532257). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> MYH7 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 160760 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial hypertrophic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> MYH7-related HCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0008647 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe deletion; stop gained NMD escaping </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 29300372; 27532257; 30696458; 30924982; 25209314; 1975517; 1944483; 1552912; 15856146; 34460321; 30681346; 25611685; 20359594; 32731933; 12788380; 33500567; 27247418; 27532257 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 10.10.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 10.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _MYH7_ pathogenic variants cause HCM through **altered gene product sequence** leading to an increase in sarcomere force generation. Variants produce an abnormal activated protein that incorporates into the sarcomere as a ‘poison peptide’. Variants can either directly affect motor function or can impact on myosin "interacting head motif" and therefore impair inactivation kinetics. There is currently no evidence to support haploinsufficiency as a disease mechanism.
 Inheritance is autosomal dominant, typified by incomplete penetrance and variable expressivity. 

 **Most pathogenic variants are missense**. There are some inframe deletions reported.
 A frameshift variant has been identified in 3.3% of Egyptian HCM patients. It is predicted to result in a premature termination codon downstream of the last exon-exon junction of the gene that is expected to escape nonsense-mediated decay (NMD). (PMID: 34460321).

 _MYH7_ loss of function (LoF) variants are very rare and their contribution to inherited cardiomyopathy is incompletely understood. Notably in a recent study of LVNC, _MYH7_ truncating variants, generally considered non-pathogenic for cardiomyopathies, were 20-fold enriched in LVNC cases over controls (PMID: 33500567). While there is currently no evidence for a disease-causing role in the heterozygous state in HCM, compound heterozygosity of LoF variants along with missense variants can lead to extremely severe presentations, mimicking recessive inheritance.

 There is a clustering of HCM variants in the head region conferring a high probability of pathogenicity (amino acid residues 181-937). (PMID: 27247418; 27532257; 29300372; 30696458). 
 Kelly et al 2018 (PMID: 29300372) provide gene-specific adaptations of ACMG criteria for _MYH7_ and HCM. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> MYL2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 160781 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial hypertrophic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> MYL2-related HCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012112 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> NBK1768; 32731933; 8673105; 9535554; 12404107; 30696458; 32453731; 23365102; 28007147; 28912181; 25611685; 16837010; 25324513; 16076902; 24111713 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 10.10.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 10.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _MYL2_ pathogenic variants cause autosomal HCM through **altered gene product sequence**. The disease mechanism is not definitively known but may involve destabilization of the interacting heads motif (PMID: 28606303).
 There is currently insufficient evidence to support haploinsufficiency (https://search.clinicalgenome.org/kb/gene-dosage/HGNC:7583).
 _MYL2_ variants account for &lt;3% of HCM cases (NBK1768; 32731933).
 Inheritance is typically autosomal dominant. There is limited information regarding penetrance.

 Homozygous and compound heterozygous variants have been described in association with a lethal myosinopathy (PMID: 23365102). A study reported a homozygous frameshift variant causing infantile onset HCM. Heterozygous parents were unaffected. The authors suggest a molecular mechanism by which loss-of-function variants in _MYL2_ are recessive while missense variants are dominant in HCM. Several loss-of-function variants are reported in gnomAD suggesting _MYL2_ is not intolerant to LoF variants. (PMID: 32453731). 

**The majority of variants reported are missense** (PMID: 8673105; 9535554; 12404107.)
  On ClinVar, nearly all pathogenic variants are missense. There is one frameshift variant associated only with cardiomyopathy, not HCM specifically and a deletion encompassing exon 7 expected to result in a truncated protein - https://www.ncbi.nlm.nih.gov/clinvar/variation/417460/.
 Walsh et al found the _MYL2_ gene to be significantly enriched for non-truncating variants, odds ratio 9.1 (6.2-13.3) (PMID: 30696458). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> MYL3 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 160790 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial hypertrophic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> MYL3-related HCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012111 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 33288880; NBK1768; PMID: 20031618; 28369730; 26443374; 12021217; 30696458; 8673105; 20031618; 25611685; 22957257; 29914921 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 10.10.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 24.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _MYL3_ pathogenic variants cause HCM through **altered gene product sequence**. The disease mechanism is not definitively known but may involve impairing protein-protein interaction with components of the sarcomere and destabilizing the interacting heads motif (PMID: 30275503; 28606303). There is currently no evidence to support haploinsufficiency as a mechanism (https://search.clinicalgenome.org/kb/gene-dosage/HGNC:7584).
 _MYL3_ variants account for &lt;3% of HCM cases (NBK1768; 20031618; 28369730).
 Inheritance is primarily autosomal dominant. There is limited information regarding penetrance given small numbers of variants identified. In a large family where a missense variant was identified (p. Arg94His), penetrance was estimated at 88% (PMID: 26443374).
 Osborn et al 2021 (PMID: 33288880) report a homozygous missense variant in a large family with HCM and sudden cardiac death. Heterozygous carriers were unaffected. Another biallelic variant was also described by Olson et al in 2002 (E143K) (PMID: 12021217), however this variant has also been associated with HCM variably in the heterozygous state. 

**Heterozygous missense variants are the major type of pathogenic variants found** (PMID: 8673105; 20031618; 25611685). There are reports on ClinVar of frameshift and splice site variants, but these are classified as uncertain significance or conflicting. There is currently insufficient evidence to support loss of function as a mechanism.
  Walsh et al reported a cluster of non-truncating _MYL3_ variants in HCM at amino acid residues 143-180 (PMID: 30696458). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TNNI3 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 191044 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial hypertrophic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TNNI3-related HCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0013369 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 9241277; 28912181; 21839045; 23270746; 15607392; NBK1768; 32731933; 26440512; 25611685; 30696458; 21415410 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 10.10.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 24.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _TNNI3_ pathogenic variants cause HCM through **altered gene product sequence**. The disease mechanism is not definitively known but a review by Tardif et al reported functional studies on missense variants that found an increase in the Ca2+ sensitivity of myofilament activation (PMID: 21415410). ClinGen also conclude that “missense mutations have [been] shown to affect Ca2+ binding to myofilaments containing the mutant _TNNI3_ (PMIDs: 16531415 and 22675533) or result in an increased myofilament response to Ca2+ (PMID: 11735257).” https://search.clinicalgenome.org/kb/genes/HGNC:11947.
 _TNNI3_ variants account for approx. 3-5% of HCM cases (NBK1768; PMID: 15607392).

 **Inheritance is autosomal dominant characterised by incomplete penetrance** (PMID: 26440512; 15607392; 9241277). Lorenzini et al found that subjects with _TNNI3_ variants had a lower penetrance than variants in MYBPC3, MYH7, and TNNT2 (PMID: 32731933). Experts commented that variants seen in _TNNI3_ in severe/early onset disease were more likely to be ___de novo___ in origin.
 Maron et al 2012 describe 4 HCM probands with both pathogenic variants in _TNNI3_ and _MYBPC3_ (PMID: 21839045). In addition, 2 siblings have been reported with homozygous _TNNI3_ missense variants (Arg162Trp) and severe myocardial hypertrophy. Parents of the affected children are consanguineous and along with other family members harbouring the same variant in the heterozygous state, were unaffected (PMID: 23270746). Multiple disease-causing sarcomeric variants appear to be associated with more severe disease. There is not enough evidence to suggest biallelic variants in _TNNI3_ have a distinct mechanism compared to monoallelic variants. Of note both dominant and recessive modes of inheritance have been reported in _TNNI3_-related DCM (PMID: 26440512).

 **Heterozygous missense variants are the major type of pathogenic variants found**. There are also reports of inframe deletions. In a systematic review, Mogensen et al reported 91% of all variants were missense variants. Six variants (Arg141Gln, Arg145Trp, Arg157Val, Arg162Gln, Ser166Phe, and Lys183Del) appeared with a particularly high frequency and were identified in 116 of the 256 probands (45%) (PMID: 26440512). There are reports on ClinVar of frameshift, splice site variants and nonsense variants but these are either classified as uncertain significance/conflicting, not associated with HCM specifically or have no functional evidence to demonstrate loss of function as a mechanism. ClinGen have concluded that there is currently limited evidence to support haploinsufficiency as a mechanism.
 
Mogensen et al found that 85% of variants were identified in exons 7 and 8 (amino acid residues 125 – 210). (PMID: 26440512).
 Similarly, Walsh et al detected a cluster of non-truncating _TNNI3_ variants in HCM at amino acid residues 141–209 (PMID: 30696458). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TNNT2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 191045 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial hypertrophic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TNNT2-related HCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0007266 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; stop gained NMD escaping; inframe deletion; splice donor variant NMD escaping </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 8205619; 28973951; 7898523; 28007147; 28912181; 30578328; 10965086; 11034944; 32731933; NBK1768; 12707239; 22144547; 25611685; 30696458 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 10.10.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 24.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _TNNT2_ pathogenic variants cause HCM through **altered gene product sequence**. Troponin T is a regulatory protein found in striated muscles that forms a complex with troponin I (TnI) and troponin C (TnC) that, together with tropomyosin (TM), is required for Ca2+ dependent regulation of muscle contraction.

  The mechanism appears to be dominant negative rather than haploinsufficiency. **The majority of pathogenic variants are missense**. A functional study of a splice variant associated with HCM concluded that the resulting truncated protein does not function as a “null protein, but rather as a dominant-negative leading to reduction in the level of calcium activated force production” (PMID: 8958207). Gangadharan et al suggest the primary reason by which _TNNT2_ variants between residues 92 and 144 cause cardiomyopathy is by changing the affinity of TnT for Tm within the N terminal part of Troponin T (PMID: 28973951).
 _TNNT2_ variants account for &lt;5% of HCM cases (NBK1768; 12860912)
 **Inheritance is autosomal dominant characterised by incomplete penetrance** (PMID: 32731933).
 Homozygous _TNNT2_ variants causing HCM are rare and have been reported to be associated with more severe disease (PMID 30578328; 10965086; 11034944). Piroddi et al investigated a patient with severe, early onset HCM with a homozygous K280N variant. There was no family information available, however the authors demonstrated that it resulted in 100% mutant cTnT with no evidence of haploinsufficiency and suggested this supports the idea of a gene dose–dependent effect of HCM variants on the severity of the phenotype (PMID: 30578328).
 Experts commented that variants seen in _TNNT2_ in severe/early onset disease were more likely to be ___de novo___ in origin.
 Nearly all pathogenic variants are missense (PMID: 12707239; 22144547; 25611685). However, there are a few reports of inframe deletions, nonsense variants in the final exon which likely escape nonsense mediated decay and splice donor variants (PMID: 8958207; 12707239; 22144547; 25611685).
 https://www.ncbi.nlm.nih.gov/clinvar/variation/43673/; 
 https://www.ncbi.nlm.nih.gov/clinvar/variation/177636/ 

 Walsh et al found non truncating variants in the _TNNT2_ gene to be significantly enriched in HCM cases, odds ratio 11.4 (8.5 – 15.2). They found a clustering of non-truncating variants conveying a high probability of pathogenicity in the tropomyosin binding domain (amino acid residues 79 – 179) (PMID: 30696458).
  </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TPM1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 191010 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial hypertrophic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TPM1-related HCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0007267 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 8205619; 24005378; 12860912; NBK1768; 7898523 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 10.10.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 24.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _TPM1_ pathogenic variants cause HCM through **altered gene product sequence**. _TPM1_ encodes alpha-tropomyosin which acts to place the troponin complex on cardiac actin. The mechanism is through altered function rather than haploinsufficiency – Bottinelli et al examined the Asp175Asn variant from 2 HCM patients and found equal expression of wild type and mutant alpha-tropomyosin proteins (PMID: 9440709). Gupte et al concluded that “_TPM1_ mutations cause differences in protein stability, actin binding, and Tn conformation. All of these differences could converge to change the Ca2+ dependence of myosin activity” (PMID: 25548289).
 _TPM1_ variants account for &lt;3% of HCM cases (NBK1768; PMID: 7898523; 12860912).
 **Inheritance is autosomal dominant characterised by incomplete penetrance** (PMID: 32731933). 
 There are reports of homozygous or compound heterozygous variants in _TPM1_. (PMID: 33642254; 32744700).
 A homozygous missense variant, (p.Gly3Arg), in exon 1 of _TPM1_ was identified in triplets (two had HCM and one patent ductus arteriosus). The parents were heterozygous for the variant and unaffected clinically and on echocardiogram (PMID: 32744700).

 **Pathogenic variants are nearly always heterozygous missense variants**. There is one ___de novo___ inframe deletion (6bp, 2 amino acids) associated with HCM classified as likely pathogenic on ClinVar, no additional details are given. 
 Redwood and Robinson reviewed _TPM1_ variants in 2013 and reported “at least 15 described in the current literature (Table1). Most of these mutations are unique and have been reported in only a single family or individual…Each mutation is missense causing a single amino acid substitution, and no truncation or nonsense mutations have been reported.” (PMID: 24005378).
 Walsh et al found that in HCM non-truncating variants across the whole _TPM1_ gene showed a high probability of pathogenicity. (PMID: 30696458) </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> PLN </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 172405 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Familial Hypertrophic Cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> PLN-related intrinsic cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012362 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe deletion; frameshift; stop gained; structural variants (whole exon deletions) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 12639993; 16432188; 33020536; 21167350 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 17.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 24.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic variants in _PLN_ cause cardiomyopathy by **decreased gene product level or altered gene product sequence**.
  Inheritance is typically autosomal dominant with incomplete penetrance but biallelic variants have been described and appear to confer an earlier onset and more severe phenotype (PMID: 12639993).

 **_PLN_ is encoded by one coding exon (52 amino acids)**. Schmitt et al described a missense variant (p.Arg9Cys) in a patient with DCM. The variant segregated with disease in the family; transgenic mice developed biventricular dilatation (PMID: 12610310). A stop gained variant and inframe deletion have also been described (PMID: 12639993; PMID 16432188). There are 6 pathogenic/likely pathogenic variants reported on ClinVar: 1 missense, 2 stop gained, 2 frameshift and a large deletion all associated with dilated cardiomyopathy.

 In the Netherlands there is a founder mutation p.Arg14del. Up to 10–15% of both dilated cardiomyopathy and arrhythmogenic cardiomyopathy patients are reported to be caused by _PLN_-R14del. (PMID: 33020536).

 ClinGen found no difference in the molecular mechanism(s) underlying _PLN_-related DCM and HCM and observed that inter and intrafamilial variability in phenotype had been reported. Haghighi et al describe the same variant (p. Leu39X) in a family causing severe DCM in the homozygous state and both DCM and HCM phenotypes in the heterozygous state (PMID: 12639993). This variant has also been reported in other HCM families (PMID: 21167350). As a result, ClinGen curated _PLN_ for an association with intrinsic cardiomyopathy and did not separately evaluate the evidence for the role in HCM and DCM phenotypes.

 _PLN_ is definitively associated with cardiomyopathy and the majority of variants reported appear to be associated with DCM. Experts commented that further investigation into _PLN_ and HCM and the variant classes associated needs to be undertaken. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> ALPK3 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 617608 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> ALPK3-related HCM </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011001 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> STRONG </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive; Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal; monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Stop gained; frameshift; splice acceptor; splice donor; missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 27106955; 31074094; 32480058; 26846950; 21441111; 34263911; 34263907; 33191771 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 11.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 24.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _ALPK3_ pathogenic variants cause hypertrophic cardiomyopathy through **decreased gene product level or altered gene product sequence**. 
**Biallelic _ALPK3_ variants were first associated with a rare recessive form of cardiomyopathy** by Almomani et al in 2016 (PMID: 26846950), with several other reports since (PMID: 27106955; PMID: 31074094; PMID: 32480058). Patients display variable phenotypes but often present at birth or early childhood (at least 4 patients have presented in utero) with dilated cardiomyopathy (DCM) that progressed to a HCM phenotype over time. **Extra-cardiac features including musculoskeletal and craniofacial abnormalities are also commonly observed in these cases** (PMID: 32480058; PMID: 34263911).
 Herkert et al 2020 reviewed the variants and phenotype in 19 paediatric patients with biallelic _ALPK3_ variants (including 9 previously published cases) and identified 11 loss-of-function (LoF) variants (including nonsense, frameshift and intronic variants with predicted severe effect on splicing), seven compound LoF and deleterious missense variants, and one homozygous deleterious missense variant, c.5155G&gt;C, p.(Ala1719Pro) (PMID: 32480058). The clinical manifestations associated with the missense variants were similar to those associated with other damaging _ALPK3_ variants. 
 To note heterozygous LoF _ALPK3_ variants have also been reported and are enriched in adults with cardiomyopathy (PMID: 34263907; PMID: 32480058; PMID: 33191771). 

Haploinsufficiency is the proposed mechanism. Herkert et al found notable differences between the clinical features associated with monoallelic and biallelic _ALPK3_ cardiomyopathy, including absence or undetected extracardiac phenotypes. “Whether these differences reflect graded dose-responses to _ALPK3_ deficits or distinct mechanisms by which monoallelic or biallelic variants cause disease remains unknown.” (PMID: 32480058). 
 **There does not appear to be a distinct mechanism between monoallelic and biallelic variants.
 The disease mechanism is loss of function** of _ALPK3_ likely due to creation of premature stop codons, leading to nonsense-mediated decay or truncated proteins with partial or complete removal of the kinase domain (PMID: 26846950; 27106955; 32480058; 21441111). Herkert et al predicted that missense variants could result in a conformational change that affects protein folding or flexibility, protein-protein or protein-DNA interaction, or the activity of the alpha-kinase domain (PMID: 32480058). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CACNA1C </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 114205 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CACNA1C-related Timothy syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0010979 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typically de novo </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> A recurrent, de novo missense variant causing Classic Timothy Syndrome has been described: p.Gly406Arg </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 28211989; 25633834; NBK1403; 15863612; 15454078; 22106044; 26253506; 24728418; 31983240; NBK1403; 30681346; 33797204; 16360093 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 11.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 24.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _CACNA1C_-related Timothy syndrome is caused by variants leading to **altered gene product sequence**. 
 Classic Timothy syndrome (TS1) is a very rare multisystem disorder characterized by marked QT prolongation, syndactyly, immune deficiency, seizures, congenital heart defects, hypertrophic cardiomyopathy, cognitive abnormalities, learning difficulties, and intermittent hypoglycaemia (PMID: 28211989; NBK1403). Infants can present with severe biventricular hypertrophy (PMID 30681346).  In some individuals LVH may be the presenting feature, without recognised syndromic features, fulfilling a clinical diagnosis of HCM. 
 **A recurrent, de novo, missense variant** in _CACNA1C_ was described in 13 Timothy syndrome patients, p.Gly406Arg in exon 8A (PMID: 15454078; PMID: 15863612). The mechanism appears to be gain-of-function through failed channel inactivation. Boczek et al suggest Ca2+ mishandling may lead to ventricular hypertrophy (PMID: 26253506).

  _CACNA1C_ has a complex genomic structure that undergoes extensive alternative splicing. Splawski et al identified 2 patients with **de novo missense variants in exon 8** of an alternate splice form (p.Gly406Arg, analogous to the exon 8a variant, and p.Gly402Ser). This splice form represents 80% of all cardiac mRNAs. The patients were described as having atypical Timothy syndrome (TS2), presenting with a more severe cardiac phenotype (biventricular hypertrophy, moderate biventricular dysfunction, more severe QT prolongation and multiple arrythmias) and without syndactyly (PMID: 15863612; PMID: 25633834).
  Boczek et al describe a novel variant **p.Arg518Cys-_CACNA1C_** as the probable pathogenic substrate for COTS [cardiac only Timothy syndrome]. The phenotype included LQTS, hypertrophic cardiomyopathy, congenital heart defects and sudden cardiac death. Follow-up cohort analysis revealed two additional pedigrees, with very similar phenotypes, having variants at the exact same amino acid positions **p.Arg518Cys and p.Arg518His**. (PMID: 26253506)

 Other missense variants in _CACNA1C_ have been reported in association with isolated LQT (PMID: 26253506; 25633834; 24728418). However as of 2020, the ClinGen Cardiovascular Domain Working Group have classified the strength of evidence supporting an association between _CACNA1C_ and LQTS as moderate (PMID: 31983240). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> DES </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 125660 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> DES-related Myofibrillar myopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0011076 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant; autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal; biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance; Typified by age-related onset </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence; absent gene product </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice acceptor variant NMD escaping; splice donor variant NMD escaping; frameshift variant NMD triggering; frameshift variant NMD escaping; stop gained NMD triggering; stop gained NMD escaping; missense variant; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 19181099; 20718792; 29926427; 19433360; 23815709; 16217025; 11073539; 31718026; 9736733 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 12.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _DES_-related myofibrillar myopathy is typically inherited as an autosomal dominant condition. Penetrance is incomplete with age-related onset, but simplex cases have often been found to be due to _de novo_ heterozygous variants rather than recessive variants. Over 100 different variants have been reported, including missense variants, in-frame indels, and splice-site variants leading to exon skipping.
 There is evidence for a possible genotype-phenotype correlation for heterozygous _DES_ variants, with variants located in specific domains more likely to cause either a neurological or cardiac phenotype.
 There is a wide range of cardiac phenotypes associated with _DES_, including hypertrophic cardiomyopathy, restrictive cardiomyopathy, dilated cardiomyopathy, arrhythmogenic cardiomyopathy, and left-ventricular non-compaction. Cardiac involvement can be the presenting feature, even without recognised syndromic features, fulfilling a clinical diagnosis of HCM.
 The mechanism of autosomal dominant disease is likely a dominant-negative effect of protein-altering variants leading to abnormal intermediate filament aggregation. The autosomal recessive phenotype has been associated with an earlier age of onset, and has been linked with combinations of protein-altering variants and/or null variants. It has been suggested that biallelic loss of function leads to the autosomal recessive phenotype, indicating a distinct mechanism of disease to the dominant phenotype. The heterozygous parents of patients with the recessive phenotype are reported as being clinically unaffected, suggesting that heterozygous loss of function is tolerated. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> FHL1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 300163 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> FHL1-related Emery-Dreifuss muscular dystrophy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0010680 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> X-linked recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic X hemizygous </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by age-related onset </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Splice region variant; frameshift variant NMD escaping; stop gained NMD escaping; stop lost; missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 19716112; 20186852; 32993534; 22523091 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 12.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _FHL1_-related Emery-Dreifuss muscular dystrophy is inherited as an X-linked condition. Carrier females are usually unaffected, but some exhibit a cardiac and/or skeletal muscle phenotype.
 The precise mechanism is unclear, but may be mediated by reduced function or dominant-negative effects of altered _FHL1_, associated with frameshift/truncating variants predicted to escape NMD, missense variants, or loss of the native stop codon.
 _FHL1_ pathogenic variants are responsible for a minority of Emery-Dreifuss muscular dystrophy cases (~1.2% according to GeneReviews). Hypertrophic cardiomyopathy is more typical of _FHL1_-related Emery-Dreifuss muscular dystrophy, while LMNA and EMD Emery-Dreifuss muscular dystrophy are more commonly associated with dilated cardiomyopathy.
 There are reports of _FHL1_ variants causing isolated hypertrophic cardiomyopathy, without skeletal myopathy. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> FLNC </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 102565 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> FLNC-related Myofibrillar myopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0019150 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by age-related onset </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Stop gained NMD escaping; missense variant; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 15929027; 25351925; 19050726; 32022900; 26666891; 27908349; 31245841; 32112656; 20697107; 21135393 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 12.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _FLNC_-related myofibrillar myopathy is inherited as an autosomal dominant condition with age-related onset. Myofibrillar myopathy and isolated hypertrophic cardiomyopathy have been associated with protein-altering variants in _FLNC_ (missense, in-frame deletion, and nonsense predicted to escape NMD).
 The specific mechanism of disease for pathogenic missense variants is not clear, but there is a cluster of HCM-related variants in the ROD2 domain, suggesting that these variants may affect interaction with the sarcomeric Z-disk. Variants associated with skeletal myopathy have been shown to induce intracellular aggregation of mutant _FLNC_.
 Loss-of-function variants in _FLNC_ have been associated with non-syndromic dilated and arrhythmogenic cardiomyopathy.
 Sequencing of _FLNC_ exons 46-48 is complicated by the presence of a pseudogene with &gt;98% homology. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> GLA </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 300644 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> GLA-related Fabry disease </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0010526 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> X-linked recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic X hemizygous </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe insertion; inframe deletion; splice donor; splice acceptor; frameshift; stop gained; intronic variants leading to cryptic splice site; structural variants </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 18940466; 26937390; 6023233; 2539398; 32640076; NBK1292; 27560961; 30988410; 11322659; 34576250; 34776082; 32640076 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 15.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 24.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic _GLA_ variants cause Fabry disease by **decreased gene product level or altered gene product sequence**. The disease mechanism is loss of function (PMID: 18940466; 26937390; 6023233; 2539398). _GLA_ pathogenic variants result in mRNA instability and/or severely truncated a-galactosidase A (a-Gal A) enzyme or an enzyme with markedly decreased activity (NBK1292).

 Fabry disease is an X-linked lysosomal storage disease caused by pathogenic variants in the _GLA_ gene leading to a greatly reduced or absent activity of a-Gal A, responsible for metabolizing glycosphingolipids. This condition is associated with a progressive accumulation of globotriaosylceramide (Gb3) and its deacylated form, globotriaosylsphingosine (lysoGb3), potentially affecting any organ or tissue (PMID: 32640076).
 **Fabry disease is inherited in an X linked manner**. Heterozygous females typically have milder symptoms at a later age of onset than males. Rarely, they may be relatively asymptomatic throughout a normal life span or may have symptoms as severe as those observed in males with the classic phenotype (NBK1292). 

 Variant classes include missense, nonsense, splice site, frameshift, in-frame deletions, and structural variants. A recurrent intronic variant (c.640-801G\&gt;A) is recognised as pathogenic and leads to aberrant mRNA splicing. 
 Many variants are unique however there are recognised recurrent variants also. (NBK1292; PMID 27560961; 30988410; 11322659; 34576250). 
 Pathogenic variants leading to complete loss of function of the gene product are usually associated with classic forms of the disease, whereas variants resulting in amino acid substitutions and residual enzyme activity can present atypically with either symptoms not specific to Fabry's (e.g. cardiomyopathy) or a milder phenotype and later onset. Attempts to correlate genotype with clinical presentation have been largely unsuccessful. (PMID 27560961; 18940466). Experts noted that conduction disease can be the presenting or only feature of disease.

 The expert consensus on the management of Fabry Disease in 2020 suggested that “Assessment of plasma lyso-Gb3 should be considered for assessment of disease severity in FD patients or in the diagnostic algorithm for patients with _GLA_ genetic variants of unknown significance.” (PMID: 32640076) </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> LAMP2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 309060 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> LAMP2-related Danon disease </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0010281 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> X-linked dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic X heterozygous </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by age-related onset </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Copy number variants; splice region variant; splice acceptor variant NMD triggering; splice donor variant NMD triggering; frameshift variant NMD triggering; stop gained NMD triggering; start lost; missense variant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 10972294; 30857840; 20173215; 19588270; 19057086; 16217705; 15907287; 15673802 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 12.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Pathogenic _LAMP2_ variants cause Danon disease. Danon disease follows X-linked inheritance, with high penetrance of cardiomyopathy (dilated or hypertrophic) in hemizygous males. Onset is age-related, with an older age of onset in females than in hemizygous males, and males are more likely to have syndromic involvement (including skeletal myopathy and cognitive impairment).
 **In the majority of cases, the mechanism is loss of function due to decreased gene product**, as a result of frameshift variants, nonsense variants, splice-site variants, and copy-number variants. Individual pathogenic missense variants have been reported. Some missense variants also cause a reduction in protein level, e.g. due to aberrant splicing, but there are pathogenic missense variants which have been shown to affect protein structure rather than reducing protein expression.
 A possible transcript-specific mechanism has been suggested, with variants affecting exon 9b reported as causing a primarily skeletal muscle phenotype, with limited cardiac involvement. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> PRKAG2 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 602743 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> PRKAG2-related cardiomyopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0010946 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe insertion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 26729852; 32259713; 32646570; 32646569; 12015471; 28009297; 11371514 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 20.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 24.11.2021 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _PRKAG2_ pathogenic variants cause disease through **altered gene product sequence**. 
 _PRKAG2_ syndrome is a rare, early-onset autosomal dominant inherited disease, characterized by ventricular pre-excitation, supraventricular arrhythmias, and cardiac hypertrophy (PMID: 26729852; 32259713; 32646570).
 There is a debate about penetrance but there appears to be variable expressivity of the clinical phenotype which may be variant specific (PMID: 32646569; 12015471).

 _PRKAG2_ variants have been recognized mainly in the context of patients with non-sarcomeric familial hypertrophic cardiomyopathy associated with Wolff-Parkinson-White (WPW) syndrome (PMID: 26729852). The _PRKAG2_ gene encodes for the 5’ Adenosine Monophosphate-Activated Protein Kinase (AMPK), specifically for its γ2 regulatory subunit (_PRKAG2_). _PRKAG2_ pathogenic variants are suspected to modify the tri-dimensional structure of AMPK, altering its affinity for AMP and modifying the enzyme activity (PMID: 32259713; 28009297).

 **Nearly all pathogenic variants are missense** (PMID: 32259713; 32646569; 12015471; 28009297). 
 Blair et al. documented a TTA codon insertion in exon 5 in 2001 in a family affected by severe early onset cardiomyopathy and multiple sudden deaths in early adult life (PMID: 11371514). The variant is in a highly conserved region and co-segregated with disease. This has not been reported again on ClinVar.
  Lopez-Sainz A et al reported one frameshift and one intronic variant (the rest missense) in their multi-centre retrospective study of 90 _PRKAG2_ variant carriers in 2020 (PMID: 32646569). However, the significance of truncating variants remains uncertain with insufficient evidence to support loss of function as a mechanism of disease. ClinGen found no evidence to support haploinsufficiency as a mechanism (https://search.clinicalgenome.org/kb/gene-dosage/HGNC:9386). 
 Two commonly reported variants are C.905G&gt;A (p.Arg302Gln) and c.1463A&gt;T (p.Asn488Ile). Lopez-Sainz A et al found they were present in 44% of the patients included in the cohort (PMID: 32646569). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> PTPN11 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 176876 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> PTPN11-related Noonan syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0008104 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 23312968; 21269411; 11992261; 16358218; 14974085; 15521065; 15240615; 18348260; 19760651; 24739123; 25974318; 21533187 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 12.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _PTPN11_-related Noonan syndrome is inherited as an autosomal dominant condition with variable expressivity. Pulmonary stenosis is the most common cardiac manifestation. The penetrance of hypertrophic cardiomyopathy in _PTPN11_-related Noonan syndrome has been reported as 20-30%.
 There is no definite evidence of _PTPN11_ variants causing isolated cardiac disease without other features of Noonan syndrome, but the syndromic phenotype can be subtle and may be underdiagnosed. This suggests that cardiac involvement can be the initial presentation in some cases, and _PTPN11_ should be considered in the differential diagnosis of HCM.
 _PTPN11_ variants are the most common cause of Noonan syndrome, accounting for approximately 50% of cases. The mechanism is likely gain of function due to missense variants (particularly variants disrupting interaction between the N-SH2 and PTP domains), leading to activation of the RAS-MAPK pathway. There are at least two reports of Noonan syndrome due to single-residue in-frame deletions in _PTPN11_.
 There are reports of structural duplications including _PTPN11_ causing Noonan syndrome, but all patients reported to date have had a relatively non-specific phenotype and large duplications encompassing other genes. It is therefore not clear whether gene duplication specifically causes Noonan syndrome.
 Allelic disorders include Noonan syndrome with multiple lentigines (NSML, previously known as LEOPARD syndrome), caused by dominant-negative _PTPN11_ variants, and metachondromatosis, caused by loss-of-function _PTPN11_ variants. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> PTPN11 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 176876 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> PTPN11-related Noonan syndrome with multiple lentigines </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0100082 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 12058348; 12161596; 17697839; 11992261; 16358218; 15121796; 16377799; 33354767 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 12.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> Noonan syndrome with multiple lentigines (NSML, previously known as LEOPARD syndrome) is inherited as an autosomal dominant condition. Missense variants in _PTPN11_ are responsible for the majority of cases. Rare cases have been attributed to variants in RAF1, BRAF, and MAP2K1. There is no definite evidence of _PTPN11_ variants causing isolated cardiac disease without other features of NSML, but the syndromic phenotype can be subtle and may be underdiagnosed. This suggests that cardiac involvement can be the initial presentation in some cases, and _PTPN11_ should be considered in the differential diagnosis of HCM.
 _PTPN11_ variants associated with NSML are clustered in the phosphotyrosine phosphatase (PTP) domain and have been shown to inhibit phosphatase activity. Since null variants have not been associated with the NSML phenotype, it has been suggested that NSML is caused by a dominant-negative effect rather than simple loss of function, possibly due to altered interaction between _PTPN11_ and binding partners.
 Allelic disorders include Noonan syndrome, caused by gain-of-function variants in _PTPN11_, and metachondromatosis, caused by loss-of-function _PTPN11_ variants. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> RAF1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 164760 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> RAF1-related Noonan syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0012690 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 17603482; 17603483; 30762279; 29271604; 24782337; 22786616; 25974318 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 12.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _RAF1_ variants are responsible for ~5% of Noonan syndrome cases. _RAF1_-related Noonan syndrome is inherited as an autosomal dominant condition.
 The mechanism is likely _RAF1_ gain of function, due to missense variants leading to activation of the Ras-MAPK pathway.
  The penetrance of hypertrophic cardiomyopathy is higher in patients with _RAF1_ variants, compared to other forms of Noonan syndrome. There is no definite evidence of _RAF1_ variants causing isolated cardiac disease without other features of Noonan syndrome, but _RAF1_ variants have been identified in multiple patients who were being investigated for HCM, indicating that the syndromic phenotype can be subtle and may be underdiagnosed. Since cardiac involvement can be the initial presentation in some cases, _RAF1_ should be considered in the differential diagnosis of HCM.
 There is evidence for a genotype-phenotype correlation, with variants in the CR2 domain of _RAF1_ more likely to be associated with HCM. There is a further mutational hotspot in the kinase domain of CR3, but pathogenic variants have also been identified outside these hotspots.
 There are reports of patients with chromosomal duplications including _RAF1_, but their phenotype is relatively non-specific and there is no definitive evidence that whole-gene duplications cause Noonan syndrome. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> RIT1 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 609591 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> RIT1-related Noonan syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0014143 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 23791108; 26446362; 27101134; 25959749 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 12.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _RIT1_ variants are responsible for ~5% of Noonan syndrome cases. _RIT1_-related Noonan syndrome is inherited as an autosomal dominant condition.
 The penetrance of hypertrophic cardiomyopathy is higher in patients with _RIT1_ variants, compared to other forms of Noonan syndrome (estimated at 70-75%). There is no definite evidence of _RIT1_ variants causing isolated cardiac disease without other features of Noonan syndrome, but the syndromic phenotype can be subtle and may be underdiagnosed. This suggests that cardiac involvement can be the initial presentation in some cases, and _RIT1_ should be considered in the differential diagnosis of HCM.
 The mechanism is likely _RIT1_ gain of function, due to missense variants leading to activation of the Ras-MAPK pathway. The precise mechanism by which missense variants lead to gain of function remains unclear. There is clustering of variants, particularly in the switch II region. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> TTR </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 176300 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> TTR-related hereditary ATTR amyloidosis </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0019441 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by incomplete penetrance </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; inframe insertion; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 25604431; NBK1194; 28213611; 15185500; 30328212; 15930086; 9191784; 29941560; 32969287 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 11.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 12.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> The mechanism in _TTR_-related hereditary ATTR amyloidosis is due to **altered gene product sequence**.
 **The mechanism appears to be gain-of-function**; pathogenic variants cause either tetramer dissociation or monomer denaturation, which both contribute to the formation of amyloid fibrils in tissue (PMID 25604431; NBK1194).
 Of note, wildtype _TTR_ can also cause an age-related late onset form of amyloidosis where patients almost exclusively present with cardiac involvement.

  _TTR_-related hereditary ATTR  amyloidosis is characterised by **autosomal dominant inheritance with incomplete penetrance**. There are reports of homozygous and compound heterozygous variants. Biallelic variants appear to cause a more severe phenotype (PMID: 28213611; 15185500; 30328212), however one study reported no difference in phenotype between homozygous and heterozygous carriers (PMID: 15930086).

 _TTR_-related hereditary ATTR  amyloidosis includes the phenotypes ATTR  amyloid neuropathy, ATTR  cardiac amyloidosis, ATTR  leptomeningeal/CNS amyloidosis. Phenotype varies considerably; individuals can present with multi-systemic phenotypes including, polyneuropathy, carpal tunnel syndrome, cardiomyopathy, gastrointestinal features, autonomic insufficiency, and renal insufficiency. "The prevalence of HCM and/or RCM in Hereditary ATTR  amyloidosis is high but exact percentages have not been accurately defined due the high prevalence of HCM and RCM (~25%) development in senile cardiac amyloidosis, which occurs due to accumulation of wildtype _TTR_ accumulation in the heart with age." (ClinGen summary)

 More than 100 different variants have been reported in the Hereditary Amyloidosis Registry (http://www.amyloidosismutations.com). 
 **Variants are all missense apart from one inframe deletion of a valine residue in exon 4 (Plasma transthyretin levels in the mutant gene carriers measured by nephelometry were very low) and one 6 nucleotide duplication in exon 3**. The duplication was reported to be associated with a particularly aggressive phenotype and although it did not alter the protein secondary or tertiary structure, it decreased the stability of the _TTR_ monomer and tetramer (PMID: 9191784; 29941560).
 Several variants including _TTR_-V30M, _TTR_-T60A, and _TTR_-V122L are commonly associated with cardiac amyloidosis (PMID: 28739313). There are other genotype phenotype correlations (PMID: 25604431; NBK1194).

The ATTR Val122Ile variant (also reported as pVal142Ile) is a risk allele in patient subpopulations of African American descent. The variant in this population has a high prevalence (estimated 3-4%) and patients are at particular risk for developing cardiac-related ATTR. (PMID: 28213611). 

 A systematic review in 2020 evaluating specific therapies for transthyretin cardiac amyloidosis supported the use of tafamidis (a tetramer stabiliser) and noted that other novel therapeutic targets including transthyretin gene silencers are currently under investigation (PMID: 32969287). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> ABCC9 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 601439 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndrome with hypertrophic cardiomyopathy - no isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> ABCC9-related Cantu Syndrome </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0009406 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30089727; NBK246980; 33529173; 22610116; 31175705; 22608503; 31828977; 34453476 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 11.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> pending </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> **Expert panel review pending.** _ABCC9_ pathogenic variants cause Cantu Syndrome (CS) through **altered gene product sequence**. The mechanism is likely gain of function (NBK246980; PMID: 22610116; 31175705; 30089727).
 Harakalova et al showed that variants in _ABCC9_ reduce the ATP-mediated potassium channel inhibition, resulting in channel opening (PMID: 22610116). Pathophysiologic mechanisms resulting in the clinical manifestations remain poorly understood. 

 Cantu syndrome is characterized by congenital hypertrichosis, distinctive facial features, osteochondrodysplasia and cardiac defects. The cardiac manifestations include patent ductus arteriosus, cardiomegaly, hypertrophy and pericardial effusion. It does not present with isolated LVH. All participants in the Cantu registry who harboured a pathogenic missense variant have at least some features in keeping with Cantu syndrome, even if mild. There is however significant variability in phenotype even amongst family members with the same variant (PMID: 33529173) and no clear genotype phenotype correlation (PMID: 31828977). 

 Inheritance is autosomal dominant characterized by variable expressivity. De novo variants are common, Grange et al found only 16/74 (22%) patients had inherited the _ABCC9_ variant from an affected parent (PMID: 31828977). 

 It is caused by heterozygous missense mutations in _ABCC9_ (or less commonly variants in KCNJ8). A review of 74 patients in the International Registry for Cantu syndrome in 2019 found 28 distinct variants (97% were in _ABCC9_). All were simple missense apart from one insertion deletion involving 2 nucleotides (c.3055_3,056 del ins GA). Missense variants were located throughout the protein, however a majority (68%) clustered around transmembrane domain 2 (TMD2). Variants at p.Arg1154 (identified in 24 subjects, 33%) and p.Arg1116 (identified in 11 subjects, 15%), were most commonly observed (PMID: 31828977). 

 Zhang et al 2021 showed that R1154Q variants in genetically modified mice might result in an effective mixed loss/gain of function phenotype, potentially explaining the diverse disease features in patients with CS. However, due to its extreme rarity, the mechanisms resulting in the clinical manifestations of CS remain to be fully elucidated (PMID: 33529173; 34453476). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> CRYAB </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 123590 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndrome with hypertrophic cardiomyopathy - no isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> CRYAB-related Alpha-related B crystallinpathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0013472 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by age-related onset </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Frameshift variant NMD escaping; stop gained NMD escaping; missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 570292; 14681890; 9731540; 21920752; 28493373; 21337604; 21130652; 32420686 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> pending </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> **Expert panel review pending.** _CRYAB_-related alpha-crystallinopathy is typically inherited as an autosomal dominant condition with age-related penetrance.

 The mechanism is likely **altered gene sequence** of _CRYAB_ due to either (1) missense variants or (2) truncating variants in the 3’ region which are predicted to escape NMD. A dominant-negative mechanism has been suggested.

 Variants in _CRYAB_ have been variably associated with dilated, hypertrophic, and restrictive cardiomyopathy, as well as cases with other systemic features of crystallinopathy (cataract, skeletal myopathy) without cardiac involvement.

 Evidence for dosage sensitivity is limited. Homozygous truncating variants predicted to cause NMD have been associated with autosomal recessive congenital/early-onset cataracts and myopathy with respiratory involvement in early childhood, and the heterozygous parents of probands are reported as being healthy. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> FXN </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 606829 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndrome with hypertrophic cardiomyopathy - no isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> FXN-related Friedreich ataxia </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0100339 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by age-related onset </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Triplet repeat expansion; whole gene deletion; intragenic deletion; splice region; splice acceptor variant NMD triggering; splice donor variant NMD triggering; start lost; frameshift variant NMD triggering; stop gained NMD triggering; missense; inframe insertion; inframe deletion </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 28405347; 10633128; 25566998; 24705334; 26704351; 22409940; 22691228 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> pending </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> **Expert panel review pending.** _FXN_-related Friedreich ataxia is inherited as an autosomal recessive condition. Heterozygous carriers of _FXN_ expansions or other pathogenic variants have not been reported to develop symptoms.

 Homozygous expansion of the GAA triplet repeat in exon 1 of _FXN_ is responsible for the majority of cases (95-98%). There is an inverse correlation between the size of the expansion and age of onset (particularly the size of the shorter allele). Anticipation is not observed as the disorder follows autosomal recessive inheritance.

 The remaining cases are due to compound heterozygosity for a GAA expansion and a sequence variant (truncating, splice-site, or missense) or structural variant (single-/multi-exon or whole-gene deletion). There is evidence that compound heterozygosity is associated with earlier onset of symptoms than homozygosity for the GAA triplet repeat expansion, but cardiomyopathy is more common in patients with a homozygous GAA triplet repeat expansion.

 The mechanism is likely loss of function of _FXN_ due to transcriptional silencing (in the case of the GAA triplet repeat), absent/decreased gene product (truncating variants, structural variants), or missense variants causing loss of function. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> GAA </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 606800 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndrome with hypertrophic cardiomyopathy - no isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> GAA-related Pompe disease </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0009290 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> whole gene deletion; splice region; splice acceptor variant NMD triggering; splice donor variant NMD triggering; start lost; frameshift variant NMD escaping; frameshift variant NMD triggering; stop gained NMD triggering; missense; inframe deletion; 5 prime UTR variant; intron variant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 31254424; 1652892; 8094613; 7981676; 7717400; 9535769; 14643388; 17056254; 19588081; 20882352; 4286143; 1856189; 3108320; 10973860; 21484825; 9668092; 11991748; 1680134; 28185884; 31254424; 31342611 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> pending </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> **Expert panel review pending.** Pompe disease is inherited as an autosomal recessive condition, caused by homozygous or compound heterozygous variants in _GAA_. **The mechanism of disease is loss of function** of acid alpha-glucosidase, resulting in lysosomal glycogen accumulation.

 There are currently over 900 variants in the Pompe variant database (N.B. this includes all known variants, regardless of pathogenicity). Multiple classes of variant have been shown to cause Pompe disease, including loss-of-function variants, missense variants, splice-site variants, 5’ UTR variants, intronic variants, and structural variants.

 There is a genotype-phenotype correlation, with bi-allelic loss of function expected to result in a severe phenotype (infantile-onset Pompe disease), while patients with residual enzymatic activity develop late-onset Pompe disease. Cardiac involvement is a classical feature of infantile-onset Pompe disease, but has also been reported in the late-onset form. </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> MYO6 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 600970 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndrome with hypertrophic cardiomyopathy - no isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> MYO6-related nonsyndromic genetic hearing loss </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0019497 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant, Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal; biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typified by age-related onset </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Decreased gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Stop gained; frameshift; splice acceptor; splice donor; missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30681346; 24105371; 15060111; 12687499; 18348273; 25999546; 26265212; 23485424 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 27.03.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> pending </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> **Expert panel review pending.** There is evidence from multiple reports that loss-of-function and missense variants in _MYO6_ are associated with autosomal dominant progressive hearing loss. Homozygosity or compound heterozygosity is associated with congenital hearing loss, suggesting a dose-dependent mechanism.

 **The evidence for cardiac involvement is limited**. Mohiddin et al. (2004, PMID: 15060111) reported a family with autosomal dominant hearing loss who were initially ascertained due to the proband presenting with hypertrophic cardiomyopathy. Linkage studies implicated the region containing _MYO6_, and a missense variant was identified (p.His246Arg) and segregated with disease (with the exception of one individual who was negative for the variant but described as a “double phenocopy”). QT prolongation was observed in 5 out of 12 carriers of the variant. Evidence of LV hypertrophy was observed in 4 out of 11 carriers who underwent echocardiogram. No specific mechanism of disease causation was suggested.

 Studies in transgenic mouse models have suggested a link with cardiac hypertrophy, but only in biallelic knockout mice.

 In summary, there is a single report linking the p.His246Arg variant in _MYO6_ with a hypertrophic cardiomyopathy phenotype exhibiting incomplete penetrance (Mohiddin et al. 2004, PMID: 15060111). The authors note that they were unable to definitively rule out an alternative cause for the cardiac phenotype in the family. This phenotype has not been observed in other families presenting with _MYO6_-related hearing loss (e.g., Sanggaard et al. 2008, PMID: 18348273; Miyagawa et al. 2015, PMID: 25999546), including patients with homozygous loss-of-function or missense variants (Ahmed et al. 2003, PMID: 12687499). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> SLC25A4 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 103220 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndrome with hypertrophic cardiomyopathy - no isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> SLC25A4-related Mitochondrial disease </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0014175 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal recessive </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Biallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Absent gene product level; altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense; frameshift NMD triggering; splice donor </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> NA </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 30013777; 27693233; 30013777; 16155110; 22187496; 25732997; 23401503 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 11.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> pending </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> **Expert panel review pending.** Pathogenic variants in _SLC25A4_ cause mitochondrial DNA depletion syndrome through **altered gene product sequence or absent gene product level**. 

 _SLC25A4_ encodes AAC1 (also known as ANT1), an ADP/ATP carrier isoform expressed at high levels in the skeletal muscle, heart and brain. “Phenotypically, _SLC25A4_ mutations manifest as progressive external ophthalmoplegia if transmitted in an autosomal dominant fashion and associated with multiple mtDNA deletions or as familial hypertrophic cardiomyopathy and skeletal muscle myopathy if transmitted in an autosomal recessive way and associated with mtDNA depletion” (PMID: 30013777). _SLC25A4_ variants do not present with isolated LVH.

 Null recessive mutations cause a mitochondrial myopathy and cardiomyopathy phenotype that presents in childhood or early adulthood (mitochondrial DNA depletion syndrome 12). It is suggested that transporter dysfunction causes insufficient adenine nucleotide availability for dATP synthesis and consequent imbalanced dNTP pools, leading to mtDNA depletion (PMID: 27693233; 30013777).

 A small number of homozygous and compound heterozygous variants have been described including missense, frameshift and splice variants. There are 4 pathogenic submissions on ClinVar. All are predicted to cause loss of function (PMID: 16155110; 22187496; 25732997). Thompson et al found that 3 previously reported recessive variants resulted in &lt;1% transport activity compared to wild-type (PMID: 27693233).

 Of note, heterozygous variants in _SLC25A4_ have also been described and can cause adult-onset autosomal-dominant progressive external ophthalmoplegia (MIM: 609283) or a distinct severe early-onset mitochondrial disease representing a third distinctive phentotypic group associated with _SLC25A4_ pathogenic variants (PMID: 27693233). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Gene </th>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> BAG3 </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;width: 6cm; "> OMIM gene number </td>
   <td style="text-align:left;"> 603883 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Referral indication </td>
   <td style="text-align:left;"> Hypertrophic Cardiomyopathy (HCM) </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease grouping </td>
   <td style="text-align:left;"> Syndrome with hypertrophic cardiomyopathy - no isolated LVH </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; font-weight: bold;"> Disease name </td>
   <td style="text-align:left;font-weight: bold;"> BAG3-related Myofibrillar myopathy </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> MONDO ID </td>
   <td style="text-align:left;"> 0013061 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Gene disease validity (ClinGen) </td>
   <td style="text-align:left;"> DEFINITIVE </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance </td>
   <td style="text-align:left;"> Autosomal dominant </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Allelic requirement </td>
   <td style="text-align:left;"> Monoallelic autosomal </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Inheritance modifiers </td>
   <td style="text-align:left;"> Typically de novo </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Disease-associated variant consequence </td>
   <td style="text-align:left;"> Altered gene product sequence </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Variant classes reported with evidence of pathogenicity </td>
   <td style="text-align:left;"> Missense </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Restricted repertoire of pathogenic variants </td>
   <td style="text-align:left;"> p.P209L </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> PMIDs </td>
   <td style="text-align:left;"> 19085932; 22734908; 21361913; 20605452; 25728519; 32453099; 23995273 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Curated date </td>
   <td style="text-align:left;"> 11.11.2021 </td>
  </tr>
  <tr>
   <td style="text-align:left;width: 6cm; "> Expert panel review date </td>
   <td style="text-align:left;"> 12.01.2022 </td>
  </tr>
</tbody>
</table><table class="table table-striped table-condensed" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
 <thead>
  <tr>
   <th style="text-align:left;font-weight: bold;font-size: 14px;"> Narrative </th>
  </tr>
 </thead>
<tbody>
  <tr>
   <td style="text-align:left;"> _BAG3_ pathogenic variants cause a rare subtype of myofibrillar myopathy through **altered gene product sequence**.

 Of note, loss of function of _BAG3_ is an established cause of dilated cardiomyopathy. Multiple missense, nonsense, frameshift variants have been described in association with DCM and there is sufficient evidence for haploinsufficiency as a mechanism (https://search.clinicalgenome.org/kb/genes/HGNC:939). 

 In contrast, missense variants are associated with myofibrillar myopathy (MFM). Typical presentation is toe walking and clumsiness with progression to cardiomyopathy and restrictive lung disease. The majority of evidence comes from one recurrent missense variant which causes MFM with prominent cardiac involvement. To our knowledge isolated hypertrophic or restrictive cardiomyopathy without skeletal muscle pathology has not been reported for _BAG3_ related MFM. In 2013, Olive et al (PMID: 23995273) summarised that only 12 patients with _BAG3_ related MFM, all carrying the same p.P209L recurrent de-novo mutation transmitted from a mosaic parent, have been reported to date (PMID: 19085932; 22734908; 21361913; 20605452). 

 Malatesta L et al 2020 found that most patients who carry the p.P209L variant have cardiomyopathy (restrictive or hypertrophic) (14 of 18 patients) (PMID: 32453099).

 Functional studies have revealed that _BAG3_ pathogenic variants impair Z-disc assembly and increase the sensitivity to stress-induced apoptosis (PMID: 25728519; 19085932; 32453099). 

 Selcen et al found that mutant _BAG3_ migrated faster and there was abnormal aggregation of mutant protein in COS cells. The authors hypothesised that this missense change may result in altered folding of _BAG3_ or affect the binding properties of the canonical domains (PMID: 19085932). </td>
  </tr>
</tbody>
</table>
---

<br>
<p style="page-break-before: always;">&nbsp;</p>



