---
title: "Generate_G2P_document"
output:
  pdf_document
---

```{r message=FALSE, warning=FALSE, include=FALSE}
library(tidyverse)
library(readxl)
library(openxlsx)
library(knitr)
library(kableExtra)
```

```{r read_raw_data ,message=FALSE, warning=FALSE, include=FALSE}
# KJosephs onedrive
# https://imperiallondon-my.sharepoint.com/:x:/r/personal/kjosephs_ic_ac_uk/Documents/Kat_Imperial/GenCC/Cardiac_G2P/G2P_panel_spreadsheets/Cardiac_G2P_MASTER_all_info_CURRENT.xlsx?d=wc3dc90132e974e65b0d8a26fd1228742&csf=1&web=1&e=RMXoMU
# https://imperiallondon-my.sharepoint.com/personal/kjosephs_ic_ac_uk/_layouts/15/download.aspx?UniqueId=c3dc9013%2D2e97%2D4e65%2Db0d8%2Da26fd1228742
Cardiac_G2P_MASTER_all_info_CURRENT <- read_excel("Cardiac_G2P_MASTER_all_info_CURRENT.xlsx")

# G2P_submission_template_20211123 <- read_excel("G2P_submission_template_20211123.xlsx") %>% colnames() 
G2P_submission_template_20211123 <- read_excel("G2P_submission_202202.xlsm") %>% colnames() 


```

```{r CG2P_edits,message=FALSE, warning=FALSE, include=FALSE}


CG2P_edit_old <- Cardiac_G2P_MASTER_all_info_CURRENT %>% 
  #filter(Ware_Mechansim_curation=="Yes") %>% 
  mutate(Disease_grouping = str_replace(Disease_grouping, "^\\w{1}", toupper)) %>% 
  mutate(mechanism_curated_PMIDs = str_replace_all(mechanism_curated_PMIDs,":",";")) %>%
  mutate(Expert_date = str_replace_all(Expert_date,"\\.","/")) %>%
  mutate(mechanism_curated_PMIDs = str_replace_all(mechanism_curated_PMIDs,",",";")) %>% 
  mutate(mechanism_curated_PMIDs = str_replace_all(mechanism_curated_PMIDs, fixed(" "), "")) %>% 
  mutate(`Allelic requirement` = str_replace(`Allelic requirement`, "aut", "autosomal")) %>% 
  mutate(`Allelic requirement` = str_replace(`Allelic requirement`, "^\\w{1}", tolower)) %>% 
  mutate(`Allelic requirement` = if_else(is.na(`Allelic requirement`),Inheritance,`Allelic requirement`)) %>% 
  mutate(`Allelic requirement` = case_when(`Allelic requirement` == "AD" ~ "Monoallelic_autosomal",
                                           `Allelic requirement` == "AR" ~ "Biallelic_autosomal",
                                           T ~ `Allelic requirement`)) %>% 
  mutate(Inheritance = case_when(Inheritance == "AD" ~ "Autosomal dominant",
                                 Inheritance == "AR" ~ "Autosomal recessive",
                                 T ~ Inheritance)) %>% 
  mutate(Gene_disease_validity = tolower(Gene_disease_validity)) %>%
  
  mutate(conseq_flag = "") %>% 
  mutate(review = "") %>% 
  
  mutate(Inheritance_AD_Imprinting       = case_when(AD_imprinting      =="T" ~ "With maternal/paternal imprinting",T ~ "")) %>% 
  mutate(Inheritance_AD_sex_lim_exp      = case_when(AD_sex_lim_exp     =="T" ~ "With sex-limited expression",T ~ "")) %>% 
  mutate(Inheritance_AD_gen_anticip      = case_when(AD_gen_anticip     =="T" ~ "With genetic anticipation",T ~ "")) %>% 
  mutate(Inheritance_AR_sex_lim_exp      = case_when(AR_sex_lim_exp     =="T" ~ "With sex-limited expression",T ~ "")) %>% 
  mutate(Inheritance_AR_gen_anticip      = case_when(AR_gen_anticip     =="T" ~ "With genetic anticipation",T ~ "")) %>% 
  mutate(Inheritance_XL_dom              = case_when(XL_dom             =="T" ~ "X-linked dominant",T ~ "")) %>% 
  mutate(Inheritance_XL_dom_gen_anticip  = case_when(XL_dom_gen_anticip =="T" ~ "With genetic anticipation",T ~ "")) %>% 
  mutate(Inheritance_XL_strict_rec       = case_when(XL_strict_rec      =="T" ~ "Strictly recessive",T ~ "")) %>% 
  mutate(Inheritance_XL_prim_rec         = case_when(XL_prim_rec        =="T" ~ "Primarily recessive (with milder female expression",T ~ "")) %>% 
  unite(inheritance_mod,
        Inheritance_AD_Imprinting,Inheritance_AD_sex_lim_exp,Inheritance_AD_gen_anticip,
        Inheritance_AR_sex_lim_exp,Inheritance_AR_gen_anticip,
        Inheritance_XL_dom,Inheritance_XL_dom_gen_anticip,Inheritance_XL_strict_rec,Inheritance_XL_prim_rec,
        sep=",") %>% 
  mutate(inheritance_mod =  gsub('(,)+',',',inheritance_mod)) %>% 
  mutate(inheritance_mod =  if_else(startsWith(inheritance_mod, ',') , str_sub(inheritance_mod, start = 2),inheritance_mod )) %>% 
  mutate(inheritance_mod =  if_else(endsWith(inheritance_mod, ',') , str_sub(inheritance_mod, end = -2),inheritance_mod )) %>% 
  
  
  mutate(cc_mod_requires_heterozygosity = case_when(requires_heterozygosity =="T" ~ "requires heterozygosity",T ~ "")) %>% 
  mutate(cc_mod_de_novo                 = case_when(de_novo                 =="T" ~ "typically de novo",T ~ "")) %>% 
  mutate(cc_mod_mosaic                  = case_when(mosaic                  =="T" ~ "typically mosaic",T ~ "")) %>% 
  mutate(cc_mod_age_related_penetrance  = case_when(age_related_penetrance  =="T" ~ "typified by age related penetrance",T ~ "")) %>% 
  mutate(cc_mod_reduced_penetrance      = case_when(reduced_penetrance      =="T" ~ "typified by reduced penetrance",T ~ "")) %>% 
  mutate(cc_mod_imprinting              = case_when(requires_heterozygosity =="T" ~ "imprinted",T ~ "")) %>% 
  unite(cc_mod,
        cc_mod_requires_heterozygosity,
        cc_mod_de_novo,
        cc_mod_mosaic,
        cc_mod_age_related_penetrance,
        cc_mod_reduced_penetrance,
        cc_mod_imprinting,
        sep=",") %>% 
  mutate(cc_mod =  gsub('(,)+',',',cc_mod)) %>% 
  mutate(cc_mod =  if_else(startsWith(cc_mod, ',') , str_sub(cc_mod, start = 2),cc_mod )) %>% 
  mutate(cc_mod =  if_else(endsWith(cc_mod, ',') , str_sub(cc_mod, end = -2),cc_mod )) %>% 
  
  mutate(csq_conseq_altered_product_structure = case_when(conseq_altered_product_structure =="T" ~ "altered_gene_product_structure",T ~ "")) %>% 
  mutate(csq_conseq_absent_product            = case_when(conseq_absent_product            =="T" ~ "absent_gene_product",T ~ "")) %>% 
  mutate(csq_conseq_increased_product_level   = case_when(conseq_increased_product_level   =="T" ~ "increased_gene_product_level",T ~ "")) %>% 
  mutate(csq_conseq_decreased_product_level   = case_when(conseq_decreased_product_level   =="T" ~ "decreased_gene_product_level",T ~ "")) %>% 
  mutate(csq_conseq_unspecified_change_level  = case_when(conseq_unspecified_change_level  =="T" ~ "uncertain",T ~ "")) %>% 
  mutate(csq_conseq_no_effect                 = case_when(conseq_no_effect                 =="T" ~ "no_effect",T ~ "")) %>% 
  unite(conseq,
        csq_conseq_unspecified_change_level,
        csq_conseq_decreased_product_level,
        csq_conseq_absent_product,
        csq_conseq_increased_product_level,
        csq_conseq_altered_product_structure,
        csq_conseq_no_effect,sep = ",") %>% 
  mutate(conseq =  gsub('(,)+',',',conseq)) %>% 
  mutate(conseq =  if_else(startsWith(conseq, ',') , str_sub(conseq, start = 2),conseq )) %>% 
  mutate(conseq =  if_else(endsWith(conseq, ',') , str_sub(conseq, end = -2),conseq )) %>% 
  

  mutate(splice_region_variant   = case_when(splice_region_variant  =="T" ~ "splice_region_variant",T ~ "")) %>% 
  mutate(splice_acceptor_variant = case_when(splice_acceptor_variant=="T" ~ "splice_acceptor_variant",T ~ "")) %>% 
  mutate(splice_donor_variant    = case_when(splice_donor_variant   =="T" ~ "splice_donor_variant",T ~ "")) %>% 
  mutate(start_lost              = case_when(start_lost             =="T" ~ "start_lost",T ~ "")) %>% 
  mutate(frameshift_variant      = case_when(frameshift_variant     =="T" ~ "frameshift_variant",T ~ "")) %>% 
  mutate(stop_gained             = case_when(stop_gained            =="T" ~ "stop_gained",T ~ "")) %>% 
  mutate(stop_lost               = case_when(stop_lost              =="T" ~ "stop_lost",T ~ "")) %>% 
  mutate(missense_variant        = case_when(missense_variant       =="T" ~ "missense_variant",T ~ "")) %>% 
  mutate(inframe_insertion       = case_when(inframe_insertion      =="T" ~ "inframe_insertion",T ~ "")) %>% 
  mutate(inframe_deletion        = case_when(inframe_deletion       =="T" ~ "inframe_deletion",T ~ "")) %>% 
  mutate(`5_prime_UTR_variant`   = case_when(`5_prime_UTR_variant`  =="T" ~ "5_prime_UTR_variant",T ~ "")) %>% 
  mutate(`3_prime_UTR_variant`   = case_when(`3_prime_UTR_variant`  =="T" ~ "3_prime_UTR_variant",T ~ "")) %>% 
  mutate(synonymous_variant      = case_when(synonymous_variant     =="T" ~ "synonymous_variant",T ~ "")) %>% 
  mutate(intron_variant          = case_when(intron_variant         =="T" ~ "intron_variant",T ~ "")) %>% 
  
  mutate(splice_acceptor_variant_NMD_triggering = case_when(splice_acceptor_variant_NMD_triggering =="T" ~ "splice_acceptor_variant_NMD_triggering",T ~ "")) %>% 
  mutate(splice_donor_variant_NMD_triggering    = case_when(splice_donor_variant_NMD_triggering    =="T" ~ "splice_donor_variant_NMD_triggering",T ~ "")) %>% 
  mutate(frameshift_variant_NMD_triggering      = case_when(frameshift_variant_NMD_triggering      =="T" ~ "frameshift_variant_NMD_triggering",T ~ "")) %>% 
  mutate(stop_gained_NMD_triggering             = case_when(stop_gained_NMD_triggering             =="T" ~ "stop_gained_NMD_triggering",T ~ "")) %>% 
  mutate(splice_acceptor_variant_NMD_escaping   = case_when(splice_acceptor_variant_NMD_escaping   =="T" ~ "splice_acceptor_variant_NMD_escaping",T ~ "")) %>% 
  mutate(splice_donor_variant_NMD_escaping      = case_when(splice_donor_variant_NMD_escaping      =="T" ~ "splice_donor_variant_NMD_escaping",T ~ "")) %>% 
  mutate(frameshift_variant_NMD_escaping        = case_when(frameshift_variant_NMD_escaping        =="T" ~ "frameshift_variant_NMD_escaping",T ~ "")) %>% 
  mutate(stop_gained_NMD_escaping               = case_when(stop_gained_NMD_escaping               =="T" ~ "stop_gained_NMD_escaping",T ~ "")) %>%   

  unite(var_class,
        splice_region_variant,splice_acceptor_variant,splice_donor_variant,start_lost,frameshift_variant,stop_gained,stop_lost,missense_variant,
        inframe_insertion,inframe_deletion,`5_prime_UTR_variant`,`3_prime_UTR_variant`,synonymous_variant,intron_variant, 
        splice_acceptor_variant_NMD_triggering,splice_donor_variant_NMD_triggering,frameshift_variant_NMD_triggering,stop_gained_NMD_triggering,
        splice_acceptor_variant_NMD_escaping,splice_donor_variant_NMD_escaping,frameshift_variant_NMD_escaping,stop_gained_NMD_escaping,
        sep = ",") %>% 
  mutate(var_class =  gsub('(,)+',',',var_class)) %>% 
  mutate(var_class =  if_else(startsWith(var_class, ',') , str_sub(var_class, start = 2),var_class )) %>% 
  mutate(var_class =  if_else(endsWith(var_class, ',') , str_sub(var_class, end = -2),var_class )) %>% 
  
  mutate(Disease_grouping = str_replace_all(Disease_grouping,"_"," ")) %>% 
  mutate(phenotypes = "") %>% 
  mutate(private_comment = "") %>% 
  mutate(public_comment = "") %>% 
  mutate(across(c("mechanism_curated_PMIDs","cc_mod","conseq","var_class","Allelic requirement","Inheritance"),~ if_else(Ware_Mechansim_curation=="No" & !(Gene_disease_validity== "definitive"),"",.)))


```

```{r curated_only,message=FALSE, warning=FALSE, include=FALSE}


CG2P_edit_gene3 <- CG2P_edit_old %>% 
  filter(Ware_Mechansim_curation=="Yes") %>% 
  mutate(Gene_disease_validity = toupper(Gene_disease_validity)) %>% 
  mutate(cc_mod = str_to_sentence(cc_mod)) %>% 
  mutate(`Allelic requirement` = str_to_sentence(`Allelic requirement`)) %>% 
  #mutate(mechanism_narrative = str_remove_all(mechanism_narrative, "\r\n")) %>% 
  select(Gene,
         "OMIM gene number" = Gene_MIM,
         "Referral indication" =  Referral_indication_disease,
         "Disease grouping" = Disease_grouping,
         "Disease name" = Disease,
         "MONDO ID" = Disease_ID,
         "Gene disease validity" = Gene_disease_validity,
         Inheritance,
         "Allelic requirement",
         "Inheritance modifiers" = inheritance_mod,
         "Cross cutting modifiers" = cc_mod,
         "Disease-associated variant consequence" = conseq,
         "Variant classes reported with evidence of pathogenicity" = var_class,
         "PMIDs" = mechanism_curated_PMIDs,
         "Mechanism narrative" = mechanism_narrative)  %>%  
  slice(2:9) %>% 
  mutate(across(everything(),~str_replace_all(.,"_"," "))) %>% 
  mutate(across(everything(),~str_replace_all(.,",",", "))) %>% 
  mutate(PMIDs = str_replace_all(PMIDs,";" , "; ")) 

#kable(CG2P_edit_gene1,)

```

```{r eval=T,message=FALSE, warning=FALSE, include=FALSE}

# italicise gene names
CG2P_genes_list <- CG2P_edit_gene3 %>%  
  select(Gene)  %>% mutate(Gene_italic=paste0("\\\\textit{",Gene,"}"))

# for bold: mutate(Gene_italic=paste0("\\\\textbf{",Gene,"}"))

CG2P_edit_gene3 <-CG2P_edit_gene3 %>% 
  mutate(across(everything(), ~ str_replace_all(.,CG2P_genes_list$Gene,CG2P_genes_list$Gene_italic))) %>% 
  mutate(across(everything(), ~ str_replace_all(.,"%","pct"))) 

```




```{r eval=T,echo=F, message=FALSE, warning=FALSE,results='asis'}

for (i in 1:nrow(CG2P_edit_gene3)) {
  print(escape=T,
    knitr::kable(format="latex",
                 escape = F,
                 booktabs=T,
                 CG2P_edit_gene3[i,-c(length(CG2P_edit_gene3))] %>% 
                   pivot_longer(everything()) %>% 
                   janitor::row_to_names(1)) %>%
      kable_styling(font_size = 8,
                  full_width = T,
                  latex_options = c("striped", "scale_down","condensed")) %>% 
      column_spec(1, width = "4cm") %>%
      row_spec(0,font_size = 10,  bold = T)
    )
  
  print(
    knitr::kable(format="latex",
                 escape = F,
                 booktabs=T,
                 CG2P_edit_gene3[i,c(length(CG2P_edit_gene3))]) %>%
      kable_styling(font_size = 7,
                    full_width = T,
                    latex_options = c("striped", "scale_down","condensed")) %>%
      row_spec(0, font_size = 7, bold = T)
    )

  cat("\n\n\\pagebreak\n")
}
```



```{r CG2P_format ,message=FALSE, warning=FALSE, include=FALSE}
CG2P_edit_excel_old <- CG2P_edit_old %>% 
  select("mechanism curation" = Ware_Mechansim_curation,
         "gene symbol" = Gene,
         "gene mim" = Gene_MIM,
         "disease name" = Disease,
         "disease accession (OMIM or MONDO)" = Disease_ID,
         "confidence category" = Gene_disease_validity,
         "allelic requirement" = `Allelic requirement`,
         "cross cutting modifier" = cc_mod,
         "mutation consequence" = conseq,
         "mutation consequences flag" = conseq_flag,
         "variant classes" = var_class,
         "phenotypes" = phenotypes,
         "organ specificity list" = G2P_organ_specificity_list,
         "pmids" = mechanism_curated_PMIDs,
         "panel" = G2P_panel,
         "private_comment" = private_comment,
         "other disease names" = Disease_grouping,
         "add after review" = review,
         "public comment" = public_comment,
         "referral indication" = Referral_indication_disease,
         "expert review" = Expert_approved,
         "expert review date" = Expert_date
  ) %>% 
  replace(is.na(.),"") %>% 
  mutate(across(private_comment, ~str_replace(.,"NA","")))

```

```{r write_to_G2P_excel_output, message=FALSE, warning=FALSE, include=FALSE,eval=T}

write.xlsx(CG2P_edit_format, "CardiacG2P_format_202204.xlsx",overwrite = T)
# write.xlsx(CG2P_edit_excel_old, "CardiacG2P_format_202303.xlsx",overwrite = T)


```
