
# Overview

Cardiac G2P is an evidence-based dataset for inherited cardiac condition
gene disease pairs. Each entry annotates a disease-associated gene with
information about the gene-disease relationship, including the
inheritance mode and allelic requirement, information pertaining to
disease mechanism (represented as a disease-associated variant
consequence), and known disease-relevant variant classes at a defined
locus.

## How to use

The Cardiac G2P Gene Panel can be downloaded in a csv file format
[here](Cardiac_G2P.csv) This dataset is also available through the
official G2P website <https://www.ebi.ac.uk/gene2phenotype>.

## Citation

*Josephs, K.S., Roberts, A.M., Theotokis, P. et al. Beyond gene-disease
validity: capturing structured data on inheritance, allelic requirement,
disease-relevant variant classes, and disease mechanism for inherited
cardiac conditions. Genome Med 15, 86 (2023).*
<https://doi.org/10.1186/s13073-023-01246-8>

## Brugada Syndrome (BrS)

### Brugada Syndrome

#### **SCN5A** - *BrS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
BrS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome (BrS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Brugada Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Brugada syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Intronic variants with functional evidence:
NM_000335.5(SCN5A):c.3228+551T\>G (p.?)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32850980; 9521325; 20564468; 32533946; 20031634; 17075016; 17442746;
25905440; 11748104; 20129283; 29959160; 33164571; 33131149; 25829473;
30203441; 32893267; 29798782; 39391988
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011001
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SCN5A-related Brugada syndrome is caused by **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
(e.g. decreased expression of cardiac sodium channel, Nav1.5 in the
sarcolemma, expression of non-functional channels, or altered gating
properties leading to a decreased INa sodium current (e.g., delayed
activation or earlier or faster inactivation)) (PMID: 29798782). The
disease mechanism is loss of function. <br/><br/> To date, SCN5A is the
only gene classified as having definitive evidence as a cause of
monogenic Brugada syndrome (BrS) by ClinGen (PMID: 29959160). SCN5A
pathogenic variants are identified in approx. 20- 30% of cases of
European ancestry (PMID: 30139433; PMID 33164571). This contribution may
differ in other populations. <br/><br/> SCN5A-related Brugada syndrome
is characterized by autosomal dominant inheritance with incomplete
penetrance (PMID 9521325; 11748104; 25905440; 20031634; 33164571;
NBK1517). <br/><br/>Hundreds of variants, both truncating and
non-truncating, have been described in association with Brugada
syndrome. Kapplinger et al 2010 (PMID: 20129283) identified 293 distinct
variants in SCN5A: 193 missense, 32 nonsense, 38 frameshift, 21
splice-site, and 9 inframe deletions/insertions. Walsh et al 2020 (PMID:
32893267) found that non truncating variants were highly enriched in
European cases in the SCN5A transmembrane regions.<br/> More recently, a
rare non-coding variant in an SCN5A intronic enhancer region has been
identified in 9 patients with Brugada syndrome in Thailand (3.9% of the
patient cohort). The variant is highly enriched compared to population
matched controls and has been functionally validated. It causes
significantly reduced SCN5A expression and a reduction in
Nav1.5-mediated sodium current density. Authors describe a severe
phenotype associated with this variant, 8/9 (89%) of the patients had
experienced a cardiac arrest. The variant is in the RE5 enhancer region,
3-38580380-A-C (GRCh38) (PMID: 39391988).<br/><br/> Gain of function
variants are associated with Long QT syndrome. Although loss of function
appears to be the accepted mechanism in Brugada syndrome the SCN5A
genotype/phenotype association is still not completely understood.
Uncommonly, a single variant can cause both loss of peak current and
gain of late current and can lead to a mixed phenotype of LQTS and BrS
(PMID: 29806494).

## Catecholaminergic polymorphic ventricular tachycardia (CPVT)

### Atypical CPVT phenotype

#### **CALM1** - *CPVT* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CALM1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114180
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
CPVT
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Catecholaminergic polymorphic ventricular tachycardia (CPVT)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Atypical CPVT phenotype
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CALM1-related Catecholaminergic polymorphic ventricular tachycardia
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typically de novo
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31170290; 26309258; 31983240; 24563457; 23040497; 25557436; 32929985
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0013966
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

The disease mechanism in CALM1-related CPVT is likely a dominant
negative effect caused by **altered gene product sequence** which
reduces the calcium dependent inactivation of the calcium channel
CaV1.2, resulting in increased inward calcium channel current (ICaL) and
repolarization delay. <br/><br/>The three CALM genes encode an identical
149aa protein, calmodulin. The CALM1 gene is located on chromosome 14,
CALM2 on chromosome 2 and CALM3 on chromosome 19. Calmodulin protein is
involved in many calcium-dependent intracellular processes. All three
CALM genes have been classified as Definitive for LQTS and Moderate for
CPVT, certain variants are only associated with one or other phenotype,
whereas others are associated with a mixed or variable phenotypes.
<br/><br/> All variants with evidence of pathogenicity identified in the
CALM genes are missense: at least 35 distinct missense variants have
been identified and reported in the International Calmodulinopathy
Registry including 11 (31%) in CALM1, 16 (46%) in CALM2, and 8 (23%) in
CALM3. The majority of variants across the CALM genes affect amino acid
residues in the EF-hand Ca2+ binding loop III and IV. Some variants have
additional evidence in that the same substitution has been seen in
paralogous CALM genes, or that the same substitution has been seen in a
related phenotype (LQT). Most variants are unique, but 9 were present in
more than one index case and among these, three (p.Asn98Ser,
p.Asp130Gly, and p.Phe142Leu, identified in 10, 5, and 4 families,
respectively.) appear to be recurrent. While p.Asp130Gly and p.Phe142Leu
have always been reported as associated with the LQTS phenotype, the
p.Asn98Ser has phenotypic variability, including LQTS, CPVT, idiopathic
VF and sudden death. The majority of CALM variants are de novo.
<br/><br/> Pathogenic variants in the CALM genes have been associated
with: presentation in infancy or early childhood (up to 5 years); marked
sinus bradycardia or atrioventricular block and QT prolongation; and,
predominantly with CALM1 variants, a mild-to-severe neurological
impairment, including seizures, development delay, motor and/or
cognitive disability. <br/><br/> The phenotype most frequently shown by
patients with CALM variants is LQTS, but some patients display other
phenotypes including CPVT, idiopathic VF and sudden unexplained death.
Some variants have been associated with both LQTS and CPVT, however
Crotti et al (PMID: 31170290) report that ‘despite the relatively small
numbers of cases, a significant association (P = 0.001) was observed
between location of mutation and phenotype (Supplementary material
online, Figure S1). Indeed, a pathogenic variant in EF-hand IV Ca2+
binding loop was found in the majority (17/32, 53%) of CALM-LQTS index
cases but in only one of the nine CALM-CPVTs (11%). Conversely, variants
identified in CALM-CPVT index cases were mostly located either in
EF-hand III (n = 5, 56%), or in the inter-EF hand I-II linker (n = 3,
33%).’ <br/><br/> For the ClinGen CPVT curation, CALM1 scored in the
moderate range based on two reported variants in patients with CPVT or
mixed CPVT/LQT phenotypes (p.Asn54Ile and p.Asn98Ser) and associated
functional data. However, due to the unique characteristics of the
calmodulin genes and their gene-disease associations, it was agreed by
the expert panel to include them in this curation. The reasons were: 1)
all three genes have already been established as disease-causing for an
inherited arrhythmia syndrome (LQTS), 2) there are multiple examples of
patients presenting with a phenotype indistinguishable from CPVT for
each, either de novo or supported by functional evidence and 3) the
genes encode for identical proteins, are all expressed in the heart and
equivalent de novo variants in the three genes have been shown to lead
to similar phenotypes, highlighting the functional equivalence of these
genes.

#### **CALM2** - *CPVT* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CALM2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114182
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
CPVT
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Catecholaminergic polymorphic ventricular tachycardia (CPVT)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Atypical CPVT phenotype
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CALM2-related Catecholaminergic polymorphic ventricular tachycardia
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typically de novo
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31170290; 26309258; 24917665; 31983240; 25557436; 27100291
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0017990
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

The disease mechanism in CALM2-related CPVT is likely a dominant
negative effect caused by **altered gene product sequence** which
reduces the calcium dependent inactivation of the calcium channel
CaV1.2, resulting in increased inward calcium channel current (ICaL) and
repolarization delay.<br/><br/> The three CALM genes encode an identical
149aa protein, calmodulin. The CALM1 gene is located on chromosome 14,
CALM2 on chromosome 2 and CALM3 on chromosome 19. Calmodulin protein is
involved in many calcium-dependent intracellular processes. All three
CALM genes have been classified as Definitive for LQTS and Moderate for
CPVT, certain variants are only associated with one or other phenotype,
whereas others are associated with a mixed or variable
phenotypes.<br/><br/> All variants with evidence of pathogenicity
identified in the CALM genes are missense: at least 35 distinct missense
variants have been identified and reported in the International
Calmodulinopathy Registry including 11 (31%) in CALM1, 16 (46%) in
CALM2, and 8 (23%) in CALM3. The majority of variants across the CALM
genes affect amino acid residues in the EF-hand Ca2+ binding loop III
and IV. Some variants have additional evidence in that the same
substitution has been seen in paralogous CALM genes, or that the same
substitution has been seen in a related phenotype (LQTS). Most variants
are unique, but 9 were present in more than one index case and among
these, three (p.Asn98Ser, p.Asp130Gly, and p.Phe142Leu, identified in
10, 5, and 4 families, respectively.) appear to be recurrent. While
variants p.Asp130Gly and p.Phe142Leu have always been reported as
associated with the LQTS phenotype, the p.Asn98Ser has phenotypic
variability, including LQTS, CPVT, idiopathic VF and sudden death. The
majority of CALM variants are de novo.<br/><br/> Pathogenic variants in
the CALM genes have been associated with: presentation in infancy or
early childhood (up to 5 years); marked sinus bradycardia or
atrioventricular block and QT prolongation; and, predominantly with
CALM1 variants, a mild-to-severe neurological impairment, including
seizures, development delay, motor and/or cognitive
disability.<br/><br/> The phenotype most frequently shown by patients
with CALM variants is LQTS, but some patients display other phenotypes
including CPVT, idiopathic VF and sudden unexplained death. Some
variants have been associated with both LQTS and CPVT, however Crotti et
al (PMID: 31170290) report that ‘despite the relatively small numbers of
cases, a significant association (P = 0.001) was observed between
location of mutation and phenotype (Supplementary material online,
Figure S1). Indeed, a pathogenic variant in EF-hand IV Ca2+ binding loop
was found in the majority (17/32, 53%) of CALM-LQTS index cases but in
only one of the nine CALM-CPVTs (11%). Conversely, variants identified
in CALM-CPVT index cases were mostly located either in EF-hand III (n =
5, 56%), or in the inter-EF hand I-II linker (n = 3, 33%).’<br/><br/>For
the ClinGen CPVT curation,CALM2 scored in the moderate range based on
four reported variants in patients with CPVT or mixed CPVT/LQT
phenotypes (p.Glu46Lys in two de novo cases, p.Asn98Ser and p.Asp132Glu)
and associated functional data. However, due to the unique
characteristics of the calmodulin genes and their gene-disease
associations, it was agreed by the expert panel to include them in this
curation. The reasons were: 1) all three genes have already been
established as disease-causing for an inherited arrhythmia syndrome
(LQTS), 2) there are multiple examples of patients presenting with a
phenotype indistinguishable from CPVT for each, either de novo or
supported by functional evidence and 3) the genes encode for identical
proteins, are all expressed in the heart and equivalent de novo variants
in the three genes have been shown to lead to similar phenotypes,
highlighting the functional equivalence of these genes.

#### **CALM3** - *CPVT* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CALM3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114183
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
CPVT
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Catecholaminergic polymorphic ventricular tachycardia (CPVT)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Atypical CPVT phenotype
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CALM3-related Catecholaminergic polymorphic ventricular tachycardia
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typically de novo
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31170290; 31983240; 27516456
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0017990
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

The disease mechanism in CALM3-related CPVT is likely a dominant
negative effect caused by **altered gene product sequence** which
reduces the calcium dependent inactivation of the calcium channel
CaV1.2, resulting in increased inward calcium channel current (ICaL) and
repolarization delay.<br/><br/> The three CALM genes encode an identical
149aa protein, calmodulin. The CALM1 gene is located on chromosome 14,
CALM2 on chromosome 2 and CALM3 on chromosome 19. Calmodulin protein is
involved in many calcium-dependent intracellular processes. All three
CALM genes have been classified as Definitive for LQTS and Moderate for
CPVT, certain variants are only associated with one or other phenotype,
whereas others are associated with a mixed or variable
phenotypes.<br/><br/> All variants with evidence of pathogenicity
identified in the CALM genes are missense: at least 35 distinct missense
variants have been identified and reported in the International
Calmodulinopathy Registry including 11 (31%) in CALM1, 16 (46%) in
CALM2, and 8 (23%) in CALM3. The majority of variants across the CALM
genes affect amino acid residues in the EF-hand Ca2+ binding loop III
and IV. Some variants have additional evidence in that the same
substitution has been seen in paralogous CALM genes, or that the same
substitution has been seen in a related phenotype (LQTS). Most variants
are unique, but 9 were present in more than one index case and among
these, three (p.Asn98Ser, p.Asp130Gly, and p.Phe142Leu, identified in
10, 5, and 4 families, respectively.) appear to be recurrent. While
variants p.Asp130Gly and p.Phe142Leu have always been reported as
associated with the LQTS phenotype, the p.Asn98Ser has phenotypic
variability, including LQTS, CPVT, idiopathic VF and sudden death. The
majority of CALM variants are de novo.<br/><br/> Pathogenic variants in
the CALM genes have been associated with: presentation in infancy or
early childhood (up to 5 years); marked sinus bradycardia or
atrioventricular block and QT prolongation; and, predominantly with
CALM1 variants, a mild-to-severe neurological impairment, including
seizures, development delay, motor and/or cognitive
disability.<br/><br/> The phenotype most frequently shown by patients
with CALM variants is LQTS, but some patients display other phenotypes
including CPVT, idiopathic VF and sudden unexplained death. Some
variants have been associated with both LQTS and CPVT, however Crotti et
al (PMID: 31170290) report that ‘despite the relatively small numbers of
cases, a significant association (P = 0.001) was observed between
location of mutation and phenotype (Supplementary material online,
Figure S1). Indeed, a pathogenic variant in EF-hand IV Ca2+ binding loop
was found in the majority (17/32, 53%) of CALM-LQTS index cases but in
only one of the nine CALM-CPVTs (11%). Conversely, variants identified
in CALM-CPVT index cases were mostly located either in EF-hand III (n =
5, 56%), or in the inter-EF hand I-II linker (n = 3, 33%).’<br/><br/>For
the ClinGen CPVT curation, CALM3 scored in the moderate range based on
four reported variants in patients with CPVT or mixed CPVT/LQT
phenotypes (p.Glu46Lys in two de novo cases, p.Asn98Ser and p.Asp132Glu)
and associated functional data. However, due to the unique
characteristics of the calmodulin genes and their gene-disease
associations, it was agreed by the expert panel to include them in this
curation. The reasons were: 1) all three genes have already been
established as disease-causing for an inherited arrhythmia syndrome
(LQTS), 2) there are multiple examples of patients presenting with a
phenotype indistinguishable from CPVT for each, either de novo or
supported by functional evidence and 3) the genes encode for identical
proteins, are all expressed in the heart and equivalent de novo variants
in the three genes have been shown to lead to similar phenotypes,
highlighting the functional equivalence of these genes.

#### **TECRL** - *CPVT* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TECRL
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
617242
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
CPVT
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Catecholaminergic polymorphic ventricular tachycardia (CPVT)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Atypical CPVT phenotype
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TECRL-related Catecholaminergic polymorphic ventricular tachycardia
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
absent gene product; altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Exon 2 deletion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; stop_gained_NMD_triggering;
splice_donor_variant_NMD_triggering; exon_loss_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
33367594; 30790670; 32173957; 27861123
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0013529
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

**Absent gene product level and altered gene product sequence** of TECRL
is associated with CPVT. The disease mechanism appears to be loss of
function.<br/> TECRL is rarely associated with CPVT, but several reports
have described biallelic loss of function variants in cases. These
include both homozygous and compound heterozygous inheritance, with a
variety of variant types described, splice donor variant, stop gained,
exon deletion (exon 2), missense variants (including homozygous
p.Arg196Gln detected in 2 patients with exome sequencing), and a large
duplication encompassing all of the TECRL gene with an uncertain
consequence.<br/><br/> These cases presented with phenotypic features
typical of CPVT, including exercise and emotion induced syncope and
cardiac arrest and ventricular arrhythmias during exercise testing. A
mild prolonged QT interval was observed in several cases, especially
after stimulation by epinephrine or exercise, although overall the
phenotypes are much more typical of CPVT than LQTS.

#### **TRDN** - *CPVT* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TRDN
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
603283
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
CPVT
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Catecholaminergic polymorphic ventricular tachycardia (CPVT)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Atypical CPVT phenotype
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TRDN-related Catecholaminergic polymorphic ventricular tachycardia
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
absent gene product; altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Exon 2 deletion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; stop_gained_NMD_triggering;
frameshift_variant_NMD_triggering; exon_loss_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25922419; 26768964; 22422768; 30479949; 26200674
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0014191
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

**Absent gene product level and altered gene product sequence** of TRDN
is associated with CPVT. The disease mechanism appears to be loss of
function.<br/> TRDN is rarely associated with CPVT, but several reports
have described biallelic loss of function variants in cases. These
include both homozygous and compound heterozygous variants, with a
variety of variant types described - stop gained, frameshift, exon
deletion (exon 2), intronic variants (with proven effect on splicing)
and missense variants (proven to lead to degraded protein).<br/>
Biallelic loss of function variants in TRDN have also been associated
with LQTS with an atypical presentation (ClinGen: strong evidence).
This, ‘triadin knockout syndrome’ can lead to variable phenotypes.

### Classic CPVT phenotype

#### **CASQ2** - *CPVT* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CASQ2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114251
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
CPVT
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Catecholaminergic polymorphic ventricular tachycardia (CPVT)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Classic CPVT phenotype
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CASQ2-related Catecholaminergic polymorphic ventricular tachycardia
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; missense_variant; stop_gained_NMD_triggering;
splice_donor_variant_NMD_triggering; frameshift_variant_NMD_triggering;
splice_acceptor_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16908766; 32693635; 12386154; 15176429; 21618644; 27157848
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012762
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

CASQ2 is an established gene for autosomal recessive CPVT (the second
most common genetic cause of the disease, responsible for up to 5% of
cases).<br/> Recent reports have also associated monoallelic or
heterozygous CASQ2 variants with CPVT. Although the evidence for this
association is not definitive, available data suggests that disease is
likely due to **decreased gene product level or altered gene product
sequence**. The main evidence for autosomal dominant CASQ2 association
comes from an international multi-centre study describing CPVT patients
with CASQ2 variants (Ng et al, 2020, PMID: 32693635). This study
included 12 probands with heterozygous variants in CASQ2, as well as an
assessment of heterozygous relatives of probands with
homozygous/compound heterozygous CASQ2 variants (8/37 of these
heterozygous relatives had a positive CPVT phenotype). While this study
provides a substantive body of evidence to support autosomal dominant
CASQ2 association with CPVT, the data should be cautiously interpreted.
The multi-centre nature of the study precluded standardised phenotyping
of the probands and relatives and therefore not every phenotype-positive
individual may have a definitive diagnosis of CPVT. Additionally,
several of the variants described have a gnomAD population minor allele
frequency that is incompatible with being a penetrant autosomal dominant
variant for a disease with the prevalence of CPVT (\>1x10-5). The
variants also included presumed nonsense mediated decay-escaping
C-terminal truncating variants and a splice region variant without a
proven effect on splicing. In contrast to 97% penetrance for
homozygous/compound heterozygous individuals, penetrance for
heterozygotes was only 33% although it is expected that the population
level penetrance of CASQ2 heterozygous variants will be lower
still.<br/> Further support for the pathogenicity of heterozygous CASQ2
variants comes from a study that described the heterozygous p.Lys180Arg
variant segregating with disease in a family (the published LOD score
was 3.0 although there were only five meioses between genotype and
phenotype positive individuals) (Gray et al, 2016, PMID: 27157848).

#### **CASQ2** - *CPVT* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CASQ2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114251
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
CPVT
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Catecholaminergic polymorphic ventricular tachycardia (CPVT)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Classic CPVT phenotype
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CASQ2-related Catecholaminergic polymorphic ventricular tachycardia
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
absent gene product; altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; missense_variant; stop_gained_NMD_triggering;
splice_donor_variant_NMD_triggering; frameshift_variant_NMD_triggering;
splice_acceptor_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16908766; 32693635; 12386154; 15176429; 21618644; 27157848
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012762
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

CASQ2 is an established gene for autosomal recessive CPVT (the second
most common genetic cause of the disease, responsible for up to 5% of
cases); disease is due to **absent gene product level or altered gene
product sequence**. For the ClinGen curation, the maximum points were
achieved with only a small subset of initial genetic and experimental
reports leading to a definitive classification. <br/><br/>Biallelic
loss-of-function variants in CASQ2 (both homozygous and compound
heterozygous) have been reported in numerous CPVT probands, including
frameshift, nonsense and splice donor/acceptor variants, as well as
other splice region variants with verified effects on splicing, and
missense variants with verified loss-of-function effects.

#### **RYR2** - *CPVT* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
RYR2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
180902
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
CPVT
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Catecholaminergic polymorphic ventricular tachycardia (CPVT)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Classic CPVT phenotype
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
RYR2-related Catecholaminergic polymorphic ventricular tachycardia
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Exon 3 deletion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; exon_loss_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31112425; 30696458; 11208676; 24394973; 17081562; 12093772; 17875969;
29453246; 19216760; 23479668; 26018045
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0017990
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.01.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

RYR2 is associated with CPVT with autosomal dominant inheritance. The
allelic requirement for pathogenicity is monoallelic autosomal, and the
disease is typified by incomplete penetrance. Disease is due to an
**altered gene product sequence**. <br/><br/> The first reports of RYR2
variants in CPVT included numerous examples of de novo inheritance
(Priori et al, 2001, PMID: 11208676; Priori et al, 2002, PMID:
12093772), although RYR2 missense variants often occur through familial
autosomal dominant inheritance. <br/><br/> The predicted functional
consequence of RYR2 pathogenic variants is altered gene product
sequence. The majority of causative RYR2 variants in CPVT patients are
heterozygous missense variants which are detected in up to 60% of cases
with a definitive CPVT diagnosis (Kapplinger et al, 2018, PMID:
29453246) and act through a gain-of-function mechanism. Pathogenic
variants in CPVT cases tend to be clustered in a number of regions in
the RYR2 gene/protein:<br/> - Original exon hotspots: 3-15, 44-50, 83-90
and 93-105 (George et al, 2007, PMID: 17081562).<br/> - Updated exon
hotspots: 3, 8, 14, 43, 47-49, 81, 83, 88-90, 93, 95, 97-101, 103, 105
(Kapplinger et al, 2018, PMID: 29453246).<br/> - Hotspots based on
unsupervised clustering algorithm (not restricted to exon boundaries):
amino acid residues 2138-2538, 3935-4196 and 4721-4959 (Walsh et al,
2019, PMID: 30696458). <br/><br/> Rare cases of whole exon deletions of
RYR2 exon 3 have been described that can lead to a CPVT and/or left
ventricular non-compaction phenotype (Bhuiyan et al, 2007, PMID:
17875969; Marjamaa et al, 2009, PMID: 19216760; Szentpali et al, 2013,
PMID: 23479668; Onho et al, 2014, PMID: 24394973; Campbell et al, 2015,
PMID: 26018045; Mazzarotto et al, 2021, PMID: 33500567).

## Long QT Syndrome (LQTS)

### Familial Long QT Syndrome

#### **KCNH2** - *LQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNH2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
152427
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial Long QT Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
KCNH2-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering;
exon_loss_variant; duplication
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21185501; 18774102; 22995932; 10841244; 27262388; 29504689; 19841300;
31983240; 24932360; 15840476; 25417810; 26320108; 19926013; 21185499;
29650123; 32893267; 27761161; 10491368; 12621127; 21787999; 19716085;
9927399
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0013367
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
14.10.2020
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

KCNH2-related LQTS is due to **decreased gene product level or altered
gene product sequence**. The disease mechanism is **loss of function**
due to a variety of mechanisms including disruption of synthesis of
channel subunits, reduction in intracellular transport or trafficking,
defects in ion permeation or channel gating.<br/><br/> Both
haploinsufficiency and a dominant negative effect are proposed
mechanisms causing loss of function of KCNH2. About 60% of LQT2
mutations are missense variants, the remaining 40% are nonsense,
frameshift, insertions, deletions, duplications (intragenic tandem
duplication), or involve a splice site. <br/><br/> KCNH2 variants
account for approximately 30% of LQTS cases (PMID: 24932360).<br/> Walsh
et al showed that non truncating variants in KCNH2 are enriched in ion
channel transmembrane regions and specific N-terminus and C-terminus
domains and have \>95% probability of pathogenicity (PMID:
32893267).<br/> Shimizu et al report correlations between specific
variant types and location and clinical phenotype (PMID: 19926013).
Patients with missense variants in the transmembrane pore region have
significantly higher cardiac event rates than those with missense
variants in either N-terminus, transmembrane non-pore, or C-terminus
regions. <br/><br/> It has been noted that approximately 10% of genotype
positive LQT patients have more than 1 pathogenic variant in \>=1 gene.
Biallelic pathogenic variants or digenic pathogenic variants appear to
be generally associated with a more severe phenotype with longer QTc
interval and a higher incidence of cardiac events.

#### **KCNQ1** - *LQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNQ1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
607542
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial Long QT Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
KCNQ1-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering;
exon_loss_variant; tandem_duplication
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27041150; 23591039; 9020846; 27868350; 23392653; 31983240; 11226272;
25187895
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0100316
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
01.11.2020
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in KCNQ1 cause LQTS due to **decreased gene product
level or altered gene product sequence**. This leads to loss of function
due to a variety of mechanisms including defects in ion permeation,
channel gating and trafficking.<br/> KCNQ1 variants account for approx.
30%-45% of LQTS cases (PMID: 22456477; 21787999; 27262388;
29504689).<br/><br/> KCNQ1-related LQTS is typically inherited as an
autosomal dominant trait characterised by incomplete penetrance. Rarely
biallelic KCNQ1 variants can cause autosomal recessive LQTS or Jervell
and Lange-Nielsen syndrome (JLNS) (severe cardiac phenotype and
sensorineural hearing loss). JLNS results when there is complete absence
of IKs (slowly activating delayed rectifier potassium channel/current)
(PMID: 23591039).<br/><br/> It has been noted that approximately 10% of
genotype positive LQT patients have more than 1 pathogenic variant in
\>=1 LQTS-related gene (PMID: 15840476; 19716085). Biallelic disease
appears to be more penetrant with a more severe phenotype.<br/><br/>
Missense variants are responsible for the majority of KCNQ1-related LQTS
cases although truncating variants including nonsense, frameshift,
splice site and structural variants (multi exon deletions and a multi
exon duplication (inframe tandem repeat of exons 3-6, leading to
duplication of the second to fifth transmembrane domains of the
channel)) are also reported (PMID: 19716085; 18774102; 25174857).
<br/><br/> ClinGen have concluded there is sufficient evidence for
haploinsufficiency. Variants conferring a dominant negative effect have
also been described. Moss et al (PMID: 17470695) reported that
dominant-negative variants are associated with a more severe phenotype
than variants conferring haploinsufficiency.<br/><br/> Phenotypes can
differ according to regionality with some regions associated with more
malignant phenotypes e.g. C-loop variants are associated with higher
risk of sudden cardiac death (PMID: 22456477). In addition, there are
gene regions where there is a high confidence for pathogenicity
e.g. transmembrane regions and C-terminus domains (PMID: 32893267;
19841300).

#### **KCNQ1** - *LQTS* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNQ1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
607542
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial Long QT Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
KCNQ1-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_triggering; frameshift_variant_NMD_triggering;
exon_loss_variant; tandem_duplication
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27041150; 23591039; 9020846; 27868350; 23392653; 31983240; 11226272;
25187895
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0100316
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
01.11.2020
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in KCNQ1 cause LQTS due to **decreased gene product
level or altered gene product sequence**. This leads to loss of function
due to a variety of mechanisms including defects in ion permeation,
channel gating and trafficking.<br/> KCNQ1 variants account for approx.
30%-45% of LQTS cases (PMID: 22456477; 21787999; 27262388;
29504689).<br/><br/> KCNQ1-related LQTS is typically inherited as an
autosomal dominant trait characterised by incomplete penetrance. Rarely
biallelic KCNQ1 variants can cause autosomal recessive LQTS or Jervell
and Lange-Nielsen syndrome (JLNS) (severe cardiac phenotype and
sensorineural hearing loss). JLNS results when there is complete absence
of IKs (slowly activating delayed rectifier potassium channel/current)
(PMID: 23591039).<br/><br/> It has been noted that approximately 10% of
genotype positive LQT patients have more than 1 pathogenic variant in
\>=1 LQTS-related gene (PMID: 15840476; 19716085). Biallelic disease
appears to be more penetrant with a more severe phenotype.<br/><br/>
Missense variants are responsible for the majority of KCNQ1-related LQTS
cases although truncating variants including nonsense, frameshift,
splice site and structural variants (multi exon deletions and a multi
exon duplication (inframe tandem repeat of exons 3-6, leading to
duplication of the second to fifth transmembrane domains of the
channel)) are also reported (PMID: 19716085; 18774102; 25174857).
<br/><br/> ClinGen have concluded there is sufficient evidence for
haploinsufficiency. Variants conferring a dominant negative effect have
also been described. Moss et al (PMID: 17470695) reported that
dominant-negative variants are associated with a more severe phenotype
than variants conferring haploinsufficiency.<br/><br/> Phenotypes can
differ according to regionality with some regions associated with more
malignant phenotypes e.g. C-loop variants are associated with higher
risk of sudden cardiac death (PMID: 22456477). In addition, there are
gene regions where there is a high confidence for pathogenicity
e.g. transmembrane regions and C-terminus domains (PMID: 32893267;
19841300).

#### **SCN5A** - *LQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial Long QT Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion; inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21185501; 15840476; 27566755; 26320108; 22995932; 7889574; 29650123;
31983240; 32893267; 11463728; 29798782; 21787999; 9927399; 29806494;
19716085
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011377
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
14.10.2020
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

**Altered gene product sequence** of SCN5A causes long QT syndrome. The
likely disease mechanism is gain of function.<br/> Over 200 pathogenic
missense variants and inframe deletions or insertions have been
reported. The sodium current mediated by Nav1.5 consists of peak and
late components (INa-P and INa-L). It is thought that gain-of-function
SCN5A pathogenic variants lead to enhanced INa-P and INa-L, which can
trigger life-threating arrhythmias. <br/> SCN5A-related LQTS accounts
for 5-10% of LQTS cases (PMID: 27566755).<br/> Rare missense variants
are estimated to occur in around 2% of healthy White and 5% of healthy
non-White subjects so collectively missense variants are not rare in the
healthy population.<br/><br/> SCN5A-related LQTS can present with
specific features: patients may have marked resting bradycardia, QT
interval prolongation more pronounced during slow heart rate (which
might explain why arrhythmic events occur more frequently at rest), a
first cardiac event that is lethal, and onset after puberty (PMID:
29798782).<br/><br/> It has been noted that approximately 10% of
genotype positive LQT patients have more than 1 mutation in \>=1 gene.
Biallelic pathogenic variants or digenic pathogenic variants appear to
be generally associated with a more severe phenotype with longer QTc
interval and a higher incidence of cardiac events.<br/><br/> Note: loss
of function variants in SCN5A are associated with Brugada syndrome and
individual variants can have hybrid loss of function and gain of
function effects causing a mixed phenotype.

#### **CACNA1C** - *LQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CACNA1C
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114205
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial Long QT Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CACNA1C-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

### Long QT syndrome with an atypical presentation

#### **CALM1** - *LQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CALM1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114180
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT syndrome with an atypical presentation
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CALM1-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typically de novo
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31170290; 31983240; 23388215; 26969752
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0014548
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
14.10.2020
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

CALM1-related LQTS is caused by an **altered gene product sequence**.
Reduction in the calcium dependent inactivation of the calcium channel
CaV1.2 results in increased inward calcium channel current (ICaL) and
repolarization delay. The disease mechanism is likely a dominant
negative effect.<br/><br/> The three CALM genes encode an identical
149aa protein, calmodulin. The CALM1 gene is located on chromosome 14,
CALM2 on chromosome 2 and CALM3 on chromosome 19. Calmodulin protein is
involved in many calcium-dependent intracellular processes. All three
CALM genes have been classified as Definitive for LQTS and Moderate for
CPVT, certain variants are only associated with one or other phenotype,
whereas others are associated with a mixed or variable phenotypes.
<br/><br/> All variants with evidence of pathogenicity identified in the
CALM genes are missense: at least 35 distinct missense variants have
been identified and reported in the International Calmodulinopathy
Registry including 11 (31%) in CALM1, 16 (46%) in CALM2, and 8 (23%) in
CALM3. The majority of variants across the CALM genes affect amino acid
residues in the EF-hand Ca2+ binding loop III and IV. Some variants have
additional evidence in that the same substitution has been seen in
paralogous CALM genes, or that the same substitution has been seen in a
related phenotype (LQTS). Most variants are unique, but 9 were present
in more than one index case and among these, three (p.Asn98Ser,
p.Asp130Gly, and p.Phe142Leu, identified in 10, 5, and 4 families,
respectively.) appear to be recurrent. While variants p.Asp130Gly and
p.Phe142Leu have always been reported as associated with the LQTS
phenotype, the variant p.Asn98Ser has phenotypic variability, including
LQTS, CPVT, idiopathic VF and sudden death. The majority of CALM
variants are de novo.<br/><br/> Pathogenic variants in the CALM genes
have been associated with: presentation in infancy or early childhood
(up to 5 years); marked sinus bradycardia or atrioventricular block and
QT prolongation; and, predominantly with CALM1 variants, a
mild-to-severe neurological impairment, including seizures, development
delay, motor and/or cognitive disability.<br/><br/> The phenotype most
frequently shown by patients with CALM variants is LQTS, but some
patients display other phenotypes including CPVT, idiopathic VF and
sudden unexplained death. Some variants have been associated with both
LQTS and CPVT, however Crotti et al (PMID: 31170290) report that
‘despite the relatively small numbers of cases, a significant
association (P = 0.001) was observed between location of mutation and
phenotype (Supplementary material online, Figure S1). Indeed, a
pathogenic variant in EF-hand IV Ca2+ binding loop was found in the
majority (17/32, 53%) of CALM-LQTS index cases but in only one of the
nine CALM-CPVTs (11%). Conversely, variants identified in CALM-CPVT
index cases were mostly located either in EF-hand III (n = 5, 56%), or
in the inter-EF hand I-II linker (n = 3, 33%).’

#### **CALM2** - *LQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CALM2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114182
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT syndrome with an atypical presentation
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CALM2-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typically de novo
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31170290; 27765793; 31983240; 23388215; 26969752
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0014550
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
14.10.2020
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

CALM2-related LQTS is caused by an**altered gene product sequence**
which reduces the calcium dependent inactivation of the calcium channel
CaV1.2, resulting in increased inward calcium channel current (ICaL) and
repolarization delay. The disease mechanism is likely a dominant
negative effect.<br/><br/> The three CALM genes encode an identical
149aa protein, calmodulin. The CALM1 gene is located on chromosome 14,
CALM2 on chromosome 2 and CALM3 on chromosome 19. Calmodulin protein is
involved in many calcium-dependent intracellular processes. All three
CALM genes have been classified as Definitive for LQTS and Moderate for
CPVT, certain variants are only associated with one or other phenotype,
whereas others are associated with a mixed or variable phenotypes.
<br/><br/> All variants with evidence of pathogenicity identified in the
CALM genes are missense: at least 35 distinct missense variants have
been identified and reported in the International Calmodulinopathy
Registry including 11 (31%) in CALM1, 16 (46%) in CALM2, and 8 (23%) in
CALM3. The majority of variants across the CALM genes affect amino acid
residues in the EF-hand Ca2+ binding loop III and IV. Some variants have
additional evidence in that the same substitution has been seen in
paralogous CALM genes, or that the same substitution has been seen in a
related phenotype (LQTS). Most variants are unique, but 9 were present
in more than one index case and among these, three (p.Asn98Ser,
p.Asp130Gly, and p.Phe142Leu, identified in 10, 5, and 4 families,
respectively.) appear to be recurrent. While variants p.Asp130Gly and
p.Phe142Leu have always been reported as associated with the LQTS
phenotype, the variant p.Asn98Ser has phenotypic variability, including
LQTS, CPVT, idiopathic VF and sudden death. The majority of CALM
variants are de novo.<br/><br/> Pathogenic variants in the CALM genes
have been associated with: presentation in infancy or early childhood
(up to 5 years); marked sinus bradycardia or atrioventricular block and
QT prolongation; and, predominantly with CALM1 variants, a
mild-to-severe neurological impairment, including seizures, development
delay, motor and/or cognitive disability.<br/><br/> The phenotype most
frequently shown by patients with CALM variants is LQTS, but some
patients display other phenotypes including CPVT, idiopathic VF and
sudden unexplained death. Some variants have been associated with both
LQTS and CPVT, however Crotti et al (PMID: 31170290) report that
‘despite the relatively small numbers of cases, a significant
association (P = 0.001) was observed between location of mutation and
phenotype (Supplementary material online, Figure S1). Indeed, a
pathogenic variant in EF-hand IV Ca2+ binding loop was found in the
majority (17/32, 53%) of CALM-LQTS index cases but in only one of the
nine CALM-CPVTs (11%). Conversely, variants identified in CALM-CPVT
index cases were mostly located either in EF-hand III (n = 5, 56%), or
in the inter-EF hand I-II linker (n = 3, 33%).’

#### **CALM3** - *LQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CALM3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114183
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT syndrome with an atypical presentation
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CALM3-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typically de novo
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31170290; 31983240; 23388215; 26969752
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0019171
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
14.10.2020
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

For LQTS, CALM3 variants lead to an **altered gene product sequence**.
The disease mechanism is likely a dominant negative effect which reduces
the calcium dependent inactivation of the calcium channel CaV1.2,
resulting in increased inward calcium channel current (ICaL) and
repolarization delay.<br/><br/> The three CALM genes encode an identical
149aa protein, calmodulin. The CALM1 gene is located on chromosome 14,
CALM2 on chromosome 2 and CALM3 on chromosome 19. Calmodulin protein is
involved in many calcium-dependent intracellular processes. All three
CALM genes have been classified as Definitive for LQTS and Moderate for
CPVT, certain variants are only associated with one or other phenotype,
whereas others are associated with a mixed or variable phenotypes.
<br/><br/> All variants with evidence of pathogenicity identified in the
CALM genes are missense: at least 35 distinct missense variants have
been identified and reported in the International Calmodulinopathy
Registry including 11 (31%) in CALM1, 16 (46%) in CALM2, and 8 (23%) in
CALM3. The majority of variants across the CALM genes affect amino acid
residues in the EF-hand Ca2+ binding loop III and IV. Some variants have
additional evidence in that the same substitution has been seen in
paralogous CALM genes, or that the same substitution has been seen in a
related phenotype (LQTS). Most variants are unique, but 9 were present
in more than one index case and among these, three (p.Asn98Ser,
p.Asp130Gly, and p.Phe142Leu, identified in 10, 5, and 4 families,
respectively.) appear to be recurrent. While variants p.Asp130Gly and
p.Phe142Leu have always been reported as associated with the LQTS
phenotype, the variant p.Asn98Ser has phenotypic variability, including
LQTS, CPVT, idiopathic VF and sudden death. The majority of CALM
variants are de novo.<br/><br/> Pathogenic variants in the CALM genes
have been associated with: presentation in infancy or early childhood
(up to 5 years); marked sinus bradycardia or atrioventricular block and
QT prolongation; and, predominantly with CALM1 variants, a
mild-to-severe neurological impairment, including seizures, development
delay, motor and/or cognitive disability.<br/><br/> The phenotype most
frequently shown by patients with CALM variants is LQTS, but some
patients display other phenotypes including CPVT, idiopathic VF and
sudden unexplained death. Some variants have been associated with both
LQTS and CPVT, however Crotti et al (PMID: 31170290) report that
‘despite the relatively small numbers of cases, a significant
association (P = 0.001) was observed between location of mutation and
phenotype (Supplementary material online, Figure S1). Indeed, a
pathogenic variant in EF-hand IV Ca2+ binding loop was found in the
majority (17/32, 53%) of CALM-LQTS index cases but in only one of the
nine CALM-CPVTs (11%). Conversely, variants identified in CALM-CPVT
index cases were mostly located either in EF-hand III (n = 5, 56%), or
in the inter-EF hand I-II linker (n = 3, 33%).’

#### **TRDN** - *LQTS* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TRDN
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
603283
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT syndrome with an atypical presentation
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TRDN-related Long QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
STRONG
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
absent gene product
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
30649896; 22422768; 32402482; 25922419; 26768964; 31983240; 30479949;
31437535; 26200674
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0019171
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
04.11..2020
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

TRDN-related LQTS is due to an **absent gene product level**. The
disease mechanism is loss of function (LoF).<br/> **TRDN is rarely
associated with LQTS with an atypical presentation**. Following
identification of a homozygous frameshift variant in TRDN, 4 other
patients with homozygous or compound heterozygous frameshift variants
were identified in a cohort of 33 unrelated genotype-negative LQTS
patients.<br/> The atypical LQTS phenotype described: extensive T-wave
inversions in precordial leads V1 through V4, persistent or transient QT
prolongation and severe disease expression of exercise-induced cardiac
arrest in early childhood (less than or equal to 3 years of
age).<br/><br/> Biallelic loss of function variants in TRDN have also
been associated with catecholaminergic polymorphic ventricular
tachycardia (CPVT). As patients can have overlapping phenotypes, there
is a term ‘triadin knockout syndrome’. The Triadin Knock Out Syndrome
(TKOS) registry had 21 patients in 2019. All were homozygous or compound
heterozygous for TRDN variants. The majority were frameshift or
nonsense. There were a small number of missense or splice altering
variants.<br/> TRDN is expressed in both cardiac and skeletal muscle and
undergoes extensive alternative splicing to produce several isoforms.
Only biallelic LoF variants affecting cardiac isoforms should be
considered as pathogenic for LQTS.

### Syndromic QT prolongation and cardiac arrhythmias

#### **CACNA1C** - *LQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CACNA1C
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114205
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndromic QT prolongation and cardiac arrhythmias
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CACNA1C-related Timothy syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typically de novo
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations; activating
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
A recurrent, de novo missense_variant causing Classic Timothy Syndrome
has been described: NM_001167623.2(CACNA1C):c.1216G\>C (p.Gly406Arg)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
15863612; 15454078; 22106044; 26253506; 4728418; 31983240; NBK1403;
30681346; 33797204; 16360093
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0010979
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
11.11.2020
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

CACNA1C-related Timothy syndrome is caused by variants leading to
**altered gene product sequence**. A recurrent, de novo, missense
variant in CACNA1C was described in 13 Timothy syndrome patients,
p.Gly406Arg in exon 8A (PMID 15454078; 15863612). The disease mechanism
appears to be gain-of-function through failed channel
inactivation.<br/><br/> Classic Timothy syndrome (TS1) is a very rare
multisystem disorder characterized by marked QT prolongation,
syndactyly, immune deficiency, seizures, congenital heart defects,
cognitive abnormalities, learning difficulties, and intermittent
hypoglycaemia (PMID: 28211989). <br/><br/> CACNA1C has a complex genomic
structure that undergoes extensive alternative splicing. Splawski et al
identified 2 patients with **de novo missense variants in exon 8** of an
alternate splice form (p.Gly406Arg, analogous to the exon 8a variant,
and p.Gly402Ser). This splice form represents 80% of all cardiac mRNAs.
The patients were described as having atypical Timothy syndrome (TS2),
presenting with a more severe cardiac phenotype and without syndactyly
(PMID: 15863612; 25633834).<br/> Other missense variants in CACNA1C have
been reported in association with isolated LQTS (PMID: 26253506;
25633834; 24728418). However as of 2020, the ClinGen Cardiovascular
Domain Working Group have classified the strength of evidence supporting
an association between CACNA1C and LQTS as moderate (PMID: 31983240).

#### **KCNE1** - *LQTS* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNE1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
176261
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndromic QT prolongation and cardiac arrhythmias
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
KCNE1-related Jervell and Lange-Nielsen syndrome 2
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
30461122; 9354783; NBK1405; 31941373; 9445165; 9328483; 10973849;
19716085; 16461811
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012871
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
01.08.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
30.08.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

KCNE1-related JLNS is due to **altered gene product sequence**.
Inheritance is autosomal recessive. Homozygous or compound heterozygous
KCNE1 variants cause JLNS through loss of function (PMID: 9354783). The
majority of JLNS patients have biallelic KCNQ1 variants (JLNS1), only a
minority have JLNS2 caused by biallelic KCNE1 variants (PMID:
16461811).<br/> The KCNE1 protein (previous names LQT5, mink, IsK)
functions as a regulatory subunit of KCNQ1. It has 7 annotated exons but
only one is protein coding (PMID: 30461122).<br/><br/> A small number of
variants have been reported, mainly missense and inframe indels. In more
recent studies, 4 nonsense variants have also been reported (PMID:
30461122; 31941373). However, in one of these studies (PMID: 30461122),
patients were ascertained for deafness and QT prolongation is either
variable or not reported in some family members. The authors did not
carry out functional studies to determine the effect of these variants
on mRNA but suggest that given there is only one coding exon, these
nonsense variants would be unlikely to result in nonsense mediated
decay.<br/> The phenotype of KCNE1-related JLNS2 appears to be milder
than JLNS1 (caused by biallelic KCNQ1 variants) (PMID: 16461811;
31941373)<br/> Heterozygous variants in KCNE1 have been associated with
LQT syndrome without deafness. However, Roberts et al observed a low ECG
penetrance in these individuals with the majority not manifesting
clinically (PMID: 31941373).<br/> ClinGen classified the gene disease
association, KCNE1 and JLNS2, as Moderate stating that although there
was a large amount of experimental evidence and no contradictory
evidence, more case level evidence was required to reach a Definitive
classification.

#### **KCNJ2** - *LQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNJ2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600681
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndromic QT prolongation and cardiac arrhythmias
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
KCNJ2-related Andersen-related Tawil syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion; inframe_insertion;
stop_gained_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12163457; 16217063; 11371347; 32299589; 21493816; 31983240; 21148745;
32947483; 32499698; 32843460
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0008222
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
01.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

KCNJ2 pathogenic variants cause Andersen Tawil syndrome (ATS) through
**altered gene product sequence**. Most of the pathogenic variants cause
Kir2.1 (inwardly-rectifying potassium channel) loss-of-function, either
via trafficking or gating defects.<br/> Kir2.1, encoded by KCNJ2,
contributes a major component of the cardiac action potential
repolarization phase. Pathogenic variants of KCNJ2 gene account for
60-70% of clinical ATS cases, termed type-1 ATS.<br/><br/> The cardiac
manifestation includes QT-U abnormalities but not typical QT
prolongation. Ventricular arrhythmias also differ from typical LQTS with
frequent premature ventricular complexes and polymorphic non-sustained
ventricular tachycardia but only rarely torsades de pointes.
Extracardiac manifestations include dysmorphic features and periodic
paralysis with hypo- and hyperkalemic episodes in some
patients.<br/><br/> The majority of pathogenic variants in KCNJ2 are
missense changes. A small number of inframe deletions, insertions and
stop gained variants predicted to escape nonsense mediated decay (NMD)
have also been reported in this single-exon gene.<br/> The p.Arg218Trp
pathogenic variant is considered a potential mutational hot spot (Davies
et al 2005 PMID: 16217063).<br/>Penetrance in families appears high,
with estimates of greater or equal to 80%.

#### **KCNQ1** - *LQTS* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNQ1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
607542
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
LQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Long QT Syndrome (LQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndromic QT prolongation and cardiac arrhythmias
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
KCNQ1-related Jervell and Lange-Nielsen syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
absent gene product; altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant; stop_gained_NMD_triggering;
frameshift_variant_NMD_triggering; exon_loss_variant;
complex_structural_alteration
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27041150; 23591039; 9020846; 27868350; 23392653; 31983240; 11226272;
25187895
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0024540
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
01.05.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16.06.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Biallelic pathogenic variants in KCNQ1 cause JLNS due to **absent gene
product level or altered gene product sequence**.<br/> Both homozygous
and compound heterozygous pathogenic variants in KCNQ1 have been
reported in JLNS. Nonsense, frameshift, splice site, whole exon
deletions, missense variants and complex rearrangements have been
reported as pathogenic.<br/><br/> Bhuiyan and Wilde (PMID: 23591039)
compared two groups of patients with homozygous variants in KCNQ1. Those
patients where there was residual IKs (slowly activating delayed
rectifier potassium channel/current) even as little as 10%, had QT
prolongation but no hearing loss (autosomal recessive LQTS), whereas
those patients with complete absence of IKs had both QT prolongation and
hearing loss (JLNS). <br/> They concluded that homozygous or compound
heterozygous nonsense, frameshift or exon skipping variants resulting in
100% loss of IKs would result in JLNS. In addition, biallelic missense
mutations that lead to a protein product that does not traffic to the
cell membrane (or is subject to nonsense mediated decay) will lead to
JLNS.<br/> Note: heterozygous variants in KCNQ1 lead to dominant LQTS.

## Short QT Syndrome (SQTS)

### Classic SQTS

#### **KCNH2** - *SQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNH2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
152427
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Short QT Syndrome (SQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Classic SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
KCNH2-related Short QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29876509; 21130771; 30582453; 30175559; 29759541; 29016797; 9547387;
18692916; 14676148; 25974115; 7889573; 30947366; 30496390; 19088443;
30571592; 7736582; 29574456; 21310316; 15673388; 15828882; 31049424;
28491588; 25335996; 19340359; 31072576
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012312
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SQTS is a rare (\< 1/10,000) AD inherited arrhythmia syndrome associated
with atrial fibrillation, ventricular arrhythmia, and risk of sudden
cardiac arrest. Diagnosis is based on a diagnostic scorecard (PMID:
21310316), similar to LQTS. Approximately 20% of diagnosed cases will
have a genetic cause, and KCNH2 is the most common identified
gene.<br/>It is noteworthy that of the 18 probands with SQTS in whom
KCNH2 variants have been identified, 13 had one of 2 variants; 7 with
p.Thr618Ile variant (ClinVar Variation ID# 67297) and 6 with p.Asn588Lys
(ClinVar Variation ID# 14436; NM_172056.2). There is high penetrance for
these recurrent variants.<br/> Experimental evidence derived from
non-patient cells, human-induced pluripotent stem cell-derived cells and
a rabbit animal model (PMID: 30496390) all support this gene’s
relationship with SQTS, with **a gain-of-function mechanism**. These
experimental studies demonstrate that genetic variants identified in
SQTS patients lead to potassium current perturbations concordant with
SQTS phenotype and shortening of the QT interval.<br/><br/> Note
loss-of-function variants in KCNH2 are associated with Long QT Syndrome
(LQTS).

#### **KCNJ2** - *SQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNJ2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600681
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Short QT Syndrome (SQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Classic SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
KCNJ2 relared Short QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19710529; 23440193; 24794859; 11410627; 15761194; 22155372; 29615871;
19285083; 2479485
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012314
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Genetic variants in KCNJ2 have been identified in 6 patients from 5
families with unique variants, including at least 2 probands with a
de-novo variant (paternity not confirmed). Experimental evidence
demonstrated these variants lead to gain-of-function of the late
repolarizing, KCNJ2-encoded Ik1 current in the heart, and abbreviation
of the action potential duration. The ClinGen GCEP reviewing
gene-disease validity considered these data sufficient for classifying
the gene-disease relationship of KCNJ2 as ‘Moderate’ but, in the absence
of segregation or case-control data, the genetic evidence was not
sufficient for a stronger classification. Further information on ClinGen
gene-disease validity can be found here:
<https://search.clinicalgenome.org/kb/genes/HGNC:6263>

#### **KCNQ1** - *SQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KCNQ1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
607542
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Short QT Syndrome (SQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Classic SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
KCNQ1-related Short QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
STRONG
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
gain of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Almost all evidence derived from: NM_000218.3(KCNQ1):c.421G\>A
(p.Val141Met); NM_000218.3(KCNQ1):c.919G\>C (p.Val307Leu)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
28491751; 16109388; 24380499; 26279191; 29213224; 15159330; 26168993;
25974115; 8900283; 28491547; 26346102; 8528244
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012313
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Almost all evidence for KCNQ1 as a cause of SQTS is derived from a
single variant (p.Val141Met) identified in 9 unrelated probands. All 9
cases presented with severe bradycardia in-utero or at birth and in 6
atrial fibrillation was also reported. In one case complete
atrioventricular block was documented. (PMIDs: 24818999; 26279191;
16109388; 24380499; 25974115; 28491547).<br/><br/> Five other variants
have been reported for SQTS, but the phenotypic features of SQTS for 4
of these have been unclear or functional data supporting the mechanism
of disease lacking in the published manuscripts. For the fifth
additionally reported variant, 70-year-old patient presented with
‘idiopathic VF’ and a short QT interval, a KCNQ1 variant, p.Val307Leu,
was reported as the likely genetic culprit, with in vitro studies
reporting a gain-of-function effect of the mutant (Bellocq et al, Circ
2004).<br/><br/> Functional characterisation in cell models confirms a
gain of function mechanism for Val141Met and Val307Leu.<br/> Four cases
of Val141Met are reported as de novo without noted confirmed paternity.
Importantly, in none of the p.Val141Met cases was cardiac arrest or
sudden cardiac death described. <br/>Note: loss-of-function variants in
KCNQ1 are associated with Long QT Syndrome (LQTS).

#### **SLC4A3** - *SQTS* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SLC4A3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
106195
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Short QT Syndrome (SQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Classic SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SLC4A3-related Short QT syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NM_201574(SLC4A3):c.1109G\>A (p.Arg370His)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29167417
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0000453
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

SLC4A3 encodes a plasma membrane anion exchange protein. Genetic
evidence supporting SLC4A3 as a SQTS-causing gene is derived from a
single publication in which exome sequencing was performed in 2
families, including one large pedigree (PMID: 29167417). The same rare
genetic variant (p.Arg370His, c.1109G\>A) was identified in both
families, suggesting they are possibly distantly related. Experimental
evidence from in vitro and zebrafish models suggests reduced membrane
localization of the mutated protein leads to intracellular
alkalinization and shortening of the cardiomyocyte action potential
duration. The genetic evidence, including the unbiased gene discovery
approach of whole exome sequencing and segregation of the identified
genetic variant with a large number of affected individuals within the
presented pedigree, was considered relatively strong by the ClinGen
GCEP. However, lack of other publications supporting this gene-disease
relationship led to a score in the ‘Moderate’ range (for gene-disease
validity). Further information on ClinGen gene-disease validity can be
found here: <https://search.clinicalgenome.org/kb/genes/HGNC:11029>

### Syndrome including shortened QT and cardiac arrhythmias

#### **SLC22A5** - *SQTS* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SLC22A5
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
603377
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
SQTS
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Short QT Syndrome (SQTS)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndrome including shortened QT and cardiac arrhythmias
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SLC22A5-related primary systemic carnitine deficiency
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31472821; 3009296; 26190315
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0008919
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21.07.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Channelopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Variants in SLC22A5 cause autosomal recessive primary systemic carnitine
deficiency (PSCD), a syndrome characterized by hypoketotic hypoglycemia,
hyperammonemia, liver dysfunction, hypotonia and cardiomyopathy (PMID:
26190315). <br/> Homozygote or compound heterozygote variants have been
identified in unexplained sudden cardiac death or resuscitated cardiac
arrest cases without overt extra-cardiac manifestations (PMIDs:
31472821; 3009296). Furthermore, a short QT interval has been
demonstrated in a carnitine deficient mouse model (PMID: 31472821) as
well as in patients with PSCD (PMID: 31472821; 3009296). Importantly,
however, the QT interval in these patients returns to normal with
carnitine supplementation treatment. <br/>Information on ClinGen
gene-disease validity conclusion: the ClinGen GCEP conclude that PSCD is
a metabolic and reversible SQTS-mimic.<br/> While there is no robust
evidence of a relationship between SLC22A5 and true SQTS \[ClinGen
‘disputed’ classification\], there is ‘definitive’ evidence for a
relationship between SLC22A5 and PSCD which may justify testing in
individuals being investigated for possible SQTS. Further information on
ClinGen gene-disease validity can be found here:
<https://search.clinicalgenome.org/kb/genes/HGNC:10969>

## Arrhythmogenic right ventricular cardiomyopathy (ARVC)

### Familial isolated arrhythmogenic cardiomyopathy

#### **DSC2** - *ARVC* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DSC2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125645
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DSC2-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
frameshift_variant; stop_gained; missense_variant; inframe_deletion;
inframe_insertion; NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17963498; 33831308; 24070718; 17186466; 19863551; 24793512; 20031616;
21636032; 23911551; 17033975; 34400560; 31028357; 23863954; 31402444;
26310507
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012506
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

DSC2-related ARVC is due to **decreased gene product level or altered
gene product sequence** due to a variety of mechanisms (e.g. null
alleles, trafficking defects, impaired proteolytic processing, absence
of or impaired protein-protein interactions) (PMID: 31028357; 23911551;
NBK1131).<br/> Loss of function is the likely disease mechanism. ClinGen
found there was some evidence to support haploinsufficiency as a
mechanism. <https://search.clinicalgenome.org/kb/gene-dosage/HGNC:3036>.
<br/> Autosomal dominant inheritance with incomplete penetrance is the
most common mode of transmission (PMID: 21636032; 33831308). Homozygous
and compound heterozygous variants have also been described in
association with ARVC with or without cutaneous features (PMID:
26310507; 23863954; 24793512; 24070718; 34400560). In some cases, these
appear to reflect autosomal recessive inheritance (PMID 24793512;
23863954, 33831308). Instances of digenic inheritance have been
identified with DSC2 variants along with other desmosomal gene
pathogenic variants (PMID: 24070718).<br/> A number of DSC2 variants
have been reported in the literature including nonsense, frameshift,
splice, missense and inframe insertions and deletions (NBK1131;
17033975; 17963498; 17186466; 20031616; 19863551; 31402444).<br/>
DSC2-related ARVC appears to be characterised by an increased risk of
biventricular involvement and heart failure when compared to
PKP2-related ARVC (PMID: 34400560).

#### **DSC2** - *ARVC* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DSC2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125645
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DSC2-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
frameshift_variant; stop_gained; missense_variant; inframe_deletion;
inframe_insertion; NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17963498; 33831308; 24070718; 17186466; 19863551; 24793512; 20031616;
21636032; 23911551; 17033975; 34400560; 31028357; 23863954; 31402444;
26310507
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012506
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

DSC2-related ARVC is due to **decreased gene product level or altered
gene product sequence** due to a variety of mechanisms (e.g. null
alleles, trafficking defects, impaired proteolytic processing, absence
of or impaired protein-protein interactions) (PMID: 31028357; 23911551;
NBK1131).<br/> Loss of function is the likely disease mechanism. ClinGen
found there was some evidence to support haploinsufficiency as a
mechanism. <https://search.clinicalgenome.org/kb/gene-dosage/HGNC:3036>.
<br/> Autosomal dominant inheritance with incomplete penetrance is the
most common mode of transmission (PMID: 21636032; 33831308). Homozygous
and compound heterozygous variants have also been described in
association with ARVC with or without cutaneous features (PMID:
26310507; 23863954; 24793512; 24070718; 34400560). In some cases, these
appear to reflect autosomal recessive inheritance (PMID 24793512;
23863954, 33831308). Instances of digenic inheritance have been
identified with DSC2 variants along with other desmosomal gene
pathogenic variants (PMID: 24070718).<br/> A number of DSC2 variants
have been reported in the literature including nonsense, frameshift,
splice, missense and inframe insertions and deletions (NBK1131;
17033975; 17963498; 17186466; 20031616; 19863551; 31402444).<br/>
DSC2-related ARVC appears to be characterised by an increased risk of
biventricular involvement and heart failure when compared to
PKP2-related ARVC (PMID: 34400560).

#### **DSG2** - *ARVC* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DSG2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125671
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DSG2-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant; inframe_deletion; inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25616645; 33831308; 33917638; 24070718; 27532257; 30790397; 21636032;
16505173; 34400560; 30454721; 27170944; 16823493
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012434
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

DSG2-related ARVC is due to **decreased gene product level or altered
gene product sequence** due to a variety of mechanisms. Much of the
underlying pathogenesis of DSG2 pathogenic variants is still unknown; it
is believed that loss of DSG2 compromises cell-to-cell adhesion between
cardiomyocytes (PMID: 26085008; NBK1131). There is also work revealing
that desmosomal variants can reduce canonical Wnt signaling and
activating Wnt with a GSK3B inhibitor can block disease pathogenesis
(PMID: 16823493; PMID: 27170944).<br/> The usual mode of inheritance is
**autosomal dominant characterized by incomplete penetrance** (PMID:
21636032; 33831308). Compound heterozygous and homozygous variants have
been described. In some families, heterozygous carriers of these
variants were not affected suggesting **autosomal recessive
inheritance** (PMID: 33917638; 34400560; 24070718; 33831308; 30454721).
Patients with \>1 variant appear to have a more severe phenotype (PMID:
25616645; PMID: 30790397).<br/> <br/> The majority of DSG2 variants are
rare missense variants with unknown significance/unknown mechanism of
pathogenicity. In addition, nonsense, frameshift, insertions, deletions,
and splice site variants have all been described (PMID: 16505173;
NBK1131; 30790397; 27532257; 33917638). <br/> DSG2-related ARVC appears
to be characterised by an increased risk of biventricular involvement
and heart failure when compared to PKP2-related ARVC (PMID: 34400560;
30790397).

#### **DSG2** - *ARVC* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DSG2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125671
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DSG2-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant; inframe_deletion; inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25616645; 33831308; 33917638; 24070718; 27532257; 30790397; 21636032;
16505173; 34400560; 30454721; 27170944; 16823493
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012434
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

DSG2-related ARVC is due to **decreased gene product level or altered
gene product sequence** due to a variety of mechanisms. Much of the
underlying pathogenesis of DSG2 pathogenic variants is still unknown; it
is believed that loss of DSG2 compromises cell-to-cell adhesion between
cardiomyocytes (PMID: 26085008; NBK1131). There is also work revealing
that desmosomal variants can reduce canonical Wnt signaling and
activating Wnt with a GSK3B inhibitor can block disease pathogenesis
(PMID: 16823493; PMID: 27170944).<br/> The usual mode of inheritance is
**autosomal dominant characterized by incomplete penetrance** (PMID:
21636032; 33831308). Compound heterozygous and homozygous variants have
been described. In some families, heterozygous carriers of these
variants were not affected suggesting **autosomal recessive
inheritance** (PMID: 33917638; 34400560; 24070718; 33831308; 30454721).
Patients with \>1 variant appear to have a more severe phenotype (PMID:
25616645; PMID: 30790397).<br/> <br/> The majority of DSG2 variants are
rare missense variants with unknown significance/unknown mechanism of
pathogenicity. In addition, nonsense, frameshift, insertions, deletions,
and splice site variants have all been described (PMID: 16505173;
NBK1131; 30790397; 27532257; 33917638). <br/> DSG2-related ARVC appears
to be characterised by an increased risk of biventricular involvement
and heart failure when compared to PKP2-related ARVC (PMID: 34400560;
30790397).

#### **DSP** - *ARVC* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DSP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125647
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DSP-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant; inframe_deletion; inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
33831308; 20716751; 27532257; 33275305; 27761164; 21636032; 31319917;
32372669; 31514951; 23810894; 20940358; 22795705; 24503780; 26604139;
23137101; 11063735; 32808748; 30382575
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011831
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

DSP-related ARVC is due to **decreased gene product level or altered
gene product sequence**.<br/> The disease mechanism is loss of function
via haploinsufficiency, dominant negative or both (PMID 32372669;
23137101; 16917092; NBK1131).<br/>DSP-related ARVC is inherited in an
**autosomal dominant manner characterized by incomplete penetrance**
(PMID: 21636032; 33831308). However, **DSP is associated with multiple
phenotypes which are heterogeneous and overlapping (including DCM, DCM
with cutaneous features, ARVC, and Carvajal syndrome) and autosomal
recessive inheritance has been reported**. There does not appear to be
distinct mechanisms leading to different phenotypes
<https://search.clinicalgenome.org/kb/gene-dosage/HGNC:3052>.<br/><br/>
The initial variant description in DSP was in Carvajal syndrome
characterized by woolly hair, keratoderma and ARVC. In 2000 three
families from Ecuador were found to be homozygous for the variant
7901delG in DSP which produces a premature stop codon leading to a
truncated desmoplakin protein missing the C domain of the tail region
(PMID 11063735). Since then both autosomal dominant and autosomal
recessive patterns of inheritance have been described in Carvajal
syndrome (PMID: 27761164; 20940358; 22795705; 26604139; 23137101). This
was followed by the description of a heterozygous variant in DSP in an
Italian family with ARVC with co-segregation of the variant with
disease. There have been reports of digenic inheritance with other
desmosomal pathogenic variants.<br/><br/> In a retrospective multicentre
study, curly hair and/or thick skin on the palms or soles (palmoplantar
keratoderma) was commonly present in DSP patients (54/98, 55%) but not
in PKP2 patients (1/46, 2%) (PMID: 32372669). Maruthappu et al 2019 also
describe 38 patients with arrhythmogenic cardiomyopathy who were
carriers of a dominant loss-of-function (nonsense or frameshift)
variants in DSP. Nearly all were found to have curly hair and
palmoplantar keratoderma. However, there was one family described where
the majority did not demonstrate a curly hair/cutaneous phenotype. The
variant in this family was located in a fragment (c.3585-5379,
(p.1195-1793)) only included in isoform 1 of DSP (it has previously been
shown that isoform 2 is the major isoform regulating keratinocyte
adhesion (PMID: 30382575).<br/><br/> Both truncating (stop gained, frame
shift, splice site) and non-truncating variants in DSP have been
reported in the literature associated with ARVC (PMID 31319917;
20716751; 23810894; 24503780; 27532257; 31514951; 27761164; 21636032;
32808748). Pathogenic truncating variants are more common.<br/> Grondin
et al 2020 re-evaluated reported missense variants and found an
enrichment localizing to the spectrin repeat domain (SRD) in cases vs
gnomAD. A similar hot spot location (amino acid residues 250-604) was
reported by Kapplinger et al in 2011 (PMID: 32808748; 21636032).<br/>
Smith E et al 2020 report that DSP variants are associated with a
distinct type of cardiomyopathy with a high prevalence of LV
inflammation, fibrosis, and systolic dysfunction, and DSP cardiomyopathy
should be considered in the differential diagnosis for myocarditis and
sarcoidosis (PMID: 32372669).

#### **DSP** - *ARVC* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DSP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125647
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DSP-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant; inframe_deletion; inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
33831308; 20716751; 27532257; 33275305; 27761164; 21636032; 31319917;
32372669; 31514951; 23810894; 20940358; 22795705; 24503780; 26604139;
23137101; 11063735; 32808748; 30382575
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011831
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

DSP-related ARVC is due to **decreased gene product level or altered
gene product sequence**.<br/> The disease mechanism is loss of function
via haploinsufficiency, dominant negative or both (PMID 32372669;
23137101; 16917092; NBK1131).<br/>DSP-related ARVC is inherited in an
**autosomal dominant manner characterized by incomplete penetrance**
(PMID: 21636032; 33831308). However, **DSP is associated with multiple
phenotypes which are heterogeneous and overlapping (including DCM, DCM
with cutaneous features, ARVC, and Carvajal syndrome) and autosomal
recessive inheritance has been reported**. There does not appear to be
distinct mechanisms leading to different phenotypes
<https://search.clinicalgenome.org/kb/gene-dosage/HGNC:3052>.<br/><br/>
The initial variant description in DSP was in Carvajal syndrome
characterized by woolly hair, keratoderma and ARVC. In 2000 three
families from Ecuador were found to be homozygous for the variant
7901delG in DSP which produces a premature stop codon leading to a
truncated desmoplakin protein missing the C domain of the tail region
(PMID 11063735). Since then both autosomal dominant and autosomal
recessive patterns of inheritance have been described in Carvajal
syndrome (PMID: 27761164; 20940358; 22795705; 26604139; 23137101). This
was followed by the description of a heterozygous variant in DSP in an
Italian family with ARVC with co-segregation of the variant with
disease. There have been reports of digenic inheritance with other
desmosomal pathogenic variants.<br/><br/> In a retrospective multicentre
study, curly hair and/or thick skin on the palms or soles (palmoplantar
keratoderma) was commonly present in DSP patients (54/98, 55%) but not
in PKP2 patients (1/46, 2%) (PMID: 32372669). Maruthappu et al 2019 also
describe 38 patients with arrhythmogenic cardiomyopathy who were
carriers of a dominant loss-of-function (nonsense or frameshift)
variants in DSP. Nearly all were found to have curly hair and
palmoplantar keratoderma. However, there was one family described where
the majority did not demonstrate a curly hair/cutaneous phenotype. The
variant in this family was located in a fragment (c.3585-5379,
(p.1195-1793)) only included in isoform 1 of DSP (it has previously been
shown that isoform 2 is the major isoform regulating keratinocyte
adhesion (PMID: 30382575).<br/><br/> Both truncating (stop gained, frame
shift, splice site) and non-truncating variants in DSP have been
reported in the literature associated with ARVC (PMID 31319917;
20716751; 23810894; 24503780; 27532257; 31514951; 27761164; 21636032;
32808748). Pathogenic truncating variants are more common.<br/> Grondin
et al 2020 re-evaluated reported missense variants and found an
enrichment localizing to the spectrin repeat domain (SRD) in cases vs
gnomAD. A similar hot spot location (amino acid residues 250-604) was
reported by Kapplinger et al in 2011 (PMID: 32808748; 21636032).<br/>
Smith E et al 2020 report that DSP variants are associated with a
distinct type of cardiomyopathy with a high prevalence of LV
inflammation, fibrosis, and systolic dysfunction, and DSP cardiomyopathy
should be considered in the differential diagnosis for myocarditis and
sarcoidosis (PMID: 32372669).

#### **PKP2** - *ARVC* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
PKP2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
602861
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
PKP2-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
frameshift_variant; stop_gained; missense_variant; deletion; duplication
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17041889; 25616645; 33831308; 23736219; 24070718; 22781308; 20301310;
24704780; 30619891; 28740174; 17010805; 21636032; 30830208; 34120153
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012180
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
14.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

PKP2 pathogenic variants cause ARVC through **decreased gene product
level or altered gene product sequence**. PKP2 encodes plakophilin-2
which is a protein of the desmosome and provides structural and
functional integrity to adjacent cells. The disease mechanism in ARVC is
loss of function (LoF). Rasmussen et al showed that truncating variants
in PKP2 resulted in PKP2 transcript and protein levels reduced to approx
50% (PMID: 24704780). Cerrone et al showed that loss of PKP2 in adult
myocytes was sufficient to generate an arrhythmogenic cardiomyopathy of
right ventricle predominance in mice (PMID: 28740174). <br/><br/> PKP2
is the major causative gene for ARVC and accounts for 34%-74% of cases
(PMID: 20301310). Inheritance is predominantly autosomal dominant
characterised by variable expression and incomplete penetrance (PMID:
34120153; 21636032; 17010805). <br/><br/> Both recessive and digenic
inheritance (with one pathogenic variant in PKP2 and a second in another
desmosomal gene) have been reported (including a recessive cryptic
splice variant PMID 17041889) and appear to confer a more severe
phenotype (PMID: 30830208; 25616645; 24070718; NBK1131). The expert
panel noted instances where PKP2 LoF variants on both alleles had
resulted in neonatal lethality. <br/><br/> There are over 250 PKP2
variants listed in ClinVar for ARVC (nonsense, frameshift, splice,
missense, deletions, duplications, and complex rearrangements (PMID:
30619891; 25616645; 21636032; 34120153). Dries et al report that PKP2
truncating variants explain a large proportion of ARVC cases but there
is no clear relationship between their transcript position and their
likelihood of disease association (PMID: 30619891). Although missense
variants are associated with disease and validated with functional
studies (PMID 22781308), their mechanism and overall impact in ARVC is
not completely understood. The majority of missense variants in ClinVar
are classified as variants of uncertain significance.

#### **TMEM43** - *ARVC* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TMEM43
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
612048
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TMEM43-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NM_024334.3(TMEM43):c.1073C\>T (p.Ser358Leu)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25343256; 29980933; 33831308; 32062046; 20301310; 22725725; 18313022;
23812740; 21214875; 24598986; 21391237
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011459
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
14.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

TMEM43-related ARVC is due to **altered gene product sequence.**<br/>
Pathogenic variants in TMEM43 are a rare cause of ARVC (PMID:
20301310).<br/> **The majority of genetic evidence comes from one
founder missense variant, (NM_024334.3, c.1073C\>T (p.S358L))** (PMID
18313022; 21214875; 23812740; 20301310; 24598986; 33831308). This was
originally identified in Newfoundland and has subsequently been found in
patients from other countries including USA, Germany, and Denmark (PMID:
33831308; 18313022; 23812740). It is reported that ‘\[the variant occurs
on\] a common haplotype with those from Newfoundland, USA, and Denmark,
suggesting that the mutation originated from a common founder.
Examination of 40 control chromosomes revealed an estimated age of
1300-1500 years for the mutation, which proves the European origin of
the Newfoundland mutation.’ (PMID: 24598986)<br/> The disease mechanism
is largely unknown. There is no evidence currently for
haploinsufficiency
(<https://search.clinicalgenome.org/kb/gene-dosage/HGNC:28472>).<br/><br/>
Although ARVC is known to display incomplete penetrance, this particular
founder variant appears to be more penetrant.<br/> TMEM43-related ARVC
is associated with a high risk of sudden cardiac death and
characteristic clinical and electrocardiographic features (PMID:
32062046). Ventricular ectopy on Holter monitoring is commonly seen and
can occur early in the natural history (PMID: 22725725).<br/> PMID
21391237 described two patients with TMEM43 heterozygous missense
variants in Emery Dreifuss Muscular Dystrophy Related Myopathy. Other
missense variants have been reported but their pathogenicity is debated.

#### **DES** - *ARVC* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DES
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125660
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DES-related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

#### **PLN** - *ARVC* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
PLN
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
172405
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial isolated arrhythmogenic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
PLN -related Arrhythmogenic right ventricular cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

### Rare familial disorder with ARVC

#### **JUP** - *ARVC* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
JUP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
173325
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Arrhythmogenic right ventricular cardiomyopathy (ARVC)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare familial disorder with ARVC
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
JUP-related Naxos disease
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
STRONG
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion; frameshift_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25820315; 21320868; 25705887; 17924338; 10902626; 11691526; 20031617;
21673311; 32966140; 8858175; 20130592; 28098346; 15851108; 8954745;
31402444
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011017
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
05.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

JUP-related Naxos disease (ARVC, woolly hair and palmoplantar
keratoderma) is due to **altered gene product sequence** causing loss of
function of JUP (PMID: 25705887; 21673311; 11691526; 10902626). JUP
encodes the protein plakoglobin. Kaplan et al found that in 4 Naxos
patients, Connexin43 expression at intercellular junctions was
significantly reduced and mutant plakoglobin was expressed but failed to
localize normally at intercellular junctions (PMID: 15851108).<br/>
**Inheritance is autosomal recessive**.<br/> The initial nine patients
described ranged in age from 7 to 41 years. Since then, more patients
have been discovered carrying the disease with an estimate of 1:1000 in
the population of the Greek islands. The disease has also been diagnosed
in other countries (PMID: 32966140).<br/> **A homozygous 2bp deletion**
in plakoglobin (JUP), c.2157delTG, causing a frameshift and premature
termination of the protein and expression of a truncated plakoglobin
lacking 56 residues from the C terminus was described in 2000 (PMID:
10902626). The truncated protein was identified on western blot.<br/>
**In 2017, a homozygous missense variant** was described in 7 unrelated
French-Canadian individuals. All had typical hair and skin findings; 4/7
had ARVC presenting after 28 years (PMID: 28098346). The effect of this
variant in the heterozygous state was not investigated.<br/> Two
siblings of consanguineous parents were found to have a **homozygous 3bp
deletion** in JUP c.901903delGAG (p.Glu301del). Both had woolly hair and
skin findings, only the older sister had ARVC and neither had
palmoplantar keratoderma (PMID: 28098346).<br/> In OMIM there have been
reports of other types of homozygous variants (nonsense, splice,
missense) in JUP causing overlapping phenotypes and segregating with
disease. Data on biallelic LoF variants are sparse. In mice, generation
of a null mutation of the plakoglobin gene by homologous recombination
results in embryonic lethality (PMID: 8954745; 8858175). There are 2
reports in humans who had skin features but no obvious cardiomyopathy
(PMID: 20130592; 21320868). In one, JUP expression in the skin was
absent. Cardiac JUP expression was not directly measured to establish
the consequence in the heart - it is not known whether variant allele
was expressed, degraded, or rescued by alternate splicing.<br/><br/> To
note dominant pathogenic variants in JUP have also been rarely described
in association with ARVC. Asimaki et al reported a dominant variant in
JUP in a German family with ARVC and no obvious cutaneous abnormalities
(PMID: 17924338). Other studies have identified heterozygous missense
variants however their pathogenicity is still debated (PMID: 25820315;
20031617; 31402444).

## Dilated Cardiomyopathy (DCM)

### Familial dilated cardiomyopathy

#### **BAG3** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
BAG3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
603883
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
BAG3-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
missense_variant; stop_gained_NMD_triggering; exon_loss_variant;
transcript_ablation
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25008357; 21353195; 21898660; 23518596; 31808029; 30442290; 26796036;
31983221; 27391596; 32160020; 28737513; 20884878
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0013479
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in BAG3 cause an estimated 2-4% of familial DCM due
to **decreased gene product level or altered gene product sequence**.
This likely leads to loss of function causing several pathological
effects on cardiomyocytes including direct destabilization of the
Z-disc, impaired protein homeostasis leading to proteotoxicity and
increased susceptibility to apoptosis.<br/> **Truncating variants are
responsible for most BAG3-related DCM cases**, the majority of which are
nonsense and frameshift, and include several single, multi-exon, and
whole gene deletions. ClinGen have concluded there is good evidence for
haploinsufficiency. There is not yet evidence for dominant-negative
effects.<br/> There is no apparent enrichment for non-truncating
mutations in any specific domain, and it should be noted that different
missense variants in similar domains can lead to different phenotypes
(e.g., p.Pro209Leu leading to myofibrillar myopathy, and p.Arg218Trp
leading to DCM).<br/> BAG3-related DCM is inherited as an autosomal
dominant trait, characterised by incomplete penetrance.

#### **DES** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DES
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125660
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DES-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; splice_acceptor_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
23349452; 23300193; 10430757; 17626518; 26724190; 11728149; 17325244
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011482
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in DES cause DCM due to **altered gene product
sequence**.<br/> **Only missense variants in DES have been confidently
reported as pathogenic in DCM**. There are no reports of truncating
variants; DES is yet to undergo ClinGen Dosage Haploinsufficiency
investigations. DES has a pLI of 0.01 in gnomAD (o/e = 0.33 (0.19 -
0.6)).<br/><br/> There is some indication that truncating variants in
DES are associated with myofibrillar myopathy rather than DCM. There is
a report of a splice site variant in a DCM case, however, it causes exon
3 skipping to produce an inframe transcript (PMID: 17626518).<br/><br/>
DES-related DCM is inherited in an autosomal dominant manner, and
disease is generally highly penetrant. DES missense variants account for
a very small proportion (estimated 2%) of genetically-explained DCM
(PMID: 17325244).

#### **DSP** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DSP
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125647
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DSP-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
frameshift_variant; missense_variant; stop_gained_NMD_triggering;
splice_donor_variant_NMD_triggering;
splice_acceptor_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
20716751; 27532257; 31317183; 32005173; 24503780; 23022708; 31983221;
32013205
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0005021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
06.08.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
01.02.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in DSP cause DCM due to **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms.
<br/><br/> Both missense and truncating mutations have been reported in
DCM cases, however, there is significantly more evidence in support of
truncating variants being pathogenic.<br/><br/> DSP-related DCM is
inherited in an autosomal dominant manner, and disease is generally
penetrant in families with an increased burden of left ventricular
fibrosis and ventricular tachyarrhythmia. Pathogenic DSP variants are
likely to account for approximately 2-3% of familial DCM cases (PMID:
24503780; 23022708).<br/> Of note DSP is associated with multiple
phenotypes which are heterogeneous and overlapping (including DCM, DCM
with cutaneous features, ARVC, and Carvajal syndrome). There does not
appear to be distinct mechanisms leading to different phenotypes
<https://search.clinicalgenome.org/kb/gene-dosage/HGNC:3052>.

#### **FLNC** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
FLNC
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
102565
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
FLNC-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; stop_gained_NMD_triggering;
frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27601210; 31627847; 27206985; 22020047; 30067491; 27908349; 2843697;
32154132; 32160020; 32150467
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0005021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in FLNC account for approximately 2-4% of familial
DCM due to **decreased gene product level**. The mechanism is likely
loss of function, leading to Z-disc disarray and weakened cell-cell
adhesion, promoting arrhythmogenesis and fibrosis.<br/> Truncating
variants (stop-gained, frameshift and splicing) are responsible for
almost all reported cases of FLNC-related DCM to date. It is possible
that some loss-of-function missense variants could cause DCM and experts
commented on their experience of families with missense variants
segregating with disease. However, there is limited evidence for this.
Xiao et al 2020 report an infant girl presenting with DCM and a
paternally inherited missense variant, p.Arg441Ile in FLNC identified on
exome sequencing. No functional studies were carried out (PMID:
32154132).<br/> The FLNC-related DCM phenotype is frequently
arrhythmogenic, characterized by a high burden of ventricular
arrhythmias and myocardial fibrosis.<br/> FLNC-related DCM is inherited
as an autosomal dominant trait, penetrance in close relatives of an
affected proband is generally high, although not complete, with mean age
of onset in the late 4th or early 5th decade, +/- 10-15 years.

#### **LMNA** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
LMNA
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
150330
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
LMNA-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
frameshift_variant; missense_variant; stop_gained_NMD_triggering;
splice_donor_variant_NMD_triggering;
splice_acceptor_variant_NMD_triggering; exon_loss_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
11897440; 20127487; 29367541; 28912180; 10662742; 12854972; 10580070;
31983221; 18926329; 29095976; 12920062
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0007269
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
30.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in LMNA cause DCM due to **decreased gene product
level or altered gene product sequence**, likely due to a variety of
mechanisms. Missense variants are more prevalent, and are responsible
for a larger proportion of DCM cases, but truncating variants (nonsense,
frameshift, and splice variants) are also associated with disease, most
probably through loss of function mechanisms. Single and multi-exon
deletions have also been reported.<br/> LMNA has a ClinGen Dosage
sensitivity score of 2, indicating there is some evidence for dosage
pathogenicity. Non-missense variants are reported to convey a higher
risk of life-threatening arrhythmia (PMID: 31155932).<br/><br/>
LMNA-related DCM is inherited in an autosomal dominant manner, and
disease is generally a highly penetrant and aggressive arrhythmogenic
phenotype with high rates of heart failure and sudden cardiac death.
LMNA missense and truncating variants account for ~5-8% of genetic
DCM.<br/><br/> Of note LMNA is also associated with several other
conditions (the laminopathies) including muscular dystrophies and
Hutchinson-Gilford progeria. Some of these conditions are autosomal
recessive and some dominant. There is evidence of clustering of variants
to specific regions of the gene. However, DCM causing variants have been
recorded across the gene and are so far all reported with dominant
inheritance.

#### **MYH7** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
MYH7
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
160760
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MYH7-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29093449; 29666183; 11106718; 31983221; 29300372
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0013262
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

MYH7 encodes the beta-myosin heavy chain, part of the sarcomere which
plays a major role in cardiac muscle contraction.<br/> Pathogenic
variants in MYH7 cause DCM due to **altered gene product sequence**
primarily due to decrease in sarcomere force generation.<br/>
Dominant-negative missense variants resulting in an altered protein with
reduced function are responsible for most MYH7-associated DCM cases, and
a small number of inframe indels have also been reported associated with
disease. The likely disease mechanism is a reduction in the passive
stiffness of myofibrils (PMID: 29093449) and deficit in force generation
and force-holding capacity (PMID: 29666183). There is no good evidence
that loss of function is a disease mechanism (e.g. PMID: 31983221), and
MYH7 is not known to be haploinsufficient.<br/> MYH7-related DCM is
inherited in an autosomal dominant manner, and disease has incomplete
penetrance, age-related onset, and variable expressivity. Pathogenic
MYH7 variants are likely to account for approximately 5-6% of familial
DCM cases. There is evidence that biallelic variants can have an
additive effect resulting in more severe cardiomyopathic phenotypes.

#### **PLN** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
PLN
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
172405
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
PLN-related Intrinsic Cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
frameshift_variant; stop_gained; missense_variant; inframe_deletion;
exon_loss_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16432188; 12639993; 33020536; 21167350
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012362
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
24.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in PLN cause cardiomyopathy by **decreased gene
product level or altered gene product sequence**.<br/> Inheritance is
typically autosomal dominant with incomplete penetrance but biallelic
variants have been described and appear to confer an earlier onset and
more severe phenotype (PMID: 12639993).<br/><br/> **PLN is encoded by
one coding exon (52 amino acids)**. Schmitt et al described a missense
variant (p.Arg9Cys) in a patient with DCM. The variant segregated with
disease in the family; transgenic mice developed biventricular
dilatation (PMID: 12610310). A stop gained variant and inframe deletion
have also been described (PMID: 12639993; 16432188). There are 6
pathogenic/likely pathogenic variants reported on ClinVar: 1 missense, 2
stop gained, 2 frameshift and a large deletion all associated with
dilated cardiomyopathy.<br/> In the Netherlands there is a founder
mutation p.Arg14del. Up to 10-15% of both dilated cardiomyopathy and
arrhythmogenic cardiomyopathy patients are reported to be caused by
PLN-R14del. (PMID: 33020536).<br/><br/> ClinGen found no difference in
the molecular mechanism(s) underlying PLN-related DCM and HCM and
observed that inter and intrafamilial variability in phenotype had been
reported. Haghighi et al describe the same variant (p.Leu39X) in a
family causing severe DCM in the homozygous state and both DCM and HCM
phenotypes in the heterozygous state (PMID: 12639993). This variant has
also been reported in other HCM families (PMID: 21167350). As a result,
ClinGen curated PLN for an association with intrinsic cardiomyopathy and
did not separately evaluate the evidence for the role in hypertrophic vs
dilated phenotypes.<br/><br/> PLN is definitively associated with
cardiomyopathy and the majority of variants reported appear to be
associated with DCM. Experts commented that further investigation into
PLN and HCM and the variant classes associated needs to be undertaken.

#### **RBM20** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
RBM20
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
613171
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
RBM20-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
There is a mutation hotspot in exon 9 (amino acids 634-638)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; stop_gained_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
30871351; 25979592; 29367541; 19712804; 29895960; 32851336; 22466703;
29650543; 26084686; 20590677; 27496873; 30871348; 22004663; 21846512;
32789749
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0013168
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in RBM20 cause DCM due to **decreased gene product
level or altered gene product sequence** due to a variety of mechanisms
including altered splicing of targets. A dominant negative effect
causing disrupted RNA binding is also a possible/likely mechanism. <br/>
<br/> Missense variants are responsible for the majority of DCM cases
although a small number of truncating variants (nonsense) have been
reported. Of note, there are multiple ClinVar entries of truncating
variants associated with DCM reported by diagnostic laboratories,
comparing to the very low number of truncating variants detected in
gnomAD.<br/> **Loss of function via truncating variants is not
absolutely established, but highly likely.**<br/> RBM20-related DCM is
inherited in an autosomal dominant manner, and disease is often an
aggressive arrhythmogenic phenotype with high rates of heart failure and
sudden cardiac death. Pathogenic RBM20 variants are likely to account
for approximately 1.5-3% of familial DCM cases. There is a pathogenic
variant hotspot in exon 9 (RS motif, amino acids 634-638).

#### **SCN5A** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SCN5A
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SCN5A-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; stop_gained_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
15671429; 27532257; 26916278; 15466643; 19808398; 21596231; 22675453;
22999724; 30847666
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011003
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
06.08.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
01.02.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in SCN5A cause DCM mainly due to **altered gene
product sequence leading to gain of function, and more rarely
decreased/absent gene product level**. The mechanism by which changes in
sodium conductivity lead to cardiomyopathy is not fully understood;
however, it is thought to be due to disruption of the voltage-sensing
mechanism of this channel and subsequent disruption to action potential
and cardiac contraction over time. It is unclear whether loss of
function is a true disease mechanism (for DCM).<br/><br/> Pathogenic
variants in SCN5A are likely to cause \<2% of familial DCM cases (PMID:
26916278, 21596231) and are associated with an arrhythmogenic phenotype
with high rates of sudden cardiac death. There are no defined hotspot
regions for SCN5A missense variants in DCM, however, they do appear to
commonly lie within voltage sensing regions (S3 and S4 transmembrane
segments) of the protein.<br/><br/> Of note SCN5A loss of function is
generally more associated with Brugada syndrome (BrS); SCN5A is the only
gene definitively associated with BrS. Gain of function variants are
also associated with Long QT syndrome.

#### **TNNC1** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TNNC1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
191040
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TNNC1-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
18212018; 19808376; 32038292; 1554228; 27532257; 17021793; 17977476;
18803402; 20458010; 27604170
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012745
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in TNNC1 are a rare (\<1%) cause of familial DCM due
to **altered gene product sequence**. This leads to a likely ‘poison
peptide’ dominant negative effect, causing alteration of troponin
interactions, and altered (decreased) calcium binding of myofilaments
and resulting in decreased force production.<br/><br/> Missense variants
are responsible for all reported cases of TNNC1-related DCM, with no
evidence that truncating variants are causative of disease.<br/> There
is no apparent enrichment for non-truncating mutations in any specific
domain, although exon 1-3 show regional constraint (gnomAD
database).<br/><br/> TNNC1-related DCM is inherited as an autosomal
dominant trait, with high penetrance observed in families. There is
currently insufficient evidence for autosomal recessive inheritance,
although there are some reports of early onset DCM and other
cardiomyopathy associated with compound heterozygosity for TNNC1
variants (PMID: 27604170).

#### **TNNT2** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TNNT2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
191045
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TNNT2-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
11106718; 29367541; 27532257; 15542288; 20978592; 20031601
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0007267
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in TNNT2 cause DCM due to **altered gene product
sequence**.<br/> Only missense variants have confidently been reported
as pathogenic in TNNT2 DCM cases. There are no reports of truncating
variants and TNNT2 has a ClinGen Dosage haploinsufficiency score of 0,
and a PLi of 0 in gnomAD (19.6 expected and 20 observed).<br/><br/>
TNNT2-related DCM is inherited in an autosomal dominant manner, and
disease is often an aggressive arrhythmogenic phenotype with high rates
of heart failure and sudden cardiac death. TNNT2 missense mutations
account for 3% of genetic DCM (PMID: 27532257).<br/> TNNT2 variants are
associated with an early-onset and more severe form of DCM.<br/> Of note
pathogenic variants in TNNT2 are also associated hypertrophic and
restrictive cardiomyopathies.

#### **TTN** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TTN
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
188840
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TTN-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance; Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Variants must occur in A-band or \>90% percentage spliced in (PSI)
exons. Pathogenic missense variants are very rare. Missense variants
with segregation evidence: NM_001267550.2(TTN):c.2926T\>C (p.Trp976Arg);
NM_001267550.2(TTN):c.533C\>A (p.Ala178Asp);
NM_001267550.2(TTN):c.11674T\>A (p.Cys3892Ser)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant; stop_gained_NMD_triggering;
frameshift_variant_NMD_triggering; exon_loss_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
23418287; 9826585; 9817758; 25759365; 19789381; 25589632; 32160020;
31849696; 18765796; 28045975; 27625337; 11788824; 37253077; 29316444;
27869827; 29238064; 26084686; 32013205; 26315439; 22335739
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011400
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
08.12.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in TTN cause an estimated 15-20% of familial DCM due
to **decreased gene product level and altered gene product sequence**.
The likely disease mechanism is loss of function however, it is unclear
whether this is due to haploinsufficiency or a dominant negative effect,
and it is likely that both mechanisms contribute. It is likely that the
reduced function has a direct effect on the sarcomere, leading to
impaired contractility.<br/> Truncating variants, specifically in exons
constitutively expressed in cardiac tissue (percentage spliced in (PSI)
\>90%) see (PMID: 25589632; 27869827; 32160020), are responsible for the
vast majority of TTN-related DCM cases. Missense variants are difficult
to interpret, and generally not classified as disease-causing, although
there are reports of at least three missense variants with evidence of
pathogenicity.<br/>TTN missense variants with segregation evidence:<br/>
p.Trp976Arg (PMID: 11788824)<br/> p.Ala178Asp (PMID: 27625337)<br/>
p.Cys3575Ser
(<https://www.biorxiv.org/content/10.1101/2020.09.05.282913v1.full.pdf>) -
note not yet peer reviewed<br/><br/> TTN-related DCM is inherited as an
autosomal dominant trait and displays incomplete and age-related onset.
TTN truncating variants are present in ~1% of the general population,
although these variants are more likely to reside in isoforms with lower
functional expression in cardiac tissue.

#### **ACTC1** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
ACTC1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
102540
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
ACTC1-related Dilated cardiomypathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

#### **JPH2** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
JPH2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
605267
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
JPH2-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

#### **NEXN** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
NEXN
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
613121
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
NEXN-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

#### **TNNI3** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TNNI3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
191044
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TNNI3-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

#### **TPM1** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TPM1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
191010
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TPM1-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

#### **VCL** - *DCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
VCL
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
193065
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Dilated Cardiomyopathy (DCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
VCL-related Dilated cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

## Hypertrophic Cardiomyopathy (HCM)

### Familial hypertrophic cardiomyopathy

#### **ACTC1** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
ACTC1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
102540
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
ACTC1-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10966831; 20031618; 17611253; 28972856; 27532257; 26061005; 10330430;
28007147; 23604709
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012799
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

ACTC1 pathogenic variants cause HCM through **altered gene product
sequence**. The disease mechanism is not definitively known but may
involve impact on sarcomere force generation. There is currently little
evidence to support haploinsufficiency as a mechanism.<br/> Inheritance
is autosomal dominant. There is limited information regarding
penetrance. The initial missense variant identified by Mogensen et al
was reported to be highly penetrant in family members but with a
variable age of onset and severity (PMID: 10330430).<br/> ACTC1 variants
account for \<3-5% of HCM cases (NBK1768; 20031618). <br/>
**Heterozygous missense variants are the major type of pathogenic
variants found**. One inframe deletion (a deletion of a single amino
acid, Phe92del) has been reported in association with HCM (PMID:
20031618). This variant is classified as likely pathogenic on ClinVar.
Liu et al suggest it may act to change the local structure and
arrangement of amino acids in the actomyosin binding site (PMID:
28972856).<br/> Large deletions or duplications have not been described.

#### **ACTN2** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
ACTN2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
102573
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
ACTN2-related cardiac and skeletal myopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
frameshift_variant_NMD_triggering; stop_gained_NMD_triggering;
missense_variant; inframe_deletion; deletion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
34802252; 34526680; 20022194; 25224718; 25173926; 30630173; 31956495;
31680489; 17097056; 27532257; 31983221; 33500567; 32973354; 30701273;
30900782; 38311799; 39971408
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0000591
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2024
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17.03.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ClinGen HCVD GCEP
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in ACTN2 are definitively associated with cardiac
and skeletal myopathy. They can cause disease through **decreased gene
product level and altered gene product sequence**.<br/><br/>ClinGen
originally curated ACTN2 for intrinsic cardiomyopathy (hypertrophic
cardiomyopathy (HCM) and dilated cardiomyopathy (DCM)) but this was
re-evaluated and expanded to include other cardiomyopathy types and
skeletal muscle phenotypes in December 2024 (PMID 39971408). ClinGen
found no difference in the mechanisms underlying these different disease
entities and therefore have grouped them together.<br/><br/> Individuals
can show variable cardiomyopathy phenotypes, arrhythmias and sudden
death (PMID: 34526680); the phenotype can vary between individuals in
the same family. Pathogenic variants in this gene can give rise to
isolated left ventricular hypertrophy (LVH). The inheritance pattern is
autosomal dominant; however there has been a report of a homozygous
ACTN2-truncating variant (in the terminal exon) with a recessive mode of
inheritance leading to restrictive cardiomyopathy (RCM) (PMID:
34802252). The skeletal phenotype is also variable and includes distal
myopathy and congenital myopathy. As per ClinGen ‘The skeletal muscle
cases reported in different families carrying ACTN2 variants had
homozygous and heterozygous missense variants reported. In the combined
cases, where patients presented with both cardiac and skeletal muscle
phenotypes, the proband showed frameshift and missense ACTN2 variants’
(PMID: 30701273, PMID: 30900782, PMID: 38311799).<br/><br/>Missense,
nonsense, frameshift and larger deletions have all been reported in
association with cardiomyopathy phenotypes. 4 missense variants have
segregation data and 3 have functional data to support pathogenicity.
Patients with these variants presented with atypical and diverse
phenotypes. In their review, Walsh et al summarise what is known about
ACTN2 variants and their role in cardiomyopathy (PMID:
34526680).<br/><br/> Missense variants with segregation evidence:
NM_001103.4(ACTN2): c.355G\>A (p.Ala119Thr) PMID: 20022194, PMID:
25224718; NM_001103.4(ACTN2): c.683T\>C (p.Met228Thr) PMID:
25173926;<br/>NM_001103.4(ACTN2): c.959T\>G (p.Leu320Arg) PMID:
30630173;<br/>NM_001103.4(ACTN2): c.1418A\>G (p.Tyr473Cys) PMID:
31956495.<br/><br/>Missense variants with functional
evidence:<br/>NM_001103.4(ACTN2): c.740C\>T (p.Thr247Met) PMID:
31680489;<br/> NM_001103.4(ACTN2):c.332G\>T (p.Gly111Val) PMID:
17097056;<br/>NM_001103.4(ACTN2):c.355G\>A (p.Ala119Thr) PMID:
20022194.<br/><br/>According to Genome Aggregation Database (gnomAD),
loss of function variants are constrained in ACTN2. There have been
reports of deletions in ACTN2 leading to disease. An exon 3-6 deletion
leading to a frameshift was found in two families with left ventricular
non compaction (LVNC) and HCM (PMID 32973354) and an exon 8-10 deletion
leading to an in frame deletion was found in a family with LVNC and
arrhythmias (PMID: 34802252, PMID: 34526680).<br/> Truncating variants
in ACTN2 have been identified in approximately 1% of patients previously
classified as having LVNC. These variants were significantly enriched in
individuals with an LVNC classification compared to controls (0.65%
vs. 0.01%, p = 1.3E-06) (PMID 33500567). Of note, it is unclear whether
these ‘LVNC’ individuals exhibited both hypertrabeculation and
cardiomyopathy or only hypertrabeculation.<br/><br/> Overall, rare
variants are not enriched for ACTN2 in HCM or DCM cohorts which suggests
they are rarely causative of disease so should be interpreted with
caution in cardiomyopathy patients (PMID: 27532257, PMID:
31983221).<br/><br/> Expert review panel: ClinGen Hereditary
Cardiovascular Disease Gene Curation Expert Panel. Review date:
17/03/2025

#### **ALPK3** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
ALPK3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
617608
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
ALPK3-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
STRONG
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
stop_gained_variant; frameshift_variant; splice_donor_variant;
splice_acceptor_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
33191771; 34263907; 34526680; 32480058; 39971408
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0054838
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2024
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17.03.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ClinGen HCVD GCEP
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in ALPK3 are strongly associated with hypertrophic
cardiomyopathy (HCM) (PMID 39971408). They cause disease through
**decreased gene product level**. The mechanism of disease is **loss of
function**. <br/><br/> Heterozygous truncating variants in ALPK3 have
been found to be enriched in HCM cohorts over population controls. They
have been detected in 1.6% of patients with HCM in two separate cohorts
(PMID: 34263907). There is also segregation data to support an
association between heterozygous truncating variants and HCM (PMID:
33191771; 34263907). Pathogenic heterozygous missense/non-truncating
variants have **not** been reported in association with cardiomyopathy
(PMID: 34526680). The phenotype associated with autosomal dominant ALPK3
related HCM appears to be later onset disease in adulthood with a higher
prevalence of apical or concentric hypertrophy. Severity was noted to be
similar to those with sarcomeric HCM. No extra-cardiac features are
reported (PMIDs: 34263907; 34526680). Age related penetrance was
observed in genotype positive family members. Given the effect size is
relatively modest compared to other causal variants in this context
(Lopes et al found an odds ratio of 16 between cases and controls,
PMIDs: 34263907), the penetrance in the population is expected to be
very low. **Therefore caution is required when interpreting these
variants particularly as secondary findings**. A research letter
describing a case series of ALPK3 related AD HCM is available here, doi:
<https://doi.org/10.1101/2025.03.27.25324722>.<br/><br/> ALPK3 is also
definitively associated with autosomal recessive cardiomyopathy.
Biallelic ALPK3 variants (both biallelic truncating and compound
heterozygous missense and truncating variants) have been associated with
early onset HCM and DCM. The phenotype is variable but several infants
have been described as having DCM that progresses to HCM (PMID:
32480058). Extra-cardiac features have also been described in some
individuals – see curation summary for ALPK3 recessively inherited HCM.
Heterozygous relatives of these individuals were mostly described as
unaffected but a minority, 10-15%, had adult onset HCM (PMID 34526680).
Expert review panel: ClinGen Hereditary Cardiovascular Disease Gene
Curation Expert Panel. Review date: 17/03/2025

#### **CSRP3** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CSRP3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600824
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CSRP3-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
frameshift_variant_NMD_triggering; stop_gained_NMD_triggering;
splice_acceptor_variant; splice_donor_variant; missense_variant;
inframe_deletion; inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
30012424; 31919335; 30681346; 28082330; 18505755; 33035702; 34526680;
16352453; 15781201; 23396983; 26656175; 21425739; 12642359; 19035361;
25351510; 23861362; 20087448; 34558151; 33012304; 37431535; 39971408
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012804
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2024
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17.03.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ClinGen HCVD GCEP
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in CSRP3 are definitively associated with
hypertrophic cardiomyopathy (HCM) (PMID 39971408). They can cause
disease through **decreased gene product level and altered gene product
sequence**. The mechanism is **loss of function**. <br/><br/> Both
autosomal dominant and autosomal recessive inheritance have been
reported. <br/><br/> Seven patients with biallelic truncating variants
and HCM have been described (PMID: 30012424, PMID: 31919335,
<PMID:37431535>). The pathogenicity of these variants in heterozygous
relatives is not known. Separately, heterozygous CSRP3-truncating
variants have been described in patients with HCM (PMID: 30681346).
<br/><br/> Twenty-five unique variants (16 missense, 4 frameshift, 2
nonsense, 2 canonical splice site, and 1 indel) have been reported in
humans (PMIDs: 16352453, 18505755, 15781201, 23396983, 26656175,
21425739, 12642359, 19035361, 25351510, 23861362, 20087448, 30012424,
34558151, 33035702, 31919335, 33012304, 37431535). A meta-analysis found
a significant excess of rare variants in cases compared with general
population controls (0.9% versus 0.3%) (PMID: 28082330). There is
segregation evidence for the missense variant NM_003476.5(CSRP3)
c.172T\>G (p.Cys58Gly) (PMID: 18505755). And these recurrent variants
are significantly enriched in HCM cases: NM_003476.5(CSRP3) c.131T\>C
(p.Leu44Pro) PMID: 28082330; NM_003476.5(CSRP3) c.449G\>A (p.Cys150Tyr)
PMID: 33035702. <br/><br/> Expert review panel: ClinGen Hereditary
Cardiovascular Disease Gene Curation Expert Panel. Review date:
17/03/2025

#### **CSRP3** - *HCM* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CSRP3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600824
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CSRP3-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
frameshift_variant_NMD_triggering; stop_gained_NMD_triggering;
splice_acceptor_variant; splice_donor_variant; missense_variant;
inframe_deletion; inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
30012424; 31919335; 30681346; 28082330; 18505755; 33035702; 34526680;
16352453; 15781201; 23396983; 26656175; 21425739; 12642359; 19035361;
25351510; 23861362; 20087448; 34558151; 33012304; 37431535; 39971408
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012804
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2024
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17.03.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ClinGen HCVD GCEP
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in CSRP3 are definitively associated with
hypertrophic cardiomyopathy (HCM) (PMID 39971408). They can cause
disease through **decreased gene product level and altered gene product
sequence**. The mechanism is **loss of function**. <br/><br/> Both
autosomal dominant and autosomal recessive inheritance have been
reported. <br/><br/> Seven patients with biallelic truncating variants
and HCM have been described (PMID: 30012424, PMID: 31919335,
<PMID:37431535>). The pathogenicity of these variants in heterozygous
relatives is not known. Separately, heterozygous CSRP3-truncating
variants have been described in patients with HCM (PMID: 30681346).
<br/><br/> Twenty-five unique variants (16 missense, 4 frameshift, 2
nonsense, 2 canonical splice site, and 1 indel) have been reported in
humans (PMIDs: 16352453, 18505755, 15781201, 23396983, 26656175,
21425739, 12642359, 19035361, 25351510, 23861362, 20087448, 30012424,
34558151, 33035702, 31919335, 33012304, 37431535). A meta-analysis found
a significant excess of rare variants in cases compared with general
population controls (0.9% versus 0.3%) (PMID: 28082330). There is
segregation evidence for the missense variant NM_003476.5(CSRP3)
c.172T\>G (p.Cys58Gly) (PMID: 18505755). And these recurrent variants
are significantly enriched in HCM cases: NM_003476.5(CSRP3) c.131T\>C
(p.Leu44Pro) PMID: 28082330; NM_003476.5(CSRP3) c.449G\>A (p.Cys150Tyr)
PMID: 33035702. <br/><br/> Expert review panel: ClinGen Hereditary
Cardiovascular Disease Gene Curation Expert Panel. Review date:
17/03/2025

#### **FHOD3** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
FHOD3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
609691
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
FHOD3-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant_NMD_escaping; splice_donor_variant_NMD_escaping;
missense_variant; inframe_deletion; inframe_insertion; exon_loss_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19706596; 30442288; 29907873; 30898215; 31742804; 32335906; 33586461;
34526680; 35288587; 38051749; 39971408
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0005045
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2024
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17.03.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ClinGen HCVD GCEP
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in FHOD3 are definitively associated with
hypertrophic cardiomyopathy (HCM) (PMID 39971408). They can cause
disease through **altered gene product sequence**. The mechanism of
disease is not known. FHOD3 encodes FH1/FH2 domain-containing protein 3,
a cardiac formin protein that plays a critical role in regulating
sarcomere organisation in cardiomyocytes (PMID: 19706596). <br/><br/>
The inheritance pattern is autosomal dominant. Ochoa et al demonstrated
that patients with FHOD3 variants develop mild to moderate left
ventricular hypertrophy (LVH), similar to that seen in general HCM
cohorts (PMID: 30442288).<br/><br/> Disease-causing variants in FHOD3
are usually non-truncating (PMID 34526680), and they cluster in two
regions in the diaphanous inhibitory domain: exon 12 and the coiled-coil
domain (encoded by exons 15 and 16). There are recurrent pathogenic
variants in exon 12: c.15781580del (p.Ser527del) (PMID: 30442288,
31742804) and c.1583A\>G (p.Tyr528Cys) (PMID: 30442288, 34526680). Whole
in frame deletions of exons 15 and 16 have also been reported (PMID:
32335906). Additionally, specific splice donor site variants in exon 11,
c.1286+2delT (PMID: 33586461), and exon 12, c.1646+1G\>A/C/T (PMID:
30898215; 35288587) and the Balkan founder variant c.1646+2T\>C (PMID:
38051749), have been observed in multiple individuals with HCM. The exon
12 splice variant, c.1646+1G\>A/C/T, is predicted to cause an inframe
deletion of exon 12. ClinGen also noted two large CNVs, an 11kb deletion
from c.1836-1527 in intron 14 to c.2022-2042 in intron 16 and an 11kb
deletion c.1836-2543 in intron 14 to c.2021+2217 in intron 16 (PMID:
32335906,
<https://www.medrxiv.org/content/10.1101/2024.07.29.24311195v1.full-text>).
<br/><br/> Expert review panel: ClinGen Hereditary Cardiovascular
Disease Gene Curation Expert Panel. Review date: 17/03/2025

#### **MYBPC3** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
MYBPC3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600958
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MYBPC3-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant; splice_donor_variant;
frameshift_variant; stop_gained; missense_variant; inframe_deletion;
intron_variant; inframe_insertion; stop_gained_NMD_triggering;
splice_donor_variant_NMD_triggering; frameshift_variant_NMD_triggering;
splice_acceptor_variant_NMD_triggering; exon_loss_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25611685; 32163302; 30696458; 28912181; 27532257; 7493025; 31877118;
17937428; 29300372; 32841044; 20624503; 28007147; 22057632; 12707239;
18467358; 32731933; 26573135; 32396390; 19151713
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0007268
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

MYBPC3 pathogenic variants cause HCM through **decreased gene product
level or altered gene product sequence** either leading to a reduction
in MyBP-C content in the sarcomere or altered function. **The disease
mechanism is loss of function**; There is evidence of haploinsufficiency
(PMID 31877118; 22057632; 32841044).<br/> Variants in MYBPC3 and MYH7
collectively account for up to 50% of all clinically recognised cases of
HCM and constitute at least 75% of probands where a variant is
identified (PMID: 28007147; NBK1768).<br/> Inheritance is usually
autosomal dominant, typified by incomplete penetrance and variable
expressivity.<br/> Homozygous and compound heterozygous variants have
been reported and can lead to severe, early onset phenotypes (PMID:
26573135; 25611685; PMID: 18467358; PMID: 17937428).<br/><br/> **The
majority of variants are heterozygous frameshift, nonsense, or splice
site variants** that result in premature termination codons (PMID:
25611685; 31877118). Missense and inframe indels are also frequently
reported and a subset have been shown to cause loss of function through
failure of myofilament incorporation and rapid degradation, further
supporting haploinsufficiency as a mechanism (PMID: 32841044). Variants
in MYBPC3 affecting canonical splice site dinucleotides are a
well-characterised cause of HCM. Furthermore, recent work has identified
more deeply intronic variants associated with disease
(<https://doi.org/10.3390/cardiogenetics11020009>; PMID: 32396390).
There are 39 pathogenic/likely pathogenic MYBPC3 intronic variants
submitted on ClinVar.<br/> The common 25 base pair intronic deletion
detected in 4% to 8% of South Asian populations, is associated with
cardiomyopathy as a risk allele (PMID: 19151713).<br/> Although there is
significant genetic and allelic heterogeneity in HCM, there are also
several MYBPC3 founder variants (PMID: 28912181; 27532257).

#### **MYH7** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
MYH7
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
160760
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MYH7-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion; stop_gained_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25611685; 20359594; 34460321; 30696458; 25209314; 27532257; 1552912;
29300372; 15856146; 1944483; 30924982; 1975517; 33500567; 12788380;
30681346; 32731933; 27247418
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0008647
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

MYH7 pathogenic variants cause HCM through **altered gene product
sequence** leading to an increase in sarcomere force generation.
Variants produce an abnormal activated protein that incorporates into
the sarcomere as a ‘poison peptide’. Variants can either directly affect
motor function or can impact on myosin ‘interacting head motif’ and
therefore impair inactivation kinetics. There is currently no evidence
to support haploinsufficiency as a disease mechanism.<br/> Inheritance
is autosomal dominant, typified by incomplete penetrance and variable
expressivity. <br/><br/> **Most pathogenic variants are missense**.
There are some inframe deletions reported.<br/> A frameshift variant has
been identified in 3.3% of Egyptian HCM patients. It is predicted to
result in a premature termination codon downstream of the last exon-exon
junction of the gene that is expected to escape nonsense-mediated decay
(NMD). (PMID: 34460321).<br/><br/> MYH7 loss of function (LoF) variants
are very rare and their contribution to inherited cardiomyopathy is
incompletely understood. Notably in a study of left ventricular
non-compaction (LVNC), MYH7 truncating variants, generally considered
non-pathogenic for cardiomyopathies, were 20-fold enriched in LVNC cases
over controls (PMID: 33500567). While there is currently no evidence for
a disease-causing role in the heterozygous state in HCM, compound
heterozygosity of LoF variants along with missense variants can lead to
extremely severe presentations, mimicking recessive
inheritance.<br/><br/> There is a clustering of HCM variants in the head
region conferring a high probability of pathogenicity (amino acid
residues 181-937). (PMID: 27247418; 27532257; 29300372; 30696458). <br/>
Kelly et al 2018 (PMID: 29300372) provide gene-specific adaptations of
ACMG criteria for MYH7 and HCM.

#### **MYL2** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
MYL2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
160781
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MYL2-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25611685; 25324513; 28912181; 30696458; 8673105; 32453731; 16076902;
12404107; 9535554; 32731933; 24111713; 16837010; 23365102; 28007147
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012112
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

MYL2 pathogenic variants cause autosomal HCM through **altered gene
product sequence**. The disease mechanism is not definitively known but
may involve destabilization of the interacting heads motif (PMID:
28606303).<br/> There is currently insufficient evidence to support
haploinsufficiency
(<https://search.clinicalgenome.org/kb/gene-dosage/HGNC:7583>).<br/>
MYL2 variants account for \<3% of HCM cases (NBK1768; 32731933).<br/>
Inheritance is typically autosomal dominant. There is limited
information regarding penetrance.<br/><br/> Homozygous and compound
heterozygous variants have been described in association with a lethal
myosinopathy (PMID: 23365102). A study reported a homozygous frameshift
variant causing infantile onset HCM. Heterozygous parents were
unaffected. The authors suggest a molecular mechanism by which
loss-of-function variants in MYL2 are recessive while missense variants
are dominant in HCM. Several loss-of-function variants are reported in
gnomAD suggesting MYL2 is not intolerant to LoF variants. (PMID:
32453731). <br/><br/>**The majority of variants reported are missense**
(PMID: 8673105; 9535554; 12404107.)<br/> On ClinVar, nearly all
pathogenic variants are missense. There is one frameshift variant
associated only with cardiomyopathy, not HCM specifically and a deletion
encompassing exon 7 expected to result in a truncated protein -
<https://www.ncbi.nlm.nih.gov/clinvar/variation/417460/>.<br/> Walsh et
al found the MYL2 gene to be significantly enriched for non-truncating
variants, odds ratio 9.1 (6.2-13.3) (PMID: 30696458).

#### **MYL3** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
MYL3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
160790
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MYL3-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25611685; 12021217; 20031618; 22957257; 8673105; 30696458; 26443374;
28369730; 33288880; 29914921
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012111
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
24.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

MYL3 pathogenic variants cause HCM through **altered gene product
sequence**. The disease mechanism is not definitively known but may
involve impairing protein-protein interaction with components of the
sarcomere and destabilizing the interacting heads motif (PMID: 30275503;
28606303). There is currently no evidence to support haploinsufficiency
as a mechanism
(<https://search.clinicalgenome.org/kb/gene-dosage/HGNC:7584>).<br/>
MYL3 variants account for \<3% of HCM cases (NBK1768; 20031618;
28369730).<br/> Inheritance is primarily autosomal dominant. There is
limited information regarding penetrance given small numbers of variants
identified. In a large family where a missense variant was identified
(p. Arg94His), penetrance was estimated at 88% (PMID: 26443374).<br/>
Osborn et al 2021 (PMID: 33288880) report a homozygous missense variant
in a large family with HCM and sudden cardiac death. Heterozygous
carriers were unaffected. Another biallelic variant was also described
by Olson et al in 2002 (E143K) (PMID: 12021217), however this variant
has also been associated with HCM variably in the heterozygous state.
<br/><br/>**Heterozygous missense variants are the major type of
pathogenic variants found** (PMID: 8673105; 20031618; 25611685). There
are reports on ClinVar of frameshift and splice site variants, but these
are classified as uncertain significance or conflicting. There is
currently insufficient evidence to support loss of function as a
mechanism.<br/> Walsh et al reported a cluster of non-truncating MYL3
variants in HCM at amino acid residues 143-180 (PMID: 30696458).

#### **PLN** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
PLN
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
172405
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
PLN-related Intrinsic Cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
frameshift_variant; stop_gained; missense_variant; inframe_deletion;
exon_loss_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16432188; 12639993; 33020536; 21167350
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012362
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
24.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in PLN cause cardiomyopathy by **decreased gene
product level or altered gene product sequence**.<br/> Inheritance is
typically autosomal dominant with incomplete penetrance but biallelic
variants have been described and appear to confer an earlier onset and
more severe phenotype (PMID: 12639993).<br/><br/> **PLN is encoded by
one coding exon (52 amino acids)**. Schmitt et al described a missense
variant (p.Arg9Cys) in a patient with DCM. The variant segregated with
disease in the family; transgenic mice developed biventricular
dilatation (PMID: 12610310). A stop gained variant and inframe deletion
have also been described (PMID: 12639993; 16432188). There are 6
pathogenic/likely pathogenic variants reported on ClinVar: 1 missense, 2
stop gained, 2 frameshift and a large deletion all associated with
dilated cardiomyopathy.<br/> In the Netherlands there is a founder
mutation p.Arg14del. Up to 10-15% of both dilated cardiomyopathy and
arrhythmogenic cardiomyopathy patients are reported to be caused by
PLN-R14del. (PMID: 33020536).<br/><br/> ClinGen found no difference in
the molecular mechanism(s) underlying PLN-related DCM and HCM and
observed that inter and intrafamilial variability in phenotype had been
reported. Haghighi et al describe the same variant (p.Leu39X) in a
family causing severe DCM in the homozygous state and both DCM and HCM
phenotypes in the heterozygous state (PMID: 12639993). This variant has
also been reported in other HCM families (PMID: 21167350). As a result,
ClinGen curated PLN for an association with intrinsic cardiomyopathy and
did not separately evaluate the evidence for the role in hypertrophic vs
dilated phenotypes.<br/><br/> PLN is definitively associated with
cardiomyopathy and the majority of variants reported appear to be
associated with DCM. Experts commented that further investigation into
PLN and HCM and the variant classes associated needs to be undertaken.

#### **TNNC1** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TNNC1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
191040
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TNNC1-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; frameshift_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
11385718; 18572189; 21262074; 22815480; 26779504; 28771489; 29121657;
29255176; 30775854; 34317385; 33658040; 34488226; 39971408
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0013195
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.10.2024
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
17.03.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
ClinGen HCVD GCEP
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in TNNC1 are definitively associated with
hypertrophic cardiomyopathy (HCM) (PMID 39971408). They can cause
disease through **altered gene product sequence**. The mechanism of
disease is not known. The inheritance pattern is autosomal dominant.
<br/><br/> According to ClinGen, at least 11 unique variants have been
reported, 10 missense and 1 frameshift variant predicted to escape
nonsense mediated decay (NMD). PMIDs: 11385718; 18572189; 21262074;
22815480; 26779504; 28771489; 29121657; 29255176; 30775854; 34317385;
33658040; 34488226. <br/><br/> Although there is no apparent enrichment
for non-truncating variants in any specific domain, exon 1-3 show
regional missense constraint (gnomAD database). <br/> TNNC1 is
definitively associated with dilated cardiomyopathy (DCM). It has also
been reported in association with left ventricular noncompaction (LVNC),
and restrictive cardiomyopathy (RCM) but these gene disease
relationships have not been curated by ClinGen. <br/><br/> Expert review
panel: ClinGen Hereditary Cardiovascular Disease Gene Curation Expert
Panel. Review date: 17/03/2025

#### **TNNI3** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TNNI3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
191044
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TNNI3-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
21415410; 25611685; 30696458; 28912181; 32731933; 23270746; 9241277;
21839045; 15607392; 26440512
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0013369
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
24.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

TNNI3 pathogenic variants cause HCM through **altered gene product
sequence**. The disease mechanism is not definitively known but a review
by Tardif et al reported functional studies on missense variants that
found an increase in the Ca2+ sensitivity of myofilament activation
(PMID: 21415410). ClinGen also conclude that ‘missense mutations have
\[been\] shown to affect Ca2+ binding to myofilaments containing the
mutant TNNI3 (PMIDs: 16531415 and 22675533) or result in an increased
myofilament response to Ca2+ (PMID: 11735257).’
<https://search.clinicalgenome.org/kb/genes/HGNC:11947>.<br/> TNNI3
variants account for approx. 3-5% of HCM cases (NBK1768; PMID:
15607392).<br/><br/> **Inheritance is autosomal dominant characterised
by incomplete penetrance** (PMID: 26440512; 15607392; 9241277).
Lorenzini et al found that subjects with TNNI3 variants had a lower
penetrance than variants in MYBPC3, MYH7, and TNNT2 (PMID: 32731933).
Experts commented that variants seen in TNNI3 in severe/early onset
disease were more likely to be de novo in origin.<br/> Maron et al 2012
describe 4 HCM probands with both pathogenic variants in TNNI3 and
MYBPC3 (PMID: 21839045). In addition, 2 siblings have been reported with
homozygous TNNI3 missense variants (Arg162Trp) and severe myocardial
hypertrophy. Parents of the affected children are consanguineous and
along with other family members harbouring the same variant in the
heterozygous state, were unaffected (PMID: 23270746). Multiple
disease-causing sarcomeric variants appear to be associated with more
severe disease. There is not enough evidence to suggest biallelic
variants in TNNI3 have a distinct mechanism compared to monoallelic
variants. Of note both dominant and recessive modes of inheritance have
been reported in TNNI3-related DCM (PMID: 26440512).<br/><br/>
**Heterozygous missense variants are the major type of pathogenic
variants found**. There are also reports of inframe deletions. In a
systematic review, Mogensen et al reported 91% of all variants were
missense variants. Six variants (Arg141Gln, Arg145Trp, Arg157Val,
Arg162Gln, Ser166Phe, and Lys183Del) appeared with a particularly high
frequency and were identified in 116 of the 256 probands (45%) (PMID:
26440512). There are reports on ClinVar of frameshift, splice site
variants and nonsense variants but these are either classified as
uncertain significance/conflicting, not associated with HCM specifically
or have no functional evidence to demonstrate loss of function as a
mechanism. ClinGen have concluded that there is currently limited
evidence to support haploinsufficiency as a mechanism.<br/> Mogensen et
al found that 85% of variants were identified in exons 7 and 8 (amino
acid residues 125 - 210). (PMID: 26440512).<br/> Similarly, Walsh et al
detected a cluster of non-truncating TNNI3 variants in HCM at amino acid
residues 141-209 (PMID: 30696458).

#### **TNNT2** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TNNT2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
191045
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TNNT2-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion; stop_gained_NMD_escaping;
splice_donor_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
25611685; 22144547; 12707239; 28973951; 30696458; 28912181; 8205619;
30578328; 32731933; 7898523; 11034944; 28007147; 10965086
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0007266
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
24.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

TNNT2 pathogenic variants cause HCM through **altered gene product
sequence**. Troponin T is a regulatory protein found in striated muscles
that forms a complex with troponin I (TnI) and troponin C (TnC) that,
together with tropomyosin (TM), is required for Ca2+ dependent
regulation of muscle contraction.<br/><br/> The mechanism appears to be
dominant negative rather than haploinsufficiency. **The majority of
pathogenic variants are missense**. A functional study of a splice
variant associated with HCM concluded that the resulting truncated
protein does not function as a ‘null protein, but rather as a
dominant-negative leading to reduction in the level of calcium activated
force production’ (PMID: 8958207). Gangadharan et al suggest the primary
reason by which TNNT2 variants between residues 92 and 144 cause
cardiomyopathy is by changing the affinity of TnT for Tm within the N
terminal part of Troponin T (PMID: 28973951).<br/> TNNT2 variants
account for \<5% of HCM cases (NBK1768; 12860912)<br/> **Inheritance is
autosomal dominant characterised by incomplete penetrance** (PMID:
32731933).<br/> Homozygous TNNT2 variants causing HCM are rare and have
been reported to be associated with more severe disease (PMID 30578328;
10965086; 11034944). Piroddi et al investigated a patient with severe,
early onset HCM with a homozygous K280N variant. There was no family
information available, however the authors demonstrated that it resulted
in 100% mutant cTnT with no evidence of haploinsufficiency and suggested
this supports the idea of a gene dose-dependent effect of HCM variants
on the severity of the phenotype (PMID: 30578328).<br/> Experts
commented that variants seen in TNNT2 in severe/early onset disease were
more likely to be de novo in origin.<br/> Nearly all pathogenic variants
are missense (PMID: 12707239; 22144547; 25611685). However, there are a
few reports of inframe deletions, nonsense variants in the final exon
which likely escape nonsense mediated decay and splice donor variants
(PMID: 8958207; 12707239; 22144547; 25611685).<br/>
<https://www.ncbi.nlm.nih.gov/clinvar/variation/43673/>; <br/>
<https://www.ncbi.nlm.nih.gov/clinvar/variation/177636/> <br/><br/>
Walsh et al found non truncating variants in the TNNT2 gene to be
significantly enriched in HCM cases, odds ratio 11.4 (8.5 - 15.2). They
found a clustering of non-truncating variants conveying a high
probability of pathogenicity in the tropomyosin binding domain (amino
acid residues 79 - 179) (PMID: 30696458).

#### **TPM1** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TPM1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
191010
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TPM1-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12860912; 7898523; 8205619; 24005378
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0007267
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
10.10.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
24.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

TPM1 pathogenic variants cause HCM through **altered gene product
sequence**. TPM1 encodes alpha-tropomyosin which acts to place the
troponin complex on cardiac actin. The mechanism is through altered
function rather than haploinsufficiency - Bottinelli et al examined the
Asp175Asn variant from 2 HCM patients and found equal expression of wild
type and mutant alpha-tropomyosin proteins (PMID: 9440709). Gupte et al
concluded that ‘TPM1 mutations cause differences in protein stability,
actin binding, and Tn conformation. All of these differences could
converge to change the Ca2+ dependence of myosin activity’ (PMID:
25548289).<br/> TPM1 variants account for \<3% of HCM cases (NBK1768;
PMID: 7898523; 12860912).<br/> **Inheritance is autosomal dominant
characterised by incomplete penetrance** (PMID: 32731933). <br/> There
are reports of homozygous or compound heterozygous variants in TPM1.
(PMID: 33642254; 32744700).<br/> A homozygous missense variant,
(p.Gly3Arg), in exon 1 of TPM1 was identified in triplets (two had HCM
and one patent ductus arteriosus). The parents were heterozygous for the
variant and unaffected clinically and on echocardiogram (PMID:
32744700).<br/><br/> **Pathogenic variants are nearly always
heterozygous missense variants**. There is one de novo inframe deletion
(6bp, 2 amino acids) associated with HCM classified as likely pathogenic
on ClinVar, no additional details are given. <br/> Redwood and Robinson
reviewed TPM1 variants in 2013 and reported ‘at least 15 described in
the current literature (Table1). Most of these mutations are unique and
have been reported in only a single family or individual. Each mutation
is missense causing a single amino acid substitution, and no truncation
or nonsense mutations have been reported.’ (PMID: 24005378).<br/> Walsh
et al found that in HCM non-truncating variants across the whole TPM1
gene showed a high probability of pathogenicity. (PMID: 30696458)

#### **JPH2** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
JPH2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
605267
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
JPH2-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

#### **TRIM63** - *HCM* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TRIM63
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
606131
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TRIM63-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

#### **MT-TI** - *HCM* - *Mitochondrial*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
MT-TI
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
590045
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MT-TI-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
mitochondrial
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Mitochondrial
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

#### **KLHL24** - *HCM* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
KLHL24
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
611295
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Familial hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
KLHL24-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
</tbody>
</table>

***Narrative***

The gene disease validity assertion is ‘moderate’ from ClinGen. No
inheritance or mechanism curations have been done in the absence of an
established gene disease relationship.

### Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH

#### **ALPK3** - *HCM* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
ALPK3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
617608
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
ALPK3-related Hypertrophic cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
34263907; 21441111; 33191771; 34263911; 32480058; 31074094; 27106955;
26846950
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0054838
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
11.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
24.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

ALPK3 pathogenic variants cause hypertrophic cardiomyopathy through
**decreased gene product level or altered gene product sequence**.
Biallelic ALPK3 variants were first associated with a rare recessive
form of cardiomyopathy by Almomani et al in 2016 (<PMID:26846950>), with
several other reports since (<PMID:27106955>; PMID: 31074094; PMID:
32480058). Patients display variable phenotypes but often present at
birth or early childhood (at least 4 patients have presented in utero)
with dilated cardiomyopathy (DCM) that progressed to a HCM phenotype
over time. Extra-cardiac features including musculoskeletal and
craniofacial abnormalities are also commonly observed in these cases
(PMID: 32480058; PMID: 34263911).<br/> Herkert et al 2020 reviewed the
variants and phenotype in 19 paediatric patients with biallelic ALPK3
variants (including 9 previously published cases) and identified 11
loss-of-function (LoF) variants (including nonsense, frameshift and
intronic variants with predicted severe effect on splicing), seven
compound LoF and deleterious missense variants, and one homozygous
deleterious missense variant, c.5155G\>C, p.(Ala1719Pro) (PMID:
32480058). The clinical manifestations associated with the missense
variants were similar to those associated with other damaging ALPK3
variants. <br/><br/> To note heterozygous LoF ALPK3 variants have also
been reported and are enriched in adults with cardiomyopathy (PMID:
34263907; PMID: 32480058; PMID: 33191771). Haploinsufficiency is the
proposed mechanism. Herkert et al found notable differences between the
clinical features associated with monoallelic and biallelic ALPK3
cardiomyopathy, including absence or undetected extracardiac phenotypes.
‘Whether these differences reflect graded dose-responses to ALPK3
deficits or distinct mechanisms by which monoallelic or biallelic
variants cause disease remains unknown.’(PMID: 32480058). <br/> There
does not appear to be a distinct mechanism between monoallelic and
biallelic variants. The disease mechanism is **loss of function of
ALPK3** likely due to creation of premature stop codons, leading to
nonsense-mediated decay or truncated proteins with partial or complete
removal of the kinase domain (<PMID:26846950>; 27106955; 32480058;
21441111). Herkert et al predicted that missense variants could result
in a conformational change that affects protein folding or flexibility,
protein-protein or protein-DNA interaction, or the activity of the
alpha-kinase domain (PMID: 32480058).

#### **CACNA1C** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CACNA1C
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
114205
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CACNA1C-related Timothy syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typically de novo
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations; activating
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
A recurrent, de novo missense_variant causing Classic Timothy Syndrome
has been described: NM_001167623.2(CACNA1C):c.1216G\>C (p.Gly406Arg)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
28211989; 25633834; 15863612; 15454078; 22106044; 26253506; 4728418;
31983240; NBK1403; 30681346; 33797204; 16360093
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0010979
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
11.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
24.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

CACNA1C-related Timothy syndrome is caused by variants leading to
**altered gene product sequence**. <br/> Classic Timothy syndrome (TS1)
is a very rare multisystem disorder characterized by marked QT
prolongation, syndactyly, immune deficiency, seizures, congenital heart
defects, hypertrophic cardiomyopathy, cognitive abnormalities, learning
difficulties, and intermittent hypoglycaemia (PMID: 28211989; NBK1403).
Infants can present with severe biventricular hypertrophy (PMID
30681346). In some individuals left ventricular hypertrophy (LVH) may be
the presenting feature, without recognised syndromic features,
fulfilling a clinical diagnosis of HCM. <br/> **A recurrent, de novo,
missense variant** in CACNA1C was described in 13 Timothy syndrome
patients, p.Gly406Arg in exon 8A (PMID: 15454078; PMID: 15863612). The
mechanism appears to be gain-of-function through failed channel
inactivation. Boczek et al suggest Ca2+ mishandling may lead to
ventricular hypertrophy (PMID: 26253506).<br/><br/> CACNA1C has a
complex genomic structure that undergoes extensive alternative splicing.
Splawski et al identified 2 patients with **de novo missense variants in
exon 8** of an alternate splice form (p.Gly406Arg, analogous to the exon
8a variant, and p.Gly402Ser). This splice form represents 80% of all
cardiac mRNAs. The patients were described as having atypical Timothy
syndrome (TS2), presenting with a more severe cardiac phenotype
(biventricular hypertrophy, moderate biventricular dysfunction, more
severe QT prolongation and multiple arrythmias) and without syndactyly
(PMID: 15863612; PMID: 25633834).<br/> Boczek et al describe a novel
variant **p.Arg518Cys** as the probable pathogenic substrate for COTS
\[cardiac only Timothy syndrome\]. The phenotype included LQTS,
hypertrophic cardiomyopathy, congenital heart defects and sudden cardiac
death. Follow-up cohort analysis revealed two additional pedigrees, with
very similar phenotypes, having variants at the exact same amino acid
positions **p.Arg518Cys and p.Arg518His**. (PMID: 26253506)<br/><br/>
Other missense variants in CACNA1C have been reported in association
with isolated LQT (PMID: 26253506; 25633834; 24728418). However as of
2020, the ClinGen Cardiovascular Domain Working Group have classified
the strength of evidence supporting an association between CACNA1C and
LQTS as moderate (PMID: 31983240).

#### **DES** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DES
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125660
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DES-related Myofibrillar myopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
absent gene product; altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion; stop_gained_NMD_triggering;
stop_gained_NMD_escaping; frameshift_variant_NMD_triggering;
splice_acceptor_variant_NMD_escaping; splice_donor_variant_NMD_escaping;
frameshift_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
20718792; 11073539; 29926427; 19433360; 9736733; 31718026; 30681346;
16217025; 23815709; 19181099
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011076
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

DES-related myofibrillar myopathy is typically inherited as an autosomal
dominant condition. Penetrance is incomplete with age-related onset, but
simplex cases have often been found to be due to de novo heterozygous
variants rather than recessive variants. Over 100 different variants
have been reported, including missense variants, in-frame indels, and
splice-site variants leading to exon skipping.<br/> There is evidence
for a possible genotype-phenotype correlation for heterozygous DES
variants, with variants located in specific domains more likely to cause
either a neurological or cardiac phenotype.<br/> There is a wide range
of cardiac phenotypes associated with DES, including hypertrophic
cardiomyopathy, restrictive cardiomyopathy, dilated cardiomyopathy,
arrhythmogenic cardiomyopathy, and left-ventricular non-compaction.
Cardiac involvement can be the presenting feature, even without
recognised syndromic features, fulfilling a clinical diagnosis of
HCM.<br/> The mechanism of autosomal dominant disease is likely a
dominant-negative effect of protein-altering variants leading to
abnormal intermediate filament aggregation. The autosomal recessive
phenotype has been associated with an earlier age of onset, and has been
linked with combinations of protein-altering variants and/or null
variants. It has been suggested that biallelic loss of function leads to
the autosomal recessive phenotype, indicating a distinct mechanism of
disease to the dominant phenotype. The heterozygous parents of patients
with the recessive phenotype are reported as being clinically
unaffected, suggesting that heterozygous loss of function is tolerated.

#### **DES** - *HCM* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
DES
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
125660
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DES-related Myofibrillar myopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance; Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
absent gene product; altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion; stop_gained_NMD_triggering;
stop_gained_NMD_escaping; frameshift_variant_NMD_triggering;
splice_acceptor_variant_NMD_escaping; splice_donor_variant_NMD_escaping;
frameshift_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
20718792; 11073539; 29926427; 19433360; 9736733; 31718026; 30681346;
16217025; 23815709; 19181099
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0011076
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

DES-related myofibrillar myopathy is typically inherited as an autosomal
dominant condition. Penetrance is incomplete with age-related onset, but
simplex cases have often been found to be due to de novo heterozygous
variants rather than recessive variants. Over 100 different variants
have been reported, including missense variants, in-frame indels, and
splice-site variants leading to exon skipping.<br/> There is evidence
for a possible genotype-phenotype correlation for heterozygous DES
variants, with variants located in specific domains more likely to cause
either a neurological or cardiac phenotype.<br/> There is a wide range
of cardiac phenotypes associated with DES, including hypertrophic
cardiomyopathy, restrictive cardiomyopathy, dilated cardiomyopathy,
arrhythmogenic cardiomyopathy, and left-ventricular non-compaction.
Cardiac involvement can be the presenting feature, even without
recognised syndromic features, fulfilling a clinical diagnosis of
HCM.<br/> The mechanism of autosomal dominant disease is likely a
dominant-negative effect of protein-altering variants leading to
abnormal intermediate filament aggregation. The autosomal recessive
phenotype has been associated with an earlier age of onset, and has been
linked with combinations of protein-altering variants and/or null
variants. It has been suggested that biallelic loss of function leads to
the autosomal recessive phenotype, indicating a distinct mechanism of
disease to the dominant phenotype. The heterozygous parents of patients
with the recessive phenotype are reported as being clinically
unaffected, suggesting that heterozygous loss of function is tolerated.

#### **FHL1** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
FHL1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
300163
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
FHL1-related Emery Dreifuss muscular dystrophy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; stop_lost; missense_variant;
stop_gained_NMD_escaping; frameshift_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
22523091; 19716112; 32993534; 20186852; 30681346
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0010680
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

FHL1-related Emery-Dreifuss muscular dystrophy is inherited as an
**X-linked condition**. Carrier females are usually unaffected, but some
exhibit a cardiac and/or skeletal muscle phenotype.<br/> The precise
mechanism is unclear, but may be mediated by reduced function or
dominant-negative effects of altered FHL1, associated with
frameshift/truncating variants predicted to escape nonsense mediated
decay (NMD), missense variants, or loss of the native stop codon.<br/>
FHL1 pathogenic variants are responsible for a minority of
Emery-Dreifuss muscular dystrophy cases (~1.2% according to
GeneReviews). Hypertrophic cardiomyopathy is more typical of
FHL1-related Emery-Dreifuss muscular dystrophy, while LMNA and EMD
Emery-Dreifuss muscular dystrophy are more commonly associated with
dilated cardiomyopathy.<br/> There are reports of FHL1 variants causing
isolated hypertrophic cardiomyopathy, without skeletal myopathy.

#### **FLNC** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
FLNC
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
102565
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
FLNC-related Myofibrillar myopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion; stop_gained_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31245841; 26666891; 21135393; 25351925; 15929027; 30681346; 27908349;
32022900; 32112656; 20697107; 19050726
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012289
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

FLNC-related myofibrillar myopathy is inherited as an autosomal dominant
condition with age-related onset. Myofibrillar myopathy and isolated
hypertrophic cardiomyopathy have been associated with protein-altering
variants in FLNC (missense, in-frame deletion, and nonsense predicted to
escape nonsense mediated decay (NMD)).<br/> The specific mechanism of
disease for pathogenic missense variants is not clear, but there is a
cluster of HCM-related variants in the ROD2 domain, suggesting that
these variants may affect interaction with the sarcomeric Z-disk.
Variants associated with skeletal myopathy have been shown to induce
intracellular aggregation of mutant FLNC.<br/> Loss-of-function variants
in FLNC have been associated with non-syndromic dilated and
arrhythmogenic cardiomyopathy.<br/> Sequencing of FLNC exons 46-48 is
complicated by the presence of a pseudogene with \>98% homology.

#### **GLA** - *HCM* - *X-linked recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
GLA
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
300644
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
GLA-related Fabry disease
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_X_hem
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
X-linked recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
A recurrent intronic variant NM_000169.3(GLA):c.640-801G\>A is
recognised as pathogenic and leads to aberrant mRNA splicing
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant; inframe_deletion; intron_variant;
inframe_insertion; stop_gained_NMD_triggering;
splice_donor_variant_NMD_triggering; frameshift_variant_NMD_triggering;
splice_acceptor_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
34776082; 30988410; 18940466; 6023233; 27560961; 26937390; 34576250;
32640076; 2539398; 11322659
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0010526
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
15.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
24.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic GLA variants cause Fabry disease by **decreased gene product
level or altered gene product sequence**. The disease mechanism is loss
of function (PMID: 18940466; 26937390; 6023233; 2539398). GLA pathogenic
variants result in mRNA instability and/or severely truncated
a-galactosidase A (a-Gal A) enzyme or an enzyme with markedly decreased
activity (NBK1292).<br/><br/> Fabry disease is an X-linked lysosomal
storage disease caused by pathogenic variants in the GLA gene leading to
a greatly reduced or absent activity of a-Gal A, responsible for
metabolizing glycosphingolipids. This condition is associated with a
progressive accumulation of globotriaosylceramide (Gb3) and its
deacylated form, globotriaosylsphingosine (lysoGb3), potentially
affecting any organ or tissue (PMID: 32640076).<br/> **Fabry disease is
inherited in an X linked manner**. Heterozygous females typically have
milder symptoms at a later age of onset than males. Rarely, they may be
relatively asymptomatic throughout a normal life span or may have
symptoms as severe as those observed in males with the classic phenotype
(NBK1292). <br/><br/> Variant classes include missense, nonsense, splice
site, frameshift, in-frame deletions, and structural variants. A
recurrent intronic variant (c.640-801G\>A) is recognised as pathogenic
and leads to aberrant mRNA splicing. <br/> Many variants are unique
however there are recognised recurrent variants also. (NBK1292; PMID
27560961; 30988410; 11322659; 34576250). <br/> Pathogenic variants
leading to complete loss of function of the gene product are usually
associated with classic forms of the disease, whereas variants resulting
in amino acid substitutions and residual enzyme activity can present
atypically with either symptoms not specific to Fabry’s
(e.g. cardiomyopathy) or a milder phenotype and later onset. Attempts to
correlate genotype with clinical presentation have been largely
unsuccessful. (PMID 27560961; 18940466). Experts noted that conduction
disease can be the presenting or only feature of disease.<br/><br/> The
expert consensus on the management of Fabry Disease in 2020 suggested
that ‘Assessment of plasma lyso-Gb3 should be considered for assessment
of disease severity in FD patients or in the diagnostic algorithm for
patients with GLA genetic variants of unknown significance.’ (PMID:
32640076)

#### **LAMP2** - *HCM* - *X-linked dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
LAMP2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
309060
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
LAMP2-related Danon disease
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_X_het
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
X-linked dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; start_lost; missense_variant;
stop_gained_NMD_triggering; splice_donor_variant_NMD_triggering;
frameshift_variant_NMD_triggering;
splice_acceptor_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
19057086; 30681346; 19588270; 20173215; 10972294; 15673802; 30857840;
15907287; 16217705
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0010281
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic LAMP2 variants cause Danon disease. Danon disease follows
X-linked inheritance, with high penetrance of cardiomyopathy (dilated or
hypertrophic) in hemizygous males. Onset is age-related, with an older
age of onset in females than in hemizygous males, and males are more
likely to have syndromic involvement (including skeletal myopathy and
cognitive impairment).<br/> **In the majority of cases, the mechanism is
loss of function due to decreased gene product**, as a result of
frameshift variants, nonsense variants, splice-site variants, and
copy-number variants. Individual pathogenic missense variants have been
reported. Some missense variants also cause a reduction in protein
level, e.g. due to aberrant splicing, but there are pathogenic missense
variants which have been shown to affect protein structure rather than
reducing protein expression.<br/> A possible transcript-specific
mechanism has been suggested, with variants affecting exon 9b reported
as causing a primarily skeletal muscle phenotype, with limited cardiac
involvement.

#### **PRKAG2** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
PRKAG2
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
602743
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
PRKAG2-related cardiomyopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
28009297; 11371514; 26729852; 12015471; 32646569; 32259713; 32646570
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0010946
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
20.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
24.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

PRKAG2 pathogenic variants cause disease through **altered gene product
sequence**. <br/> PRKAG2 syndrome is a rare, early-onset autosomal
dominant inherited disease, characterized by ventricular pre-excitation,
supraventricular arrhythmias, and cardiac hypertrophy (PMID: 26729852;
32259713; 32646570).<br/> There is a debate about penetrance but there
appears to be variable expressivity of the clinical phenotype which may
be variant specific (PMID: 32646569; 12015471).<br/><br/> PRKAG2
variants have been recognized mainly in the context of patients with
non-sarcomeric familial hypertrophic cardiomyopathy associated with
Wolff-Parkinson-White (WPW) syndrome (PMID: 26729852). The PRKAG2 gene
encodes for the 5’ Adenosine Monophosphate-Activated Protein Kinase
(AMPK), specifically for its gamma2 regulatory subunit (PRKAG2). PRKAG2
pathogenic variants are suspected to modify the tri-dimensional
structure of AMPK, altering its affinity for AMP and modifying the
enzyme activity (PMID: 32259713; 28009297).<br/><br/> **Nearly all
pathogenic variants are missense** (PMID: 32259713; 32646569; 12015471;
28009297). <br/> Blair et al. documented a TTA codon insertion in exon 5
in 2001 in a family affected by severe early onset cardiomyopathy and
multiple sudden deaths in early adult life (PMID: 11371514). The variant
is in a highly conserved region and co-segregated with disease. This has
not been reported again on ClinVar.<br/> Lopez-Sainz A et al reported
one frameshift and one intronic variant (the rest missense) in their
multi-centre retrospective study of 90 PRKAG2 variant carriers in 2020
(PMID: 32646569). However, the significance of truncating variants
remains uncertain with insufficient evidence to support loss of function
as a mechanism of disease. ClinGen found no evidence to support
haploinsufficiency as a mechanism
(<https://search.clinicalgenome.org/kb/gene-dosage/HGNC:9386>). <br/>
Two commonly reported variants are C.905G\>A (p.Arg302Gln) and
c.1463A\>T (p.Asn488Ile). Lopez-Sainz A et al found they were present in
44% of the patients included in the cohort (PMID: 32646569).

#### **PTPN11** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
PTPN11
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
176876
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
PTPN11-related Noonan syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
15521065; 21533187; 24739123; 16358218; 30681346; 25974318; 15240615;
14974085; 18348260; 21269411; 11992261; 19760651; 23312968
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0008104
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

PTPN11-related Noonan syndrome is inherited as an autosomal dominant
condition with variable expressivity. Pulmonary stenosis is the most
common cardiac manifestation. The penetrance of hypertrophic
cardiomyopathy in PTPN11-related Noonan syndrome has been reported as
20-30%.<br/> There is no definite evidence of PTPN11 variants causing
isolated cardiac disease without other features of Noonan syndrome, but
the syndromic phenotype can be subtle and may be underdiagnosed. This
suggests that cardiac involvement can be the initial presentation in
some cases, and PTPN11 should be considered in the differential
diagnosis of HCM.<br/> PTPN11 variants are the most common cause of
Noonan syndrome, accounting for approximately 50% of cases. The
mechanism is likely gain of function due to missense variants
(particularly variants disrupting interaction between the N-SH2 and PTP
domains), leading to activation of the RAS-MAPK pathway. There are at
least two reports of Noonan syndrome due to single-residue in-frame
deletions in PTPN11.<br/> There are reports of structural duplications
including PTPN11 causing Noonan syndrome, but all patients reported to
date have had a relatively non-specific phenotype and large duplications
encompassing other genes. It is therefore not clear whether gene
duplication specifically causes Noonan syndrome.<br/> Allelic disorders
include Noonan syndrome with multiple lentigines (NSML, previously known
as LEOPARD syndrome), caused by dominant-negative PTPN11 variants, and
metachondromatosis, caused by loss-of-function PTPN11 variants.

#### **PTPN11** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
PTPN11
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
176876
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
PTPN11-related Noonan syndrome with multiple lentigines
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
16377799; 15121796; 12161596; 16358218; 12058348; 33354767; 11992261;
17697839
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0100082
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

Noonan syndrome with multiple lentigines (NSML, previously known as
LEOPARD syndrome) is inherited as an autosomal dominant condition.
Missense variants in PTPN11 are responsible for the majority of cases.
Rare cases have been attributed to variants in RAF1, BRAF, and MAP2K1.
There is no definite evidence of PTPN11 variants causing isolated
cardiac disease without other features of NSML, but the syndromic
phenotype can be subtle and may be underdiagnosed. This suggests that
cardiac involvement can be the initial presentation in some cases, and
PTPN11 should be considered in the differential diagnosis of HCM.<br/>
PTPN11 variants associated with NSML are clustered in the
phosphotyrosine phosphatase (PTP) domain and have been shown to inhibit
phosphatase activity. Since null variants have not been associated with
the NSML phenotype, it has been suggested that NSML is caused by a
dominant-negative effect rather than simple loss of function, possibly
due to altered interaction between PTPN11 and binding partners.<br/>
Allelic disorders include Noonan syndrome, caused by gain-of-function
variants in PTPN11, and metachondromatosis, caused by loss-of-function
PTPN11 variants.

#### **RAF1** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
RAF1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
164760
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
RAF1-related Noonan syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
29271604; 17603483; 30681346; 25974318; 17603482; 22786616; 24782337;
30762279
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0012690
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

RAF1 variants are responsible for ~5% of Noonan syndrome cases.
RAF1-related Noonan syndrome is inherited as an autosomal dominant
condition.<br/> The mechanism is likely RAF1 gain of function, due to
missense variants leading to activation of the Ras-MAPK pathway.<br/>
The penetrance of hypertrophic cardiomyopathy is higher in patients with
RAF1 variants, compared to other forms of Noonan syndrome. There is no
definite evidence of RAF1 variants causing isolated cardiac disease
without other features of Noonan syndrome, but RAF1 variants have been
identified in multiple patients who were being investigated for HCM,
indicating that the syndromic phenotype can be subtle and may be
underdiagnosed. Since cardiac involvement can be the initial
presentation in some cases, RAF1 should be considered in the
differential diagnosis of HCM.<br/> There is evidence for a
genotype-phenotype correlation, with variants in the CR2 domain of RAF1
more likely to be associated with HCM. There is a further mutational
hotspot in the kinase domain of CR3, but pathogenic variants have also
been identified outside these hotspots.<br/> There are reports of
patients with chromosomal duplications including RAF1, but their
phenotype is relatively non-specific and there is no definitive evidence
that whole-gene duplications cause Noonan syndrome.

#### **RIT1** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
RIT1
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
609591
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
RIT1-related Noonan syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
26446362; 27101134; 30681346; 25959749; 23791108
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0014143
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

RIT1 variants are responsible for ~5% of Noonan syndrome cases.
RIT1-related Noonan syndrome is inherited as an autosomal dominant
condition.<br/> The penetrance of hypertrophic cardiomyopathy is higher
in patients with RIT1 variants, compared to other forms of Noonan
syndrome (estimated at 70-75%). There is no definite evidence of RIT1
variants causing isolated cardiac disease without other features of
Noonan syndrome, but the syndromic phenotype can be subtle and may be
underdiagnosed. This suggests that cardiac involvement can be the
initial presentation in some cases, and RIT1 should be considered in the
differential diagnosis of HCM.<br/> The mechanism is likely RIT1 gain of
function, due to missense variants leading to activation of the Ras-MAPK
pathway. The precise mechanism by which missense variants lead to gain
of function remains unclear. There is clustering of variants,
particularly in the switch II region.

#### **TTR** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
TTR
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
176300
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Rare syndromic disorder with hypertrophic cardiomyopathy - isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
TTR-related hereditary ATTR amyloidosis
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by incomplete penetrance
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; inframe_deletion; inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
32969287; 25604431; 9191784; 28213611; 15185500; 29941560; 30328212;
15930086
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0019441
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
11.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

The mechanism in TTR-related hereditary ATTR amyloidosis is due to
**altered gene product sequence**.<br/> **The mechanism appears to be
gain-of-function**; pathogenic variants cause either tetramer
dissociation or monomer denaturation, which both contribute to the
formation of amyloid fibrils in tissue (PMID 25604431; NBK1194).<br/> Of
note, wildtype TTR can also cause an age-related late onset form of
amyloidosis where patients almost exclusively present with cardiac
involvement.<br/><br/> TTR-related hereditary ATTR amyloidosis is
characterised by **autosomal dominant inheritance with incomplete
penetrance**. There are reports of homozygous and compound heterozygous
variants. Biallelic variants appear to cause a more severe phenotype
(PMID: 28213611; 15185500; 30328212), however one study reported no
difference in phenotype between homozygous and heterozygous carriers
(PMID: 15930086).<br/><br/> TTR-related hereditary ATTR amyloidosis
includes the phenotypes ATTR amyloid neuropathy, ATTR cardiac
amyloidosis, ATTR leptomeningeal/CNS amyloidosis. Phenotype varies
considerably; individuals can present with multi-systemic phenotypes
including, polyneuropathy, carpal tunnel syndrome, cardiomyopathy,
gastrointestinal features, autonomic insufficiency, and renal
insufficiency. ‘The prevalence of HCM and/or RCM in Hereditary ATTR
amyloidosis is high but exact percentages have not been accurately
defined due the high prevalence of HCM and RCM (~25%) development in
senile cardiac amyloidosis, which occurs due to accumulation of wildtype
TTR accumulation in the heart with age.’ (ClinGen summary)<br/><br/>
More than 100 different variants have been reported in the Hereditary
Amyloidosis Registry (<http://www.amyloidosismutations.com>). <br/>
**Variants are all missense apart from one inframe deletion of a valine
residue in exon 4 (Plasma transthyretin levels in the mutant gene
carriers measured by nephelometry were very low) and one 6 nucleotide
duplication in exon 3**. The duplication was reported to be associated
with a particularly aggressive phenotype and although it did not alter
the protein secondary or tertiary structure, it decreased the stability
of the TTR monomer and tetramer (PMID: 9191784; 29941560).<br/> Several
variants including TTR-V30M, TTR-T60A, and TTR-V122L are commonly
associated with cardiac amyloidosis (PMID: 28739313). There are other
genotype phenotype correlations (PMID: 25604431; NBK1194).<br/><br/>The
ATTR Val122Ile variant (also reported as pVal142Ile) is a risk allele in
patient subpopulations of African American descent. The variant in this
population has a high prevalence (estimated 3-4%) and patients are at
particular risk for developing cardiac-related ATTR. (PMID: 28213611).
<br/><br/> A systematic review in 2020 evaluating specific therapies for
transthyretin cardiac amyloidosis supported the use of tafamidis (a
tetramer stabiliser) and noted that other novel therapeutic targets
including transthyretin gene silencers are currently under investigation
(PMID: 32969287).

### Syndrome with hypertrophic cardiomyopathy - no isolated LVH

#### **ABCC9** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
ABCC9
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
601439
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndrome with hypertrophic cardiomyopathy - no isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
ABCC9-related Cantu Syndrome
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
gain of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
31828977; 22608503; 34453476; 33529173; 31175705; 22610116; 30089727
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0009406
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
11.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
13.01.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DDG2P team
</td>
</tr>
</tbody>
</table>

***Narrative***

ABCC9 pathogenic variants cause Cantu Syndrome (CS) through **altered
gene product sequence**. The mechanism is likely gain of function
(NBK246980; PMIDs: 22610116; 31175705; 30089727).<br/> Harakalova et al
showed that variants in ABCC9 reduce the ATP-mediated potassium channel
inhibition, resulting in channel opening (PMID: 22610116).
Pathophysiologic mechanisms resulting in the clinical manifestations
remain poorly understood. <br/><br/> Cantu syndrome is characterized by
congenital hypertrichosis, distinctive facial features,
osteochondrodysplasia and cardiac defects. The cardiac manifestations
include patent ductus arteriosus, cardiomegaly, hypertrophy and
pericardial effusion. Our literature review has not found patients who
have presented with isolated left ventricular hypertrophy (LVH). All
participants with LVH in the Cantu registry who harboured a pathogenic
missense variant have at least some additional features in keeping with
Cantu syndrome, even if mild. There is however significant variability
in phenotype even amongst family members with the same variant (PMID
33529173) and no clear genotype phenotype correlation (PMID
31828977).<br/><br/> Inheritance is autosomal dominant characterized by
variable expressivity.<br/><br/> De novo variants are common, Grange et
al found only 16/74 (22%) patients had inherited the ABCC9 variant from
an affected parent (PMID: 31828977). It is caused by heterozygous
missense mutations in ABCC9 (or less commonly variants in KCNJ8). A
review of 74 patients in the International Registry for Cantu syndrome
in 2019 found 28 distinct variants (97% were in ABCC9). All were simple
missense apart from one insertion deletion involving 2 nucleotides
(c.3055_3056 del ins GA). Missense variants were located throughout the
protein, however a majority (68%) clustered around transmembrane domain
2 (TMD2). Variants at NM_020297.3(ABCC9): p.Arg1154 (identified in 24
subjects, 33%) and NM_020297.3(ABCC9):p.Arg1116 (identified in 11
subjects, 15%), were most commonly observed (PMID: 31828977). Zhang et
al 2021 showed that p.Arg1154Gln variants in genetically modified mice
might result in an effective mixed loss/gain of function phenotype,
potentially explaining the diverse disease features in patients with CS.
The mechanisms resulting in the clinical manifestations of CS remain to
be fully elucidated (PMIDs 33529173; 34453476).

#### **BAG3** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
BAG3
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
603883
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndrome with hypertrophic cardiomyopathy - no isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
BAG3-related Myofibrillar myopathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typically de novo
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NM_004281.4(BAG3):c.626C\>T (p.Pro209Leu)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
23995273; 20605452; 32453099; 22734908; 25728519; 19085932; 21361913
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0018943
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
11.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
12.01.2022
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Cardiomyopathy expert panel
</td>
</tr>
</tbody>
</table>

***Narrative***

BAG3 pathogenic variants cause a rare subtype of myofibrillar myopathy
through **altered gene product sequence**.<br/><br/> Of note, loss of
function of BAG3 is an established cause of dilated cardiomyopathy.
Multiple missense, nonsense, frameshift variants have been described in
association with DCM and there is sufficient evidence for
haploinsufficiency as a mechanism
(<https://search.clinicalgenome.org/kb/genes/HGNC:939>). <br/><br/> In
contrast, a specific (missense) variant has been associated with
myofibrillar myopathy (MFM). Typical presentation is toe walking and
clumsiness with progression to cardiomyopathy and restrictive lung
disease. The majority of evidence comes from one recurrent missense
variant which causes MFM with prominent cardiac involvement. To our
knowledge isolated hypertrophic or restrictive cardiomyopathy without
skeletal muscle pathology has not been reported for BAG3 related MFM. In
2013, Olive et al (PMID: 23995273) summarised that only 12 patients with
BAG3 related MFM, all carrying the same p.P209L recurrent de-novo
mutation transmitted from a mosaic parent, have been reported to date
(PMID: 19085932; 22734908; 21361913; 20605452). <br/><br/> A missense
variant, p.Ala558Val, was reported in an infant with myofibrillar
myopathy in 2011 (<https://doi.org/10.1016/j.nmd.2011.06.1083>) and the
variant Pro209Gln is described in a patient with adult onset disease in
2014 (PMID 25208129). Malatesta L et al 2020 found that most patients
who carry the p.Pro209Leu variant have cardiomyopathy (restrictive or
hypertrophic) (14 of 18 patients) (PMID: 32453099). BAG3 is a
cytoprotective protein that plays a role in the breakdown of misfolded
proteins. It is produced in response to stress and is thought to be
essential for the survival of mature skeletal and heart muscle cells by
inhibiting the activation of apoptosis (PMID 25728519).

#### **CRYAB** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
CRYAB
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
123590
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndrome with hypertrophic cardiomyopathy - no isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
CRYAB-related Alpha-related B crystallinpathy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
undetermined non-loss-of-function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
missense_variant; stop_gained_NMD_escaping;
frameshift_variant_NMD_escaping; stop_lost
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
14681890; 21130652; 21337604; 570292; 28493373; 9731540; 30681346;
32420686; 21920752; 38212463; 33272090
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0013472
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
13.01.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DDG2P team
</td>
</tr>
</tbody>
</table>

***Narrative***

CRYAB-related alpha-crystallinopathy is typically inherited as an
autosomal dominant condition with age-related penetrance.<br/><br/> The
mechanism is likely **altered gene sequence** of CRYAB due to either (1)
missense variants or (2) truncating variants in the 3’ region which are
predicted to escape nonsense mediated decay (NMD). A dominant-negative
mechanism has been suggested (Selcen et al 2003 <PMID:14681890>, Wang et
al 2001 <PMID:11440982>). In addition two stop lost variants have been
reported (PMIDs:38212463, 33272090).<br/><br/> Variants in CRYAB have
been associated with systemic features of crystallinopathy (cataract,
skeletal myopathy) with and without cardiac involvement. The
cardiomyopathy phenotype includes various subtypes. Our literature
search did not identify cases presenting with isolated left ventricular
hypertrophy (LVH).<br/><br/> Evidence for dosage sensitivity is limited.
Homozygous truncating variants predicted to cause NMD have been
associated with autosomal recessive congenital/early-onset cataracts and
myopathy with respiratory involvement in early childhood, and the
heterozygous parents of probands are reported as being healthy.

#### **FXN** - *HCM* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
FXN
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
606829
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndrome with hypertrophic cardiomyopathy - no isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
FXN-related Friedreich ataxia
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; start_lost; inframe_deletion; inframe_insertion;
splice_donor_variant_NMD_triggering; frameshift_variant_NMD_triggering;
splice_acceptor_variant_NMD_triggering; trinucleotide_repeat_expansion;
exon_loss_variant; transcript_ablation
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
28405347; 25566998; 30681346; 24705334; 10633128; 22409940; 26704351;
22691228
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0100339
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
13.01.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DDG2P team
</td>
</tr>
</tbody>
</table>

***Narrative***

FXN-related Friedreich ataxia is inherited as an autosomal recessive
condition. It is characterised by a progressive ataxia usually
manifesting as poor balance when walking before progressing to upper
limb ataxia and dysarthria. Two thirds of individuals have hypertrophic
cardiomyopathy (HCM); diabetes mellitus can also be associated. Our
literature search did not identify cases presenting with isolated left
ventricular hypertrophy (LVH). Heterozygous carriers of FXN expansions
or other pathogenic variants have not been reported to develop
symptoms.<br/><br/> Homozygous expansion of the GAA triplet repeat in
exon 1 of FXN is responsible for the majority of cases (95-98%). There
is an inverse correlation between the size of the expansion and age of
onset (particularly the size of the shorter allele). Anticipation is not
observed as the disorder follows autosomal recessive
inheritance.<br/><br/> The remaining cases are due to compound
heterozygosity for a GAA expansion and a sequence variant (truncating,
splice-site, or missense) or structural variant (single/multi-exon or
whole-gene deletion). There is evidence that compound heterozygosity is
associated with earlier onset of symptoms than homozygosity for the GAA
triplet repeat expansion, but cardiomyopathy is more common in patients
with a homozygous GAA triplet repeat expansion. <br/><br/> The mechanism
is likely loss of function of *FXN* due to transcriptional silencing (in
the case of the GAA triplet repeat), absent/decreased gene product
(truncating variants, structural variants), or missense variants causing
loss of function.

#### **GAA** - *HCM* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
GAA
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
606800
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndrome with hypertrophic cardiomyopathy - no isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
GAA-related Pompe disease
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
restricted repertoire of mutations
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
A specific 5’ UTR variant, NM_000152.5(GAA):c.-32-13T\>G, ClinVar
variation ID 4027 has been reported in 40-70% of the alleles in patients
affected with late onset disease (PMID 24150945; 7881425).
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_region_variant; splice_acceptor_variant_NMD_triggering;
splice_donor_variant_NMD_triggering; start_lost;
frameshift_variant_NMD_escaping; frameshift_variant_NMD_triggering;
stop_gained_NMD_triggering; missense_variant; inframe_deletion;
transcript_ablation; 5_prime_UTR_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
1680134; 1856189; 9668092; 14643388; 19588081; 31254424; 9535769;
8094613; 1652892; 7717400; 20882352; 30681346; 21484825; 11991748;
7981676; 10973860; 28185884; 31342611; 4286143; 17056254; 3108320
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0009290
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
13.01.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DDG2P team
</td>
</tr>
</tbody>
</table>

***Narrative***

Pompe disease is inherited as an autosomal recessive condition, caused
by homozygous or compound heterozygous variants in GAA. Infantile-onset
Pompe disease is characterised by hypotonia, generalized muscle
weakness, feeding difficulties, failure to thrive, respiratory distress,
and hypertrophic cardiomyopathy; late onset Pompe disease is
characterised by proximal muscle weakness and respiratory insufficiency.
**The mechanism of disease is loss of function** of acid
alpha-glucosidase, resulting in lysosomal glycogen
accumulation.<br/><br/> There are currently over 900 variants in the
Pompe variant database (N.B. this includes all known variants,
regardless of pathogenicity). Multiple classes of variant have been
shown to cause Pompe disease, including loss-of-function variants,
missense variants, splice-site variants, 5 prime UTR variants, intronic
variants, and structural variants.<br/> A specific 5 prime UTR variant,
NM_000152.5(GAA):c.-32-13T\>G, (the variant is 13 bases upstream of a
splice junction, which is located in the UTR 32 bases upstream of the
start codon), ClinVar variation ID 4027, has been reported in 40-70% of
the alleles in patients affected with late onset disease (PMID
24150945). Functional studies have shown the presence of the
c.-32-13T\>G variant causes aberrant splicing, resulting in exclusion of
exon 2 in the processed transcript (PMID 7881425) <br/><br/> There is a
genotype-phenotype correlation, with biallelic loss of function expected
to result in a severe phenotype (infantile-onset Pompe disease), while
patients with residual enzymatic activity develop late-onset Pompe
disease. Cardiac involvement is a classical feature of infantile-onset
Pompe disease, but has also been reported in the late-onset form. Our
literature search did not identify cases presenting with isolated left
ventricular hypertrophy (LVH).

#### **MYO6** - *HCM* - *Autosomal dominant*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
MYO6
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600970
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndrome with hypertrophic cardiomyopathy - no isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MYO6-related nonsyndromic genetic hearing loss with left ventricular
hypertrophy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
monoallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal dominant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
26265212; 15060111; 30681346; 18348273; 23485424; 12687499; 25999546;
24105371
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0019497
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
13.01.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DDG2P team
</td>
</tr>
</tbody>
</table>

***Narrative***

There is evidence from multiple reports that protein truncating and
missense variants in MYO6 are associated with autosomal dominant
progressive hearing loss. Homozygosity or compound heterozygosity is
associated with congenital hearing loss, suggesting a dose-dependent
mechanism. ClinGen classified the relationship between MYO6 and hearing
loss as Definitive. The relationship for hearing loss and left
ventricular hypertrophy (LVH) was not classified separately as there
wasn’t evidence of a distinct mechanism for the two phenotypes. Overall
the evidence for cardiac involvement is limited. <br/><br/> Mohiddin et
al. (2004, PMID: 15060111) reported a family with autosomal dominant
hearing loss who were initially ascertained due to the proband
presenting with hypertrophic cardiomyopathy. Linkage studies implicated
the region containing MYO6, and a missense variant was identified
(p.His246Arg) and segregated with disease (with the exception of one
individual who was negative for the variant but described as a double
phenocopy with presbycusis and cardiac abnormalities consistent with
valvar and ischaemic disease). QT prolongation was observed in 5 out of
12 carriers of the variant. Evidence of LVH was observed in 4 out of 11
carriers who underwent echocardiogram. No specific mechanism of disease
causation was suggested. <br/> Studies in transgenic mouse models have
suggested a link with cardiac hypertrophy, but only in biallelic
knockout mice. <br/><br/> In summary, there is a single report linking
the p.His246Arg variant in MYO6 with a hypertrophic cardiomyopathy
phenotype exhibiting incomplete penetrance (Mohiddin et al. 2004, PMID:
15060111). The authors note that they were unable to definitively rule
out an alternative cause for the cardiac phenotype in the family. This
phenotype has not been observed in other families presenting with
MYO6-related hearing loss (e.g., Sanggaard et al. 2008, PMID: 18348273;
Miyagawa et al. 2015, PMID: 25999546), including patients with
homozygous loss-of-function or missense variants (Ahmed et al. 2003,
PMID: 12687499). <br/><br/> The G2P team have assigned the relationship
between MYO6 and hearing loss and LVH as moderate to ensure that
variants in this gene are at least considered in a patient presenting
with hearing loss and LVH. Our literature search did not identify cases
presenting with isolated LVH.

#### **MYO6** - *HCM* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
MYO6
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
600970
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndrome with hypertrophic cardiomyopathy - no isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MYO6-related nonsyndromic genetic hearing loss with left ventricular
hypertrophy
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
MODERATE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Typified by age-related onset
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
altered gene product sequence; decreased gene product level
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_acceptor_variant; splice_donor_variant; frameshift_variant;
stop_gained; missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
26265212; 15060111; 30681346; 18348273; 23485424; 12687499; 25999546;
24105371
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0019497
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
27.03.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
13.01.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DDG2P team
</td>
</tr>
</tbody>
</table>

***Narrative***

There is evidence from multiple reports that protein truncating and
missense variants in MYO6 are associated with autosomal dominant
progressive hearing loss. Homozygosity or compound heterozygosity is
associated with congenital hearing loss, suggesting a dose-dependent
mechanism. ClinGen classified the relationship between MYO6 and hearing
loss as Definitive. The relationship for hearing loss and left
ventricular hypertrophy (LVH) was not classified separately as there
wasn’t evidence of a distinct mechanism for the two phenotypes. Overall
the evidence for cardiac involvement is limited. <br/><br/> Mohiddin et
al. (2004, PMID: 15060111) reported a family with autosomal dominant
hearing loss who were initially ascertained due to the proband
presenting with hypertrophic cardiomyopathy. Linkage studies implicated
the region containing MYO6, and a missense variant was identified
(p.His246Arg) and segregated with disease (with the exception of one
individual who was negative for the variant but described as a double
phenocopy with presbycusis and cardiac abnormalities consistent with
valvar and ischaemic disease). QT prolongation was observed in 5 out of
12 carriers of the variant. Evidence of LVH was observed in 4 out of 11
carriers who underwent echocardiogram. No specific mechanism of disease
causation was suggested. <br/> Studies in transgenic mouse models have
suggested a link with cardiac hypertrophy, but only in biallelic
knockout mice. <br/><br/> In summary, there is a single report linking
the p.His246Arg variant in MYO6 with a hypertrophic cardiomyopathy
phenotype exhibiting incomplete penetrance (Mohiddin et al. 2004, PMID:
15060111). The authors note that they were unable to definitively rule
out an alternative cause for the cardiac phenotype in the family. This
phenotype has not been observed in other families presenting with
MYO6-related hearing loss (e.g., Sanggaard et al. 2008, PMID: 18348273;
Miyagawa et al. 2015, PMID: 25999546), including patients with
homozygous loss-of-function or missense variants (Ahmed et al. 2003,
PMID: 12687499). <br/><br/> The G2P team have assigned the relationship
between MYO6 and hearing loss and LVH as moderate to ensure that
variants in this gene are at least considered in a patient presenting
with hearing loss and LVH. Our literature search did not identify cases
presenting with isolated LVH.

#### **SLC25A4** - *HCM* - *Autosomal recessive*

<table class="table table-striped table-responsive" style="font-size: 12px; font-family: sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
Gene symbol
</th>
<th style="text-align:left;font-weight: bold;font-size: 14px;">
SLC25A4
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Gene mim
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
103220
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
HCM
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Referral indication
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Hypertrophic Cardiomyopathy (HCM)
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease grouping
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Syndrome with hypertrophic cardiomyopathy - no isolated LVH
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Disease name
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
SLC25A4-related Mitochondrial disease
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
Gene disease validity (ClinGen)
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;font-weight: bold;">
DEFINITIVE
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Allelic requirement
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
biallelic_autosomal
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
Autosomal recessive
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Inheritance modifiers
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease-associated variant consequence
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
absent gene product; altered gene product sequence
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mechanism
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
loss of function
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Mutation consequence flag
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Restricted repertoire of pathogenic variants
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Variant classes reported with evidence of pathogenicity
<a href="#other-potentially-relevant-variant-classes">(Other potentially
relevant variant classes)</a>
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
splice_donor_variant; missense_variant;
frameshift_variant_NMD_triggering
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
PMIDs
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
22187496; 30013777; 16155110; 23401503; 25732997; 27693233
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Disease accession
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
MONDO:0044970
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Curated date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
11.11.2021
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel review date
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
13.01.2025
</td>
</tr>
<tr>
<td style="text-align:left;width: 8cm; word-wrap: break-word; white-space: normal;">
Expert panel
</td>
<td style="text-align:left;width: 10cm; word-wrap: break-word; white-space: normal;">
DDG2P team
</td>
</tr>
</tbody>
</table>

***Narrative***

Pathogenic variants in SLC25A4 cause mitochondrial disease through
**altered gene product sequence or absent gene product level**. SLC25A4
encodes AAC1 (also known as ANT1), an ADP/ATP carrier isoform expressed
at high levels in the skeletal muscle, heart and brain. <br/><br/> Loss
of function recessive mutations cause a mitochondrial myopathy and
cardiomyopathy phenotype that presents in childhood or early adulthood
(also known as mitochondrial DNA depletion syndrome 12). These comments
refer to this phenotype category. There are two other distinct phenotype
groups: inherited heterozygous missense variants have been associated
with an adult-onset autosomal-dominant progressive external
ophthalmoplegia; and 2 recurrent de novo missense variants have been
reported in seven individuals with a characteristic and severe early
onset mitochondrial disease (PMIDs: 27693233; 30013777). As far as we
are aware SCL25A4 variants do not present with isolated left ventricular
hypertrophy (LVH). <br/><br/> It is suggested that variants in SLC25A4
leading to transporter dysfunction causes insufficient adenine
nucleotide availability for dATP synthesis and consequent imbalanced
dNTP pools, leading to mtDNA depletion (PMID 27693233; 30013777). A
small number of homozygous and compound heterozygous variants have been
described including missense, frameshift and splice variants. There are
4 pathogenic submissions on ClinVar. All are predicted to cause loss of
function (<PMID:16155110>; <PMID:22187496>; <PMID:25732997>). Thompson
et al found that 3 previously reported recessive variants resulted in
\<1% transport activity compared to wild-type (PMID 27693233).

## Other potentially relevant variant classes

Which variant classes should I filter for? The main table shows variant
classes that have been reported to cause disease. We would expect other
variant classes with similar consequences to cause disease (e.g. if
NMD_triggering frameshift cause disease, then NMD_triggering nonsense
probably do too). This table below shows the full list of variant
classes mapped to each variant consequence.

<table class="table table-striped table-condensed" style="font-size: 12px; margin-left: auto; margin-right: auto;border-bottom: 0;">
<thead>
<tr>
<th style="border-bottom:hidden;padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="3">

<div style="border-bottom: 1px solid #ddd; padding-bottom: 5px; ">

Disease-associated variant consequence

</div>

</th>
</tr>
<tr>
<th style="text-align:left;">
Decreased_gene_product_level
</th>
<th style="text-align:left;">
Absent_gene_product_level
</th>
<th style="text-align:left;">
Altered_gene_product_sequence
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
splice_region_variant
</td>
<td style="text-align:left;">
splice_acceptor_variant
</td>
<td style="text-align:left;">
splice_acceptor_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_acceptor_variant
</td>
<td style="text-align:left;">
splice_acceptor_variant_NMD_triggering
</td>
<td style="text-align:left;">
splice_acceptor_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_acceptor_variant_NMD_triggering
</td>
<td style="text-align:left;">
splice_donor_variant
</td>
<td style="text-align:left;">
splice_donor_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_donor_variant
</td>
<td style="text-align:left;">
splice_donor_variant_NMD_triggering
</td>
<td style="text-align:left;">
splice_donor_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
splice_donor_variant_NMD_triggering
</td>
<td style="text-align:left;">
start_lost
</td>
<td style="text-align:left;">
frameshift_variant_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
start_lost
</td>
<td style="text-align:left;">
frameshift_variant
</td>
<td style="text-align:left;">
stop_gained_NMD_escaping
</td>
</tr>
<tr>
<td style="text-align:left;">
frameshift_variant
</td>
<td style="text-align:left;">
frameshift_variant_NMD_triggering
</td>
<td style="text-align:left;">
stop_lost
</td>
</tr>
<tr>
<td style="text-align:left;">
frameshift_variant_NMD_triggering
</td>
<td style="text-align:left;">
stop_gained
</td>
<td style="text-align:left;">
missense_variant
</td>
</tr>
<tr>
<td style="text-align:left;">
stop_gained
</td>
<td style="text-align:left;">
stop_gained_NMD_triggering
</td>
<td style="text-align:left;">
inframe_insertion
</td>
</tr>
<tr>
<td style="text-align:left;">
stop_gained_NMD_triggering
</td>
<td style="text-align:left;">
</td>
<td style="text-align:left;">
inframe_deletion
</td>
</tr>
</tbody>
<tfoot>
<tr>
<td style="padding: 0; " colspan="100%">
<span style="font-style: italic;text-decoration: underline;">Note:
</span>
</td>
</tr>
<tr>
<td style="padding: 0; " colspan="100%">
<sup></sup> Variant classes with a likelihood score of ‘3:possible’,
‘4:probable’ and ‘5:almost always’ <br>For the different disease
associated variant consequences, see Tables 2 and 3 from Roberts et al.
</td>
</tr>
</tfoot>
</table>

*Roberts AM, DiStefano MT, Riggs ER, Josephs KS, Alkuraya FS, Amberger
J, Amin M, Berg JS, Cunningham F, Eilbeck K, Firth HV, Foreman J, Hamosh
A, Hay E, Leigh S, Martin CL, McDonagh EM, Perrett D, Ramos EM, Robinson
PN, Rath A, Sant DW, Stark Z, Whiffin N, Rehm HL, Ware JS. Toward robust
clinical genome interpretation: Developing a consistent terminology to
characterize Mendelian disease-gene relationships-allelic requirement,
inheritance modes, and disease mechanisms. Genet Med. 2024
Feb;26(2):101029.* <https://doi.org/10.1016/j.gim.2023.101029>
